{"cells":[{"cell_type":"markdown","metadata":{"id":"Lmi2j555eDBa"},"source":["# Import Libraries"]},{"cell_type":"code","execution_count":1,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":36485,"status":"ok","timestamp":1729622616678,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"KS6pDBJNeMlt","outputId":"6d915d94-cea1-475a-9ef1-c2ee33661ad6"},"outputs":[{"name":"stdout","output_type":"stream","text":["Mounted at /content/drive\n"]}],"source":["from google.colab import drive\n","drive.mount('/content/drive')"]},{"cell_type":"code","execution_count":2,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":6285,"status":"ok","timestamp":1729622622950,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"Sf9G3s81fNV-","outputId":"f3ad1199-de33-4d91-8852-45f37fa38cb2"},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[?25l   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/644.1 kB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r\u001b[2K   \u001b[91m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[90m╺\u001b[0m\u001b[90m━━━━━━━━\u001b[0m \u001b[32m501.8/644.1 kB\u001b[0m \u001b[31m16.1 MB/s\u001b[0m eta \u001b[36m0:00:01\u001b[0m\r\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m644.1/644.1 kB\u001b[0m \u001b[31m12.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25h\u001b[?25l   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/129.1 kB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m129.1/129.1 kB\u001b[0m \u001b[31m9.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25h\u001b[?25l   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/5.2 MB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r\u001b[2K   \u001b[91m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[91m╸\u001b[0m\u001b[90m━\u001b[0m \u001b[32m5.1/5.2 MB\u001b[0m \u001b[31m124.2 MB/s\u001b[0m eta \u001b[36m0:00:01\u001b[0m\r\u001b[2K   \u001b[91m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[91m╸\u001b[0m \u001b[32m5.2/5.2 MB\u001b[0m \u001b[31m117.3 MB/s\u001b[0m eta \u001b[36m0:00:01\u001b[0m\r\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m5.2/5.2 MB\u001b[0m \u001b[31m65.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n","\u001b[?25h"]}],"source":["!pip -q install keras keras_nlp keras_tuner"]},{"cell_type":"code","execution_count":3,"metadata":{"executionInfo":{"elapsed":21,"status":"ok","timestamp":1729622622952,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"r2Er16RIZSMj"},"outputs":[],"source":["# Select path\n","path = '/content/drive/MyDrive/Portfolio/treat_cancer/' # colab rbalbinotti\n","# path = '/content/drive/MyDrive/treat_cancer/' # colab\n","# path = '' # vscode\n","\n","# Select if Train Mode\n","train_mode = False"]},{"cell_type":"code","execution_count":4,"metadata":{"executionInfo":{"elapsed":17146,"status":"ok","timestamp":1729622640079,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"MEvNti2YeDBn"},"outputs":[],"source":["import os\n","os.environ[\"KERAS_BACKEND\"] = \"tensorflow\"\n","\n","# Base\n","import warnings\n","import joblib\n","\n","# Plot\n","import plotly.express as px\n","\n","# Models\n","import pandas as pd\n","import numpy as np\n","import tensorflow as tf\n","import keras\n","from sklearn.model_selection import train_test_split\n","from sklearn.utils.class_weight import compute_class_weight\n","from keras_nlp.models import DistilBertTokenizer, DistilBertPreprocessor, DistilBertClassifier\n","from keras.optimizers import Adafactor\n","from keras.callbacks import EarlyStopping, CSVLogger\n","from keras.utils import to_categorical\n","from keras_tuner import RandomSearch\n","\n","# NLP Model\n","# model_pre = \"bert_base_en_uncased\"\n","model_pre = \"distil_bert_base_en_uncased\"\n","\n","# Config notebook\n","warnings.filterwarnings('ignore')"]},{"cell_type":"code","execution_count":5,"metadata":{"executionInfo":{"elapsed":17,"status":"ok","timestamp":1729622640083,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"syRDyQ5PeDBv"},"outputs":[],"source":["def pre_process(data, col1='gene', col2='variation', col3='clinical_evidence', target_col=None):\n","    \"\"\"\n","        Combine data columns and adjust column classification\n","    \"\"\"\n","\n","    data['combined_text'] = data[col1] + \" \" + data[col2] + \" \" + data[col3]\n","\n","    # Adjust class to 0 - 8 it's necessary by keras to_categorical\n","    if target_col == None:\n","        data = data[['combined_text']]\n","\n","    else:\n","        data['class_adjusted'] = data[target_col] - 1\n","        data = data[['combined_text', 'class_adjusted']]\n","\n","    return data"]},{"cell_type":"markdown","metadata":{"id":"INo2wi-1eDBx"},"source":["#### Train Data"]},{"cell_type":"code","execution_count":6,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":4599,"status":"ok","timestamp":1729622644669,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"3vNc488-eDBy","outputId":"c38fd34c-0b96-4887-bd30-37742519f504"},"outputs":[{"name":"stdout","output_type":"stream","text":["data_train: (3316, 4))\n"]}],"source":["# Load data for train\n","data_train = pd.read_parquet(path + 'data_for_train.parquet')\n","print(f\"data_train: {data_train.shape})\")"]},{"cell_type":"code","execution_count":7,"metadata":{"executionInfo":{"elapsed":333,"status":"ok","timestamp":1729622646949,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"f-WLWHUjeDB0"},"outputs":[],"source":["data_train_pre = pre_process(data_train, target_col='class')"]},{"cell_type":"code","execution_count":8,"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":112},"executionInfo":{"elapsed":3556,"status":"ok","timestamp":1729622652460,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"puir1JsveDB1","outputId":"7b4191fa-87ce-4112-97a1-972c13a94d1d"},"outputs":[{"data":{"application/vnd.google.colaboratory.intrinsic+json":{"summary":"{\n  \"name\": \"data_train_pre\",\n  \"rows\": 3316,\n  \"fields\": [\n    {\n      \"column\": \"combined_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 3316,\n        \"samples\": [\n          \"PTEN G129R the pten phosphatase and tensin homolog phosphatase is unique in mammals in terms of its tumor suppressor activity exerted by dephosphorylation of the lipid second messenger pip3 phosphatidylinositol 345trisphosphate which activates the phosphoinositide 3kinaseaktmtor mammalian target of rapamycin oncogenic pathway lossoffunction mutations in the pten gene are frequent in human cancer and in the germline of patients with pten hamartoma tumorrelated syndromes phtss in addition pten is mutated in patients with autism spectrum disorders asds although no functional information on these mutations is available here we report a comprehensive in vivo functional analysis of human pten using a heterologous yeast reconstitution system alascanning mutagenesis at the catalytic loops of pten outlined the critical role of residues within the pcatalytic loop for pip3 phosphatase activity in vivo pten mutations that mimic the pcatalytic loop of mammalian ptenlike proteins tpte tpip tensins and auxilins affected pten function variably whereas tumor or phtsassociated mutations targeting the pten ploop produced complete loss of function conversely alasubstitutions as well as tumorrelated mutations at the wpd and ticatalytic loops displayed partial activity in many cases interestingly a tumorrelated d92n mutation was partially active supporting the notion that the pten asp92 residue might not function as the catalytic general acid the analysis of a panel of asdassociated hereditary pten mutations revealed that most of them did not substantially abrogate pten activity in vivo whereas most of phtsassociated mutations did our findings reveal distinctive functional patterns among pten mutations found in tumors and in the germline of phts and asd patients which could be relevant for therapy topic mutation catalysis phosphoric monoester hydrolases yeasts neoplasms pten gene issue section articles introduction  pten phosphatase and tensin homolog deleted in chromosome 10 is a mammalian lipid and protein phosphatase with orthologs in lower animals plants and fungi 1 in mammals ptenrelated proteins exist whose activity has been poorly explored these proteins include tpip and tpte orthologs of a voltagesensor phosphatase from ciona intestinalis as well as the actinbinding tensin 123 and the clathrinbinding auxilin 12 24 pten most relevant function is based on its ability to dephosphorylate phosphatidylinositol 345trisphosphate pip3 in the 3\\u2032 phosphate of the inositol ring resulting in the bisphosphate product phosphatidylinositol 45bisphosphate pip2 thus broadly inhibiting the oncogenic phosphoinositide 3kinase pi3kdependent signaling 58 as a tumor suppressor pten exerts its function by counteracting the antiapoptotic and growth stimulatory effects of pdk1 phosphoinositidedependent kinase and protein kinase b pkbakt which rely on pip3 for their activation 910 sporadic somatic mutations in pten make this tumor suppressor gene one of the two most frequently affected in human malignancies together with p53 especially in solid tumors 1114 germline mutations of pten lead to inherited hamartoma cowden syndrome and related disorders generally known as pten hamartoma tumorrelated syndromes phtss 1517 pten is considered haploinsufficient to prevent certain malignancies suggesting that dosage is important for its function as it must be the influence of a given point mutation in the catalytic activity of the enzyme 18 besides its clear involvement in phts germline pten mutations have been described in patients with neurodevelopmental disorders such as autism spectrum disorders asds and developmental delay dd 1922 oncogenic pten mutations have been generally related to a loss of function of the encoded phosphatase however few studies have been done to understand the influence of putative neurodevelopmentrelated pten mutations on its catalytic activity  although phosphataseindependent functions of pten have also been described to contribute to its tumor suppressive activity 82325 buffering pi3k signaling in virtue of its catalytic activity is assumed to be its key role thus the precise dissection of pten catalytic properties is highly relevant for human disease the crystal structure of pten 26 revealed an active site pocket similar to those of protein tyrosine phosphatases ptps and dualspecificity thrsertyr phosphatases as predicted by primary sequence analyses although richer in basic residues this explained the ability of pten to dephosphorylate acidic phosphopeptides 27 in addition the pten active site pocket was wider than that of ptps to accommodate the rather bulky phosphorylated inositol moiety that constitutes the polar head of pip3 26 the pten phosphatase signature motif 123hckagkgr130 ploop including the catalytic cys124 and arg130 residues is located at the bottom of the active site pocket the walls of the pocket are delimited in part by a wpdloop thus designed after the equivalent region in ptp1b and a tiloop so named because of conserved thr and ile residues which are thought to be important for the activity of the enzyme the fact that the asp92 within the pten wpdloop is located in the pocket in an identical fashion to the equivalent proton donor catalytic residue on ptp1b led to the hypothesis that it may serve as the general acid that allows protonation of the leaving phosphate during the dephosphorylation reaction although in vitro studies using an asp92 to ala pten mutation indicated that asp92 does not act as the general acid 2829 no conclusive studies have been done to analyze the contribution of asp92 in the in vivo dephosphorylation of pip3 by pten  we have previously validated a humanized yeast model to study the function of pi3k and pten in vivo based on the heterologous coexpression of these proteins 13031 briefly expression of hyperactive p110\\u03b1 the catalytic subunit of mammalian class i pi3k results in high toxicity in yeast cells as a result of the depletion of essential pip2 pools coexpression of pten restores pip2 levels and suppresses pi3k toxicity besides coexpression of gfptagged mammalian akt in this system allowed the detection of pools of pi3kgenerated pip3 in the yeast cell where akt localized providing a visual readout of pi3k and pten activities 3032 here we present an extensive sitedirected mutagenesis and functional study of pten by using this system humanized yeastbased bioassays combine the advantages of powerful yeast genetics an evergrowing availability of postgenomic tools their ease and safety of manipulation and the fast achievement of results 1 our results offer insights into the catalytic properties of pten and provide a platform to compare in vivo the contribution to lipid phosphatase inactivation of mutations found in tumors and in the germline of patients with ptenrelated syndromes our analysis reveals that pten germline mutations found in patients with asddd neurodevelopmental disorders do not generally confer complete loss of function of the pten pip3 phosphatase activity when compared with mutations found in the germline of phts patients with tumor disorders  results  alascanning of the catalytic loops at the pten active site  the pip3 lipid phosphatase activity of pten is essential for its tumor suppressor activity which is evidenced by the abundance of tumorrelated mutations in exons encoding the catalytic phosphatase core of the enzyme 111533 thus the precise definition of pip3 catalytic properties of pten in vivo and its relation with pten residues mutated in tumor samples or in patients is highly relevant in oncology the pten active site is conformed by three catalytic loops 26 the wpdloop residues 8898 the ploop residues 121131 and the tiloop residues 160171 fig 1a the pcatalytic loop containing the essential catalytic cysteine cys124 in human pten is highly conserved between species from mammals to lower eukaryotes whereas the wpd and tiloops are more divergent fig 1b to test the contribution of all individual residues within the three pten catalytic loops to the in vivo activity of the enzyme a comprehensive alascanning mutagenesis was performed the pip3 catalytic activity of the resulting mutations was assessed in vivo using a reconstitution heterologous system in the yeast saccharomyces cerevisiae 31 as illustrated in figure 2 ala substitutions of the residues at the core ploop residues 121131 substantially abrogated the pten ability to dephosphorylate pip3in vivo with the exception of the a121v g129a and t131a mutations which displayed partial or total activity note that ala residues have been substituted by val residues in our alascanning mutational analysis remarkably a cowden disease mutation targeting the gly129 residue g129e is defective for pip3 phosphatase activity but maintains in vitro activity against phosphotyrosine peptides 34 overall the residues at the wpd and tiloops were less important for pip3 catalytic activity thus ala substitutions of most wpd residues 8898 did not affect or only partially affected pten catalytic activity with the exception of asp92 and his93 that fully abrogated the pten function fig 2 also ala substitutions of most ti residues 160171 only weakly affected the pten activity except for val166 gln171 and to some extent asp162 fig 2 the lossoffunction properties of the d92a mutation is in accordance with the importance of this residue in pten phosphatase catalysis although its role as the general acid in the hydrolysis reaction remains unclear 2829  figure 1 pten catalytic loops a detail of the threedimensional structure of the pten catalytic site regions spanning the wpd p and ticatalytic loops are marked in yellow red and blue respectively image was generated with pymol httppymolorg based on pdb id 1d5r 26 b amino acid sequence alignment of pten catalytic loops from homo sapiens danio rerio strongylocentrotus purpuratus drosophila melanogaster caenorhabditis elegans dictyostelium discoideum schizosaccharomyces pombe and s cerevisiae c amino acid sequence alignment of pten catalytic loops with the equivalent regions from ptenlike proteins h sapiens sequences amino acid numbering from h sapiens pten is in brackets amino acid identity with regard to h sapiens pten is marked in yellow alignments were made manually view largedownload slide pten catalytic loops a detail of the threedimensional structure of the pten catalytic site regions spanning the wpd p and ticatalytic loops are marked in yellow red and blue respectively image was generated with pymol httppymolorg based on pdb id 1d5r 26 b amino acid sequence alignment of pten catalytic loops from homo sapiens danio rerio strongylocentrotus purpuratus drosophila melanogaster caenorhabditis elegans dictyostelium discoideum schizosaccharomyces pombe and s cerevisiae c amino acid sequence alignment of pten catalytic loops with the equivalent regions from ptenlike proteins h sapiens sequences amino acid numbering from h sapiens pten is in brackets amino acid identity with regard to h sapiens pten is marked in yellow alignments were made manually figure 2 functional assessment in the yeast in vivo model of alascanning mutations covering the pten catalytic site yph499 s cerevisiae transformants expressing p110\\u03b1caax gfpakt1 and pten mutations ala substitutions of the indicated amino acids residues 8898 at the wpdloop residues 121131 at the ploop catalytic cys in bold residues 160171 at the tloop val substitutions in the case of residues ala121 and ala126 were incubated for 5 h in the presence of galactose all heterologous proteins are coexpressed from the gal1 galactoseinducible promoter and gfp staining was monitored by fluorescence microscopy as a reference controls expressing wildtype wt pten or the empty vector \\u03c6 are shown at the top left blackcolor bars indicates the percentage of cells with plasma membrane gfp stain high pip3 levels  low pten activity whereas white color corresponds to the percentage of cells that display a nucleocytoplasmic gfp signal low pip3 levels  high pten activity over 100 cells were analyzed microscopically per experiment data are the average from three different experiments error bars correspond to the standard deviation + or \\u2212 signs indicate the ability of each pten mutation to rescue growth arrest induced by p110\\u03b1caax as determined by serial dilution drop growth assays on solid medium see inlet at the top as a reference the drop growth experiments were performed on cotransformants expressing only p110\\u03b1caax and the indicated pten mutation asterisks * mark mutations to ala that have been reported in tumorsgermline phts view largedownload slide functional assessment in the yeast in vivo model of alascanning mutations covering the pten catalytic site yph499 s cerevisiae transformants expressing p110\\u03b1caax gfpakt1 and pten mutations ala substitutions of the indicated amino acids residues 8898 at the wpdloop residues 121131 at the ploop catalytic cys in bold residues 160171 at the tloop val substitutions in the case of residues ala121 and ala126 were incubated for 5 h in the presence of galactose all heterologous proteins are coexpressed from the gal1 galactoseinducible promoter and gfp staining was monitored by fluorescence microscopy as a reference controls expressing wildtype wt pten or the empty vector \\u03c6 are shown at the top left blackcolor bars indicates the percentage of cells with plasma membrane gfp stain high pip3 levels  low pten activity whereas white color corresponds to the percentage of cells that display a nucleocytoplasmic gfp signal low pip3 levels  high pten activity over 100 cells were analyzed microscopically per experiment data are the average from three different experiments error bars correspond to the standard deviation + or \\u2212 signs indicate the ability of each pten mutation to rescue growth arrest induced by p110\\u03b1caax as determined by serial dilution drop growth assays on solid medium see inlet at the top as a reference the drop growth experiments were performed on cotransformants expressing only p110\\u03b1caax and the indicated pten mutation asterisks * mark mutations to ala that have been reported in tumorsgermline phts functional analysis of mutations at the pten ploop mimicking ptenlike proteins  ptenlike proteins include tpip and tpte as well as tensins and auxilins the wpd and tiloops are divergent between pten and ptenlike proteins whereas the ploop is more conserved fig 1c as part of our mutational analysis of the pten active site we designed single amino acid substitution mutants on pten that mimicked the ploop from the different ptenlike proteins and tested their pip3 phosphatase activity in the yeast system mutations included a121v i122l i122v h123y c124n k125l k125m k125r a126d a126g g127n k128r k128t g129d g129a r130a r130k t131i t131l and t131s note that a121v g129a and r130a mutations are included in fig 2 a group of mutations h123y c124n a126d g127n k128t g129d r130a r130k t131i and t131l fully abrogated the pip3 phosphatase activity of pten in yeast whereas other two groups only affected partially i122l k125l k125m a126g and k128r or did not affect pten pip3 phosphatase activity a121v i122v k125r g129a and t131s fig 3 note that results for a121v g129a and r130a mutations are included in fig 2 the results from this mutational analysis are in agreement with the in vitro pip3 phosphatase activity reported for tpip and with the lack of activity reported for tpte 35 our results also suggest that the ploops from tensin and auxilin proteins do not bear catalytic activity moreover this analysis revealed that some conservative mutations at the pten ploop such as i122v k125r and t131s are permissive for its activity in vivo  figure 3 functional assessment of pten mutations mimicking the ploop sequence from ptenlike proteins experiments were performed and are presented as in figure 2 the alignment marks divergences within the ploop sequence between pten and ptenrelated proteins which correspond to the substitutions made in the ptenmutated versions the numbering of residues for pten is shown  indicates the same residue at that position than pten the data for substitutions marked in italics in this alignment are already presented in figure 2 the asterisk marks a mutation reported in tumorsgermline phts view largedownload slide functional assessment of pten mutations mimicking the ploop sequence from ptenlike proteins experiments were performed and are presented as in figure 2 the alignment marks divergences within the ploop sequence between pten and ptenrelated proteins which correspond to the substitutions made in the ptenmutated versions the numbering of residues for pten is shown  indicates the same residue at that position than pten the data for substitutions marked in italics in this alignment are already presented in figure 2 the asterisk marks a mutation reported in tumorsgermline phts functional analysis of tumorrelated mutations targeting pten active site residues  tumorrelated mutations are spread along the entire pten amino acid sequence nevertheless the pten cterminal tail last 50 residues is targeted by mutations less frequently in agreement with our previous finding using the yeast heterologous system that pten 1349 is the minimal fully functional pten protein 30 on the other hand the core ptp catalytic domain residues 14185 is more frequently targeted by mutations in either tumors or phts patients some of these mutations have been analyzed in terms of intrinsic catalytic activity 3640 but the majority of them have not been directly tested for function we have analyzed in the yeast system the pip3 phosphatase activity of tumorrelated mutations targeting pten residues at the pten catalytic loops note that some of these mutations have been analyzed previously and are shown in figs 2 and 3 and figs 5 and 6 also include some of them as expected all the tumor mutations at the ploop tested a121e i122s h123d h123y c124s k125e a126s a126v g127e k128n g129e g129v g129r r130g t131a and t131i gave rise to proteins that were fully inactive except for a126s and k128n that produced partially inactive proteins fig 4 and figs 2 and 3 mutations at the tiloop included t160i r161g d162g g165e g165v v166i t167a i168f p169h and s170r among these only the t160i g165e g165v and s170r were lossoffunction mutations fig 4 and fig 2 finally tumorrelated mutations targeting the wpd loop included y88c f90l e91a d92a d92e d92g d92h d92n d92v h93d h93q h93r h93y n94i p95l and p96q proteins bearing y88c f90l or e91a mutations were active whereas the rest of mutant proteins were partially or fully inactive fig 4 and figs 2 5 and 6 a summary of the tumorrelated pten mutants analyzed is shown in supplementary material table s1 taken together our findings illustrate the heterogeneity in the catalytic activity of tumorrelated pten mutant proteins  figure 4 functional assessment of tumorrelated pten mutations targeting pten catalytic loops experiments were performed as those in figure 2 view largedownload slide functional assessment of tumorrelated pten mutations targeting pten catalytic loops experiments were performed as those in figure 2 figure 5 functional assessment of pten mutations targeting the asp92 residue a tolerance to mutation of the asp92 residue mutations were assessed in yeast as those in figure 2 asterisks mark mutations that have been reported in tumorsgermline phts the image at the right corresponds to growth rescue assays from representative pten mutations and controls from the data presented b in vitro phosphatase activity of recombinant pten proteins purified from cos7 cell line transfectants as determined by the malachite green assay using dic8phosphatydilinositol pip3 as the substrate data are the average from three independent experiments error bars correspond to the standard deviation c ability of pten mutations to abolish haakt activation in mammalian cells pten wildtype or the indicated mutations were coexpressed with haakt1 in cos7 cells cellfree lysates were subjected to immunoblot using antiphosphoactiveakt antibodies pthr308 and pser473 the content of pten and haakt1 in the lysates was monitored using antipten all pten mutations were expressed equally not shown and antiha antibodies respectively bands were quantified and results are presented as the average \\u00b1 sd of the paktakt ratio obtained from three independent experiments view largedownload slide functional assessment of pten mutations targeting the asp92 residue a tolerance to mutation of the asp92 residue mutations were assessed in yeast as those in figure 2 asterisks mark mutations that have been reported in tumorsgermline phts the image at the right corresponds to growth rescue assays from representative pten mutations and controls from the data presented b in vitro phosphatase activity of recombinant pten proteins purified from cos7 cell line transfectants as determined by the malachite green assay using dic8phosphatydilinositol pip3 as the substrate data are the average from three independent experiments error bars correspond to the standard deviation c ability of pten mutations to abolish haakt activation in mammalian cells pten wildtype or the indicated mutations were coexpressed with haakt1 in cos7 cells cellfree lysates were subjected to immunoblot using antiphosphoactiveakt antibodies pthr308 and pser473 the content of pten and haakt1 in the lysates was monitored using antipten all pten mutations were expressed equally not shown and antiha antibodies respectively bands were quantified and results are presented as the average \\u00b1 sd of the paktakt ratio obtained from three independent experiments figure 6 functional assessment of pten mutations related to phts a or asddd b hereditary syndromes experiments were performed as described in figure 2 asterisks in b mark mutations that have been reported in germline phts c global functional analyses of the impact on catalytic activity of phts versus asdddrelated mutations the yaxis refers to relative in vivo pten activity as measured by the percentage of yeast cells with low pip3 levels at the plasma membrane as determined by gfpakt1 localization as in figs 24 and 6a each spot corresponds to the average data from three independent experiments on each particular mutation from the indicated category phts or asddd spots surrounded by a circle correspond to mutations that are common to phts or asddd those marked with asterisks in b vertical lines mark the standard deviation paired ttest of these independent set of data revealed a significant difference in activity for asddd mutations with respect to phts mutations p < 0001 d comparative analysis of phts and asddd pten mutations referred to their ability to rescue growth arrest induced in s cerevisiae by p110\\u03b1caax expression asdddrelated mutations are generally more competent than phtsrelated mutations in counteracting p110\\u03b1caax activity phts mutations included in this study are listed in supplementary material table s1 and results correspond to those in a and in figures 4 and 5 asddd mutations data correspond to those in b view large download slide functional assessment of pten mutations related to phts a or asddd b hereditary syndromes experiments were performed as described in figure 2 asterisks in b mark mutations that have been reported in germline phts c global functional analyses of the impact on catalytic activity of phts versus asdddrelated mutations the yaxis refers to relative in vivo pten activity as measured by the percentage of yeast cells with low pip3 levels at the plasma membrane as determined by gfpakt1 localization as in figs 24 and 6a each spot corresponds to the average data from three independent experiments on each particular mutation from the indicated category phts or asddd spots surrounded by a circle correspond to mutations that are common to phts or asddd those marked with asterisks in b vertical lines mark the standard deviation paired ttest of these independent set of data revealed a significant difference in activity for asddd mutations with respect to phts mutations p < 0001 d comparative analysis of phts and asddd pten mutations referred to their ability to rescue growth arrest induced in s cerevisiae by p110\\u03b1caax expression asdddrelated mutations are generally more competent than phtsrelated mutations in counteracting p110\\u03b1caax activity phts mutations included in this study are listed in supplementary material table s1 and results correspond to those in a and in figures 4 and 5 asddd mutations data correspond to those in b relevance of the asp92 residue in pten activity in vivo  remarkably a mutation in the putative catalytic asp92 residue to asn d92n rendered a protein partially active in vivo in the yeast system whereas substitution of this residue to any of the other 18 amino acids generated fully inactive pten enzymes fig 5a this outlines the importance of asp92 in pten enzymatic activity but also demonstrates that the presence of an aspartic acid residue at this position is not a strict requirement for ptendriven catalysis to further substantiate this observation the pip3 phosphatase activity of the pten mutations d92a nonconservative d92e chargeconservative and d92n sizeconservative expressed in mammalian cells was tested both in vitro and in vivo as shown the mutations d92a and d92e fully abrogated hydrolysis of pip3 by pten in vitro whereas the mutation d92n only partially affected it fig 5b upon coexpression with akt the d92n pten mutant also manifested more activity indirectly measured as the levels of phosphoakt pakt content than the d92a or the d92e mutant proteins fig 5c our results suggest that asp92 does not behave as the general acid in the hydrolysis of pip3 by pten in vitro and in vivo  most of germline pten mutations associated with autism syndrome disorders do not fully abrogate pip3 phosphatase activity  the involvement of pten in brain developmental disorders has been proposed on the basis of the identification of hereditary pten mutations in patients with these disorders 4142 but little information is available on the severity of such mutations in terms of pten function one of the mutations present in glioblastoma tumor samples h93r has also been found in the germline of a patient with autism 19 when performing the experiments shown above on tumorrelated pten mutations we included the h93r and found it to display a modest loss of activity in the yeast model when compared with wildtype pten although it was capable of fully recovering yeast growth fig 6b this prompted us to examine additional inherited mutations found in patients with autismrelated disorders asd or with ddmental retardation these mutations included besides h93r the g44d h118p h123q m134t e157g r173h y176c t202i f241s d252g w274l n276s and d326n involving not only catalytic but other protein domains remarkably proteins bearing most of these mutations were either fully or partially active in the yeast fig 6b and supplementary material table s1 although their ability to recover growth from pi3kimposed inhibition was variable these results were in contrast to those obtained by analyzing the function of a panel of phtsassociated mutations also distributed along the entire pten coding region mutations a34d m35r g36r n48k h61d y68d d92e h93y p96q h123d c124s g129e r130g y155c g165e g165v s170r l181p v217n p246l r335l v343e l345v and f347l fig 6a and supplementary material table s1 most of the phtsassociated mutations generated total or partial loss of function when analyzed in vivo in yeast moreover global comparison of phts and asdddrelated mutations in terms of their ability both to deplete pip3 generated by pi3k in yeast fig 6c p < 0001 and to rescue yeast growth fig 6d revealed that asdddrelated pten mutations have a lower effect in pten activity than those associated with phts these findings indicate that specific pten functional patterns correlate with the mutations found on either phts or asddd patients and suggest that a severe loss of the pten pip3 catalytic activity may not be the major driver of autism or autismrelated neurological disorders  discussion  the pten phosphatase is a key regulatory hub of pi3kmediated signaling while mammals express a variety of isoforms of pi3k and pip3dependent downstream kinases in this pathway the pten tumor suppressor is rather unique being the only one gene that encodes for a common inactivator of the pathway thus mutations in exons of the pten gene are frequent in tumors and hereditary cancer syndromes the regulation of pten is complex involving multiple processes related to its phosphorylation ubiquitination nuclear translocation interaction with membranes and complexing with different proteins 68244344 although recent reports point towards pten functions independent of its lipid phosphatase activity 23254546 pten tumor suppressor function mainly relies on its catalytic activity on pip3 pools locally generated by pi3k at the plasma membrane the contribution of pten mutations isolated in the clinics to the function of the encoded protein is crucial to understand the molecular mechanisms that lead to tumor development and progression we have developed a simple model for functional assessment of pten mutations based on heterologous expression in yeast 131 this model allows ready evaluation of the involvement of pten mutations on the ability to counteract pi3k activity in vivo by compromising pten pip3 catalytic activity or by affecting pten protein stability two common mechanisms by which pten is inactivated in tumors 30 here we report extensive use of this model with two aims i to provide clues for structurefunction relationship by systematic mutagenesis and ii to study the influence of individual diseaserelated pten mutations in its lipid phosphatase activity  from our systematic functional analysis on the pten catalytic loops several conclusions can be driven first as expected the ploop including the catalytic cys residue is very sensitive to mutagenesis in consonance with the observation that it constitutes a hot spot for tumorrelated mutations 111533 whereas the wpd and ti loops are more permissive to point ala substitutions still residues like asp92 or his93 wpdloop asp162 val166 or gln171 tiloop were critical for function we made a comprehensive study on tolerance to mutation of the asp92 residue equivalent to the acid proton donor involved in catalysis in ptps that are structurally related to pten 47 tolerance to mutation was very low in this residue since its substitution to any other amino acid yielded an inactive enzyme except for asptoasn substitution which partially retained activity in vivo not only in the yeast model but also in mammalian cells since the d92n mutation has been reported in the clinic such reduced activity may substantially contribute to disease interestingly this argues against the need for an acid residue in that position revealing an alternative mechanism of catalysis for the pten phosphatase as proposed by other authors based on in vitro experiments 2829 we also made substitutions within the catalytic loops mimicking ptenrelated proteins such as tpip tpte tensin and auxilin although these proteins possess distinct nonptprelated domains that control protein targeting proteinprotein interactions and cell signaling they share the presence of ptp and c2like domains with significant similarity with those from pten 2348 with the exception of tpip ptenlike proteins are predicted to be catalytically inactive due either to the lack of essential residues at the catalytic site or to amino acid substitutions potentially deleterious for catalysis 354950 nevertheless tensin2c1ten has been proposed to be an active phosphatase based on functional studies on a mutation that targets its putative catalytic cysteine residue 51 our observations favor the hypothesis that the ploop of tpip might be competent for activity whereas tpte tensins and auxilins would be inactive as phosphatases as evidenced in other studies 355253  a large number of different pten mutations have been described in human tumors and in the germline of patients with phts or phtsrelated syndromes 111533 for updates check hgmd httpwwwhgmdcfacukacindexphp and cosmic httpwwwsangeracukgeneticscgpcosmic databases these mutations target the pten gene not only at its coding regions but also at exonintron boundaries and promoter regions most of pten mutations at exonintron boundaries or promoter regions have a major impact in the levels of expression of the pten protein being causative of pten functional deficiency and considered as pathogenic 5456 mutations targeting the pten coding region include frameshift and nonsense mutations which also generate unstable truncated pten proteins in most of the cases 36 as well as missense mutations that result in individual amino acid changes functional analyses of a limited number of these missense mutations have revealed that in many cases the resultant amino acid substitutions generate pten proteins with impaired intrinsic catalytic activity andor protein stability 3639 also some examples have been described of missense pten mutations that do not affect the intrinsic catalysis nor the stability of the protein but rather impair essential regulatory pten properties such as binding to membranes or nuclear entry 4057 our previous study using the heterologous yeast system as a reporter of the activity of pten mutations in vivo revealed that such system reproduces with fidelity the effects on mammalian cells of mutations that diminish either the intrinsic pten catalytic activity andor binding to membranes or the intrinsic pten protein stability 30 our current comprehensive functional study of tumorassociated pten mutations indicates that almost all mutations targeting the pcatalytic loop confer loss of pten intrinsic catalytic activity 12 of 13 whereas mutations targeting the wpd or the tiloops are less frequently associated with full loss of catalysis 5 of 9 and 4 of 9 respectively these results are in agreement with the alascanning analysis of these regions and suggest the existence of a relatively large number of tumorassociated mutations at the pten gene that do not fully abrogate the pten catalytic activity but may be relevant in clinics recently a comprehensive study on pten mutations in phts disease has yielded predictive clinical parameters 58 association of reported mutations to the activity of pten by means of ready in vivo functional assays as the one exploited here might help clinicians in decisions towards putative mutationspecific therapies  phts and asddd are syndromes with distinct clinical entities which share macrocephaly as a frequent common alteration 59 also clinicians have noted that phts patients often display neurobehavioral features resembling autism 19 and interestingly at least five pten mutations h93r y176c f241s d252g and r173h see supplementary material table s1 have been reported as associated with both phts and asddd 19414258 our comparative analysis of germline mutations found in phts patients and in asddd patients revealed a distinct functional pattern for these two groups of mutations phtsassociated mutations which included in our study mutations distributed all along the pten protein were found significantly more deleterious for pten catalysis than asdddassociated mutations fig 6 this could be important for the distinct etiology and progress of tumorrelated and tumor nonrelated syndromes associated with pten mutations and may be of relevance in the design and implementation of new therapies for these diseases it is likely that small variations in the lipid phosphatase activity or in the stability of heterozygotic pten mutations can explain the higher penetrance found for the set of phtsassociated mutations when compared with asdddassociated mutations although it has been suggested that asd and macrocephalyrelated pten mutations may behave differentially in terms of loss of function 19 the lack of functional models has made difficult to provide support for this hypothesis if this is the case as suggested by our study additional unknown factors might exist affecting differentially the etiology andor progression of these diseases in this regard a study focused on the functional analysis of the phts and asdassociated h93r pten mutation reported a partial loss of catalytic activity for this mutation as well as a defect in the binding to ptenactivating pip2 model membranes remarkably the binding of the h93r mutation to phosphatidylserine membranes which are abundant in mammalian cells was augmented 60 our results show that the h93r mutation retained most of its lipid phosphatase activity in the yeast suggesting that differences in the thresholds that control membranebinding mediated pten activation may exist in distinct cell systems also of interest mice with diminution of pten expression levels as subtle as 20 develop a wide spectrum of tumors with high penetrance and it has been proposed that altered gene dosage may be relevant in asddd 1861 in this regard asddd has been linked with abnormal synapsis growth and plasticity an alteration that has also been observed in mice with ptendeficient neurons 6263 furthermore a mice model lacking pten expression in mature neuronal populations in the cerebral cortex and the hippocampus displayed behavioral abnormalities reminiscent of human asddd 64 our findings show a relatively weak association between asdddrelated mutations at the pten gene and full loss of intrinsic lipid phosphatase activity or protein stability which is in line with the lack of concordance between asddd and childhood cancer 65 asddd constitutes a heterogeneous multigenelinked group of developmental disorders 414266 further studies will be required to ascertain the functional changes on the activity of the pten protein from asddd patients with mutations at the pten gene the comprehensive analysis of the functional properties of putative pathogenic mutant proteins as the one we present here for pten mutations will give some light on the molecular pathways altered in the neurons of patients with asddd  materials and methods  cells media growth conditions transfection and protein detection  the s cerevisiae strain yph499 mata ade2101 trp163 leu21 ura352 his3\\u25b5200 lys2801 was used for heterologous expression of mammalian proteins for all the analyses synthetic complete medium was used containing 017  yeast nitrogen base without amino acids 05 ammonium sulfate supplemented with appropriate amino acids and nucleic acid bases and added 2 glucose sd galactose sg or raffinose sr as required yeasts were transformed by standard procedures growth of yeast on plates was tested by spotting transformant cells onto sd or sg plates lacking the corresponding auxotrophic markers transformants were grown overnight in sd lacking uracile leucine or both sdu sdl or sdul as required and adjusted to an od600 of 05 five microliter aliquots of each sample plus three serial 110 dilutions were deposited on the surfaces of solid media sgu sgl or sgul growth was monitored after 23 days at 30\\u00b0c expression of all pten mutations in yeast was verified by immunoblot on yeast lysates with 421b or 425a antipten mab 67 followed by horseradishperoxidaseconjugated antimouse antibody calbiochem to measure gfpakt1 plasma membrane localization as an indirect indicator of cellular pip3 levels transformant cells were grown to log phase in liquid sr medium lacking the corresponding auxotrophic markers and then 2  galactose was added for 68 h gfpakt1 was visualized by fluorescence microscopy to obtain statistically significant data \\u2265100 cells were examined from each condition or experiment for either cytoplasmic or membraneassociated localization cells were examined under an eclipse te2000u microscope nikon and digital images were acquired with orca c47429512er chargecoupled device camera hamamatsu and aquacosmos imaging systems or hcimage software to overexpress pten mutations in mammalian cells cos7 cells simian kidney were transfected by the deaedextran method and processed after 48 h to measure pakt content in cos7 cells in the presence of pten mutations cells were cotransfected with prk5 pten and psg5 haakt1 as described 68 cells were lysed and postnuclear lysates were subjected by immunoblot using a mixture of antipthr308 and antipser473akt antibodies cell signalling technologies membranes were stripped and probed for antiakt antibody cell signalling technologies bands were quantified using imagequant tl software and results are represented as the average \\u00b1 sd of the paktakt ratio  plasmids mutagenesis and in vitro phosphatase assays  ycplg mycp110caax pyes2 pten prk5 pten psg5 haakt1 and pyes3 gfpakt1 have been described previously 30 mutagenesis was performed by a twostep pcr strategy all mutations were checked by dna sequencing the sequences of oligonucleotides used for mutagenesis are available on request in vitro phosphatase assays were made using pten wildtype or mutations from cos7transfected cells as the enzymes and dic8phosphatydilinositol pip3 echelon as the substrate as described previously 30 germline mutations in the phosphatase pten are associated with diverse human pathologies including tumour susceptibility developmental abnormalities and autism but any genotypephenotype relationships are poorly understood  methods  we have studied the functional consequences of seven pten mutations identified in patients diagnosed with autism and macrocephaly and five mutations from severe tumour bearing sufferers of pten hamartoma tumour syndrome phts  results  all seven autismassociated pten mutants investigated retained the ability to suppress cellular akt signalling although five were highly unstable observed effects on akt also correlated with the ability to suppress soma size and the length and density of dendritic spines in primary neurons conversely all five pten mutations from severe cases of phts appeared to directly and strongly disrupt the ability to inhibit akt signalling  conclusions  our work implies that alleles causing incomplete loss of pten function are more commonly linked to autism than to severe phts cases  keywords cancer breast neurosciences cell biology go to introduction phosphatase and tensin homologue deleted on chromosome ten pten is a phosphatase that suppresses the activity of the class i phosphoinositide 3kinaseakt signalling pathway1 it has been heavily studied due to its status as a tumour suppressor gene in which loss of function mutations are identified in many sporadic tumours and in the germline of patients with diverse phenotypes inherited dominant pten mutations have been identified in patients with cowden syndrome and bannayanrileyruvalcaba syndrome conditions that are often grouped together as pten hamartoma tumour syndrome phts they have also been identified in a fraction of patients with autism spectrum disorder asd who also display macrocephaly with and without additional developmental phenotypes characteristic of phts28 supporting the causality of pten mutations in these phenotypes a similar tumour spectrum and autismlike phenotypes have been identified in mice either carrying a single null pten allele or with tissuespecific deletion of pten911  the major diagnostic criteria for phts include malignancies of the breast thyroid and endometrium in addition to benign hamartomas skin lesions and macrocephaly12 however the symptoms associated with pten mutations are diverse and in some cases germline mutations have been identified in adult patients only upon presentation with malignancy13 14 and in patients with macrocephaly autism andor learning disability without further symptoms5 15 a series of clinical and laboratory based studies have presented evidence that mutational functional diversity and genetic background may each contribute to the phenotypical diversity observed in patients carrying pten mutations in some families individuals carrying the same pten mutation have been noted to display very different phenotypes to the extent of separate diagnoses of cowden syndrome and bannayanrileyruvalcaba syndrome16 17 additionally the spectrum of tumours arising in heterozygous mice carrying a null allele of pten also appears to be strongly dependent on genetic background18 studies of the mutation types missense vs truncation and positions phosphatase vs c2 domain within pten have argued for19 and against2 20 genotypephenotype relationships on the other hand evidence that not all pten mutations cause one phenotypical profile has been provided by studies of heterozygous mice expressing a stable pten mutant protein either lacking all phosphatase activity pten c124s or that selectively lacks lipid phosphatase activity pten g129e these mice display a more severe tumour phenotype than mice carrying a pten deletion allele21 22 indicating that inactive pten can aggravate phenotypes particularly tumour severity through dominant negative mechanisms however whether this represents a consistent genotypephenotype relationship within pten mutation carriers in particular relating to the occurrence of asd is unclear  go to results pten missense mutations identified in patients with autism are catalytically competent  recombinant pten protein purified from bacteria has been used in many previous studies to demonstrate that phosphatase activity is critical for its tumour suppressor function in preliminary experiments studying three pten mutants identified in patients with autism we could detect little or no catalytic activity in vitro from protein purified from bacteria yet transient expression of these mutants in mammalian cells led to a robust suppression of the phosphorylation of the pten regulated kinase akt see online supplementary figure s1a b this apparent difference between assay formats encouraged us to analyse a larger group of seven mutant proteins identified in patients with autism figure 1a lacking other manifestations of phts37 therefore we used lentiviruses to express either pten wildtype wt or individual missense pten mutants c124s g129r h118p h123q e157g f241s d252g n276s and d326n in ptennull u87mg glioblastoma cells these include two recognised active site pten mutants that lack catalytic activity pten c124s and pten g129r we expressed pten wt and each mutant using five increasing doses of each viral vector figure 1c investigating effects on cellular akt phosphorylation when expressed at similar levels all seven of the autismrelated mutants showed similar or in some cases possibly greater effects to pten wt on akt phosphorylation whereas a lack of activity was observed with the inactive mutants pten c124s and g129r figure 1c d and see online supplementary figure s3a these data show that the mutations identified in patients with autism do not greatly reduce the ability of the expressed pten proteins to inhibit cellular akt signalling and imply that these mutants retain catalytic activity  figure 1 figure 1 pten lipid phosphatase activity a a diagram representing pten protein structure and the localisation of the autismrelated mutations analysed b pten crystal structure indicating mutations the catalytic cys124 is yellow c u87mg cells were transduced  to test their phosphatase activity directly we overexpressed each protein in u87mg cells and assayed freshly immunopurified pten proteins in vitro against radiolabelled phosphatidylinositol 345trisphosphate pip3 vesicles recovered activities were weak for many of the pten mutants but this correlated with a low recovery of immunoprecipitated pten probably due to poor stability of these mutants during the immunopurification process see online supplementary figure s2 in particular pten h118p f241s d252g and n276s were recovered poorly and gave very weak activity activity normalised to reflect pten protein content showed robust activity for all mutants although weaker than wild type enzyme see online supplementary figure s2 these experiments also indicate that testing the protein phosphatase activity would be problematic in addition to this difficulty immunoprecipitating active protein during these experiments it was noted that to express most of the autismrelated pten missense mutations at the same level as pten wt protein a higher titre of lentivirus was consistently required see online supplementary figures s2a and s3a therefore we investigated the possibility that these mutations affected the stability of the pten protein  reduced protein stability of autismassociated pten mutants  to investigate the relationship between the cellular abundance of pten mrna and protein for the mutants we expressed pten wt and mutants in u87mg cells and measured pten mrna levels by realtime qpcr and protein by western blotting in all cases the mrna abundance corresponded to the titre of lentivirus applied in the experiment the expression of pten wt and the widely used stable mutants c124s and g129r was robust and similar as was the expression of two of the autismassociated mutants h123q and e157g however we observed that for five of the autismassociated mutants h118p f241s d252g n276s and d326n the amount of protein expressed was very low figure 1d e  to assess more directly the stability of these mutant proteins relative to pten wt we used the protein synthesis inhibitor cycloheximide 200 \\u00b5gml transduced cells were incubated with the inhibitor for 0 h 2 h 4 h 6 h and 8 h followed by immunodetection using antipten antibody figure 2a b whereas the protein level of pten wt and some mutants c124s g129r h123q and e157g was reduced by \\u223c20 after 8 h of inhibition of protein synthesis the protein level for the pten mutants h118p f241s d252g n276s and d326n was decreased by \\u223c60 after 8 h figure 2a b and was significantly lower than pten wt supporting the hypothesis that these five mutants are unstable  figure 2 figure 2 autismassociated pten mutations reduce protein stability the stability of pten mutation proteins was determined using the cycloheximide inhibitor a u87mg cells expressing pten wt or pten point mutants were treated with 200 \\u00b5gml of  molecular characterisation in primary neurons of the pten mutations identified in patients with autism  macrocephaly is one of the several anatomical and cellular abnormalities that have been suggested to be important factors in the pathogenesis of asd23 work carried out in mice showed that pten can control neuronal morphology and growth with hypertrophy and dendritic overgrowth and development of macrocephaly and additional behavioural phenotypes similar to human autism observed in mice lacking pten in neurons10 2427 we have therefore used neurons from which endogenous pten is deleted to investigate whether the physiological level reexpression of pten wt or autismassociated pten mutants is able to support a wild type phenotype of normal dendritic growth and neuronal size hippocampal neurons prepared from e155 floxflox pten mice were transduced simultaneously with lentiviruses expressing cre recombinase rfpcre to delete endogenous pten and with viruses encoding pten wt or pten mutants the mutants pten h123q f241s and d326n were selected to represent a range of protein stabilities see figure 2 the deletion of the endogenous pten genes and the reexpression of human pten mutants were verified by western blot figure 3d deletion of pten led to an increase in akt phosphorylation which was not reduced by the expression of pten c124s the mutant f241s expressed very poorly however the mutants h123q and d326n were expressed and able to downregulate akt phosphorylation as was pten wt these results confirmed that when expressed at physiological levels these mutants are biologically active we wanted then to verify the effect that they have on soma size and the morphology of dendritic spines by immunocytochemistry in order to analyse neuronal morphology 6 days after transduction and 24 h before being fixed and stained neuronal cultures were lipofected with vectors encoding yellow fluorescent protein to allow the morphological analysis of a transfected neuronal subpopulation including identifiable transduced and untransduced cells pten wt significantly reduced the soma size figure 3b c of hippocampal neurons as well as the density and length of dendritic spines figure 3eg to levels similar to control neurons from which endogenous pten had not been deleted relative to neurons expressing cre with or without additional inactive pten c124s in agreement with the previous results we found that the autismrelated mutants h123q and d326n were capable of controlling these morphological changes in broad correlation with the amount of protein expressed showing effect similar to pten wt when expressing similar amount of protein pten f241s expressed very poorly and did not affect neuronal signalling or morphology in accordance with its very low apparent stability  figure 3 figure 3 characterisation of autismrelated pten mutants in primary neurons reexpression at physiological level of pten wt and mutants in pten knockout neurons is able to support wild type neuronal morphology a workflow of the experiment neurons were transduced  pten mutations identified in severe cases of phts lack activity  considering the view that patients diagnosed only with asd and macrocephaly may reflect a group at the milder end of the spectrum of phenotypes associated with pten mutations we chose to select a comparator group of mutations associated with a severe tumour bearing phts phenotype to do this we conducted an extensive search of data published in journals and online resources associating specific pten mutations with phenotype see online supplementary tables s1s3 although inconsistencies in the categorisation of patients severity are likely to be introduced by diversity in the diagnosis and recording of separate phenotypical characteristics between publications our analysis identified 112 missense mutations that had sufficient associated phenotypical data and which had been identified in a total of 204 patients see online supplementary tables s2 s3 this analysis shows that missense mutations rather than truncating mutations were overrepresented among all pten mutation carriers described with asd compared with all carriers of pten mutations consistent with the hypothesis that more of these mutants may retain some activity figure s3  five pten missense mutations associated with the most severe phenotype score >6 were selected for characterisation pten a39p n48k l108p l112p and r130l figure 4a these were expressed in u87mg cells together with the already described stable and inactive pten mutants c124s g129r and g129e the latter two have been identified in cowden syndrome sufferers and in our classification were associated with a mildmoderate phenotype when transduced into u87mg cells using lentiviruses two of these mutant cdnas a39p and n48k were expressed with similar efficiency to wt pten but did not suppress cellular akt phosphorylation figure 4c the other three mutants l108p l112p and r130l expressed less efficiency compared with pten wt but when expression was forced to this level they also failed to downregulate akt phosphorylation figure 4c and see online supplementary figure s3b analysis of the halflife of pten mutant proteins showed that two of the mutations l108p and l112p destabilise the protein with a decrease in protein levels by \\u223c6080 after 8 h of protein synthesis inhibition whereas the protein level for the mutants a39p n48k and r130l as well as for pten wt c124s g129e and g129r was reduced by \\u223c20 figure 4d e this finding that when expressed at the same level as wild type pten all the seven pten mutations associated with asd but none of the five associated with severe cowden syndrome analysed was able to regulate akt is significant twotailed fishers exact test p00013  figure 4 figure 4 missense pten mutations associated with severe pten hamartoma tumour syndrome phts phenotypes cause complete loss of function a studied mutations in pten associated with phts b pten crystal structure with catalytic cys124 in yellow and indicated  go to discussion our data show that all seven pten mutant proteins that we investigated due to their identification in patients with asd retained the capacity to suppress cellular phosphoinositide 3kinaseakt signalling when they were expressed to the same level as wild type pten however most of these mutant pten proteins are unstable and it would be expected that in vivo this instability would lead to a very considerable reduction in protein abundance relative to pten wt and a resultant reduction in biological activity functional evaluation of pten mutants in a yeast based system has previously found significantly lower activity in a set of phtscowdenassociated pten mutations than in autismassociated mutations28 although it was unclear whether this effect related to direct inactivating mutations or poor protein stability it has already been reported for another mutant associated with asd pten h93r that this mutation caused a significant reduction in activity in vitro and in cells to approximately 15 of wild type activity in vitro29 in agreement with our own data it seems possible that strong but incomplete loss of activity may be related to the observation of asd symptoms in the absence of some of the more severe phenotypes seen in some patients with cowden syndrome in this regard we have observed mutations that would lead to loss of function through instability and through direct incomplete loss of activity  in contrast when a set of pten mutations associated with the most severe cases of cowden syndrome were tested for their cellular activity all of these five mutant proteins were found to display no detectable effect on akt phosphorylation even when overexpressed this implies that complete loss of activity may associate with a more severe phtscowden phenotype of these five proteins three showed stability similar to wild type pten whereas two were unstable in the cases of the stable mutants this supports a hypothesis that the expression of stable inactive protein may exacerbate the phenotype caused by pten loss of function through interference with the functions of the remaining wt pten protein and possibly other proteins strong evidence supporting the potential importance of this effect comes from the observed phenotypes of pten knockin mice in which the most severe tumour phenotype was observed in mice expressing stable pten mutants lacking lipid phosphatase activity g129e or c124s relative to either mice carrying a pten deletion or with an unstable pten mutation21 22 however other mechanisms may be involved as this would not seem to explain the occurrence of unstable catalytically inactive pten mutants l108p and l112p in severe phts therefore although the mechanisms mediating these effects are unclear our work characterises a novel correlation between specific human patient groups and the features of the pten mutations they carry  our work implies that mutationspecific factors do contribute to the severity of pten mutation carriers symptoms these effects may relate to the retention of some biological pten activity in the case of patients displaying macrocephaly and asd in the absence of more severe developmental phenotypes and to the expression of dominant negative inactive pten protein in some cases combining severe developmental and eventually tumour phenotypes biochemically potential dominant negative effects of mutant pten proteins have not been extensively studied although data supporting the functional importance of pten dimerisation has very recently been published21 the first trials are underway of potential targeted therapies such as inhibitors of the mammalian target of rapamycin mtor kinase activated downstream of pten loss to treat patients with inherited pten mutations and severe phenotypes through a deeper understanding of the signalling mechanisms driving specific phenotypes it may eventually be possible to tailor treatments to individual patient groups based upon their early symptoms and on an understanding of the cellular effects of defined classes of mutation  the tumor suppressor gene pten is frequently mutated in diverse human cancers and in autosomal dominant cancer predisposition disorders recent studies have shown that the lipid phosphatase activity of pten is critical for its tumor suppressor function and that pten negatively regulates the phosphatidylinositol 3*kinaseprotein kinase b pathway although more than half of pten mutations result in protein truncation a significant fraction of pten mutations are missense mutations to examine whether tumorderived and germlinederived missense mutations inactivate pten lipid phosphatase function we constructed 42 distinct types of pten missense mutations and expressed them in escherichia coli the purified his6tagged pten proteins were tested for their ability to dephosphorylate inositol 1345tetrakisphosphate and phosphatidylinositol 345triphosphate in addition we examined the effect of mutant ptens on the ability of pten to bind to the phospholipid membrane the results revealed that the majority of pten missense mutations 38 of 42 90 eliminated or reduced phosphatase activity and that all of the mutations examined had no effect on the membrane binding activity of pten our study indicated that phosphoinositide phosphatase activity is important for the tumor suppressor function of pten and that there may be other mechanisms of pten inactivation that are not monitored by in vitro phosphatase assay and in vitro membrane binding assay introduction the ptenmmac1tep1 gene referred to hereafter as pten was identified recently as a putative tumor suppressor gene located on human chromosome 10q233 13 somatic deletions or small mutations of pten have been observed with high frequency in malignant glioma 4 5 and endometrial cancer 6 7 and at a lower rate in other malignancies such as prostate cancer 8 small cell lung cancer 9 melanoma 10 hepatocellular carcinoma 11 and breast cancer 12 germline mutations of pten have been detected frequently 80 in the autosomal dominant hamartoma cancer syndromes cowden disease omim158350 bannayanzonana syndrome omim153480 and lhermitteduclos disease 13 14 in a murine model heterozygous knockout mice develop tumors in multiple organs whereas homozygous deletion of pten is embryonically lethal 15 16 furthermore enforced expression of pten cdna inhibits cell migration and spreading 17 18 and suppresses tumor cell growth by arresting the cell cycle at g1 phase andor by inducing apoptosis 19 this genetic and biological evidence suggests that pten has an important function in tumorigenesis as well as in normal embryonic development the pten gene contains nine exons and encodes a 403amino acid cytoplasmic protein showing extensive nh2terminal homology with tensin and auxillin as well as having a central catalytic domain showing perfect homology with ptp3 and dualspecific phosphatases 13 a key step in understanding the function of pten as a tumor suppressor is to identify its physiological substrates data thus far suggest that pten possesses two distinct phosphatase activities one is a phosphoinositide phosphatase activity against phosphoinositides such as ins1345p4 and ptdins345p3 which involves dephosphorylation of the d3position phosphate of the inositol ring 20 the other is protein phosphatase activity including focal adhesion kinase 21 several studies have shown that the phosphoinositide phosphatase activity of pten is critical for its tumor suppressor function whereas activity toward the protein substrate is not essential for growth suppression 18 2225 this suggests that the former activity may play a more important role in tumor suppression by pten because ptdins345p3 is a phospholipid second messenger produced by pi3k pten seems to be involved in the pi3k signaling pathway 26 one of the important downstream targets of pi3k is protein kinase bakt which controls cell proliferation and protects cells from apoptosis 27 the fact that both pi3k and akt play critical roles in a variety of growth factor signaling pathways raises the possibility that pten functions as a tumor suppressor through negative regulation of the pi3kakt pathway loss of pten function can occur through homozygous gene deletion point mutation plus loss of the remaining allele or loss of expression 1 2 4 28 we reviewed both somatic and germline pten mutations by surveying a database of the published literature there were about 417 mutations reported by the end of august 1999 among them 143 34 were missense mutations including 90 distinct types 162 39 were frameshift mutations 73 18 were nonsense mutations and 39 9 were other mutations containing inframe deletioninsertion or splicing site mutations the frequency of missense mutations in the pten gene was higher than that in other tumor suppressor genes except for the p53 gene which had the highest frequency of missense mutations 90 although both truncating and missense pten mutations were scattered throughout the nine exons of pten the missense mutations tended to cluster around the phosphatase domain in fact several studies showed that a limited number of reported missense mutations within the phosphatase domain could abrogate the protein phosphatase andor phosphoinositide phosphatase activity of pten 22 25 even so the majority of missense mutations including mutations outside the phosphatase doreceived 3100 accepted 5400 the costs of publication of this article were defrayed in part by the payment of page charges this article must therefore be hereby marked advertisement in accordance with 18 usc section 1734 solely to indicate this fact 1 supported in part by grantsinaid from the ministry of education science sports and culture and the ministry of health and welfare japan 2 to whom requests for reprints should be addressed at department of clinical oncology institute of development aging and cancer tohoku university 41 seiryomachi aobaku sendai 9808575 japan phone 81227178547 fax 81227178548 email chikashi@idactohokuacjp 3 the abbreviations used are ptp protein tyrosine phosphatase ins1345p4 inositol 1345tetrakisphosphate ptdins345p3 phosphatidylinositol 345triphosphate pi3k phosphatidylinositol 39kinase lmv large multilamellar vesicle ps phosphatidylserine pe phosphatidylethanolamine pc phosphatidylcholine 3147 downloaded from cancerresaacrjournalsorg on may 28 2017 \\u00a9 2000 american association for cancer research main remain to be analyzed in general analyses of the effects of missense mutations derived from a disease on the functional properties of the gene product are advantageous to elucidate the most important function for pathogenesis therefore we evaluated the pathogenic effect of a series of pten missense mutations in this study we constructed 42 distinct missense mutations that mapped within the pten open reading frame and examined the phosphoinositide phosphatase activity of the his6tagged pten protein corresponding to each point mutation against ins1345p4 and ptdins345p3 as well as their ability to bind a phospholipid membrane in vitro materials and methods vector construction and cloning of mutant pten alleles the wildtype histidinetagged pten his6pten expression vector was constructed as follows the fulllength open reading frame of pten cdna was amplified by pcr using pfu dna polymerase stratagene la jolla ca and a pair of primers with bamhi and hindiii sites the restriction endonucleasetreated pcr product was inserted into the bamhihindiii sites of the pqe30 vector qiagen hilden germany pten cdna with missense mutations s10n y16c g20e y27s l42r h61r y68h c71y h93y c105f d107y l112p l112r a121p c124r g129r g129e r130g r130l r130q v133i m134l c136y y155c g165r s170n s170r r173c r173h r173p y174n s227f g251c k289e d331g f341v k342n v343e l345q f347l v369g and t401i was constructed by megaprimer pcr 29 the bamhihindiii fragment of mutant pten cdna was introduced into the bamhihindiii site of the pqe30 vector to generate mutant his6pten expression vectors all of the plasmids were sequenced to confirm that the appropriate mutation had been incorporated and that no additional mutations were present these vectors are identical to the wildtype his6pten expression vector except for the specific mutations bacterial expression and purification of pten to induce the expression of his6pten the vector was transformed into escherichia coli strain m15 harboring prep4 qiagen the resulting transformant was cultured in 50 ml of luriabertani medium at 37\\u00b0c by midlog phase a600 nm 5 06 isopropyl bdthiogalactopyranoside was added at a concentration of 02 mm and the culture was incubated for an additional 6 h at 25\\u00b0c thereafter all of the procedures were performed at 4\\u00b0c the bacterial cells were harvested by centrifugation and the bacterial pellet was frozen at 280\\u00b0c until use the frozen pellets were resuspended in 1 ml of icecold lysis buffer containing 50 mm nah2po4 ph 80 500 mm nacl 5 mm imidazole 5 mm 2mercaptoethanol and 1 mm phenylmethylsulfonyl fluoride and bacteriolysis was performed by sonication until the cell suspension became transparent after the addition of 10 ml of tween 20 the lysate was incubated on ice for 30 min and then centrifuged at 15000 3 g for 20 min the supernatant was mixed with 50 ml of nintaagarose qiagen for 30 min at 4\\u00b0c washed three times with 250 ml of the wash buffer containing 50 mm nah2po4 ph 80 300 mm nacl and 20 mm imidazole and eluted three times with 50 ml of the elution buffer containing 50 mm nah2po4 ph 80 300 mm nacl and 250 mm imidazole the buffer of the eluted solution was then replaced with 500 ml of ted buffer containing 20 mm trishcl ph 80 2 mm edta 2 mm dtt 300 mm nacl and 1 mm phenylmethylsulfonyl fluoride by using nanosep pall filtron northborough ma and centrifuged to a volume of 50 ml the purified protein was stored in the presence of 2 vv glycerin at 280\\u00b0c until use protein concentrations and the integrity of fusion proteins were determined by sdspage and by comparison with known concentrations of bsa ins1345p4 phosphatase assay the inositol phosphatase assay was performed using the method described by maehama and dixon 20 in a 20ml reaction volume consisting of 100 mm trishcl ph 80 10 mm dtt 60 mm  3 hins1345p4 001 mci new england nuclear boston ma and 1 mg of the purified his6pten protein at 37\\u00b0c for 30 min the reaction was terminated by the addition of 1 ml of stop solution consisting of 01 m hcooh and 06 m hcoonh4 to separate the dephosphorylated product  3 hins145p3 from the substrate the reaction sample was applied to a ag1x8 column 05 ml biorad hercules ca equilibrated with the stop solution and eluted with 5 ml of the stop solution radioactivity in the eluted solution was measured using a liquid scintillation counter ptdins345p3 phosphatase assay the his6pten protein used for the ptdins345p3 phosphatase assay was extracted again as described above except that nah2po4 was replaced with 50 mm trishcl ph 80 in the buffer used for lysis washing and elution the phosphatase assays 30 were performed in 50 ml of reaction buffer containing 100 mm trishcl ph 80 10 mm dtt 200 mm watersoluble dic8ptdins345p3 echelon salt lake city ut and 2 mg of his6pten protein at 37\\u00b0c for 40 min the phosphate released from the substrate was measured using green reagent biomol plymouth meeting pa according to the manufacturers instructions in brief 500 ml of green reagent were added followed by incubation for 20 min at room temperature the concentration of the released phosphate was determined by measuring the a620 nm a standard curve was generated in each assay and the amount of free phosphate was calculated from the standard curve linefit data phospholipid binding assay the pten protein binding to lmvs consisting of three different phosphoglycerides ps pe and pc sigma st louis mo was carried out according to the procedures published previously 31 33 with some modifications briefly 100 mg of lmvs pspepc ratio 355015 were prepared in 10 ml of buffer containing 50 mm hepeskoh ph 72 and 100 mm nacl by sonication and collected by centrifugation an aliquot of his6pten protein 25 mg and 100 ml of lmvs were incubated in 50 mm hepeskoh ph 72 and 2 mm cacl2 for 15 min at 25\\u00b0c after centrifugation at 12000 3 g for 10 min at 4\\u00b0c the pellets containing lipid and bound proteins were washed in 500 ml of the abovementioned buffer and dissolved in sds sample buffer the addition of an equal volume of 20 wv trichloroacetic acid precipitated the proteins in the supernatants after a 15min incubation on ice the samples were centrifuged at 12000 3 g for 15 min at 4\\u00b0c washed twice with acetone and mixed with sds sample buffer equal proportions of the supernatants and pellets were analyzed by 10 wv sdspage followed by silver staining daiichi pure chemicals tokyo japan results to examine whether the missense mutation inactivates the phosphoinositide phosphatase function of pten we selected 42 distinct tumorderived or germlinederived missense mutations covering more than 40 of the reported missense mutation types each mutation was constructed by sitedirected mutagenesis expressed as a his6tagged fusion protein his6pten in e coli and purified as described in \\u201cmaterials and methods\\u201d the amount of expressed his6pten was measured by spectrophotometer and confirmed by sdspage the representative data are shown in fig 1 to assess phosphoinositide phosphatase activity we used two phosphoinositide substrates with ptdins345p3 or without ins1345p4 a diacylglycerol chain bacterially expressed pten has been shown to dephosphorylate the phosphate in the d3 position of the inositol ring in each substrate 20 30 33 phosphatase activities of pten against ins1345p4 to examine whether the missense mutations inactivate normal pten phosfig 1 bacterial expression and purification of pten recombinant his6pten expressed from wildtype pten null pqe30 vector and mutant pten y68h s170n g251c k289e d331g and l345q in e coli was purified using nintaagarose as described in \\u201cmaterials and methods\\u201d approximately 051 mg of the proteins was separated by sdspage and visualized by coomassie blue staining bsa was used for judging the concentration of the pten protein null null pqe30 vector as a negative control pten wildtype pten as a positive control 3148 functional evaluation of pten missense mutations downloaded from cancerresaacrjournalsorg on may 28 2017 \\u00a9 2000 american association for cancer research phatase activity against ins1345p4 his6pten harboring the indicated amino acid substitutions was assayed for its ability to dephosphorylate the phosphate in the d3 position of ins1345p4 using the method described previously ref 20 fig 2 the results showed that four mutant ptens 9 s10n l42r v369g and t401i retained activity comparable with or even higher than that of normal pten and seven mutant ptens 17 g20e m134l s227f k289e d331g k342n and f347l retained partial activity fig 2 the remaining 31 mutant ptens 74 had no activity phosphatase activities of pten against ptdins345p3 because ins1345p4 lacks the diacylglycerol chain of ptdins345p3 we compared the phosphoinositide phosphatase activities of 12 selected mutant ptens and normal pten toward the two substrates these ptens contained mutants with full s10n l42r v369g and t401i partial g20e m134l s227f and k342n and no y68h g129e r130g and r173c phosphatase activities against ins1345p4 watersoluble ptdins345p3 was used to examine the released phosphate the eight mutant ptens with full or partial ins1345p4 phosphatase activity showed nearly the same level of phosphatase activity against ptdins345p3 as wildtype pten whereas the four mutant ptens with no ins1345p4 phosphatase activity also eliminated ptdins345p3 phosphatase activity fig 3 these results indicated that the effects of the missense mutations on pten phosphatase activity against the two phosphoinositides were nearly identical although the assay using ins1345p4 may be more sensitive to subtle structural changes in pten than that using ptdins345p3 ability of pten to bind to lmvs during our study of pten phosphatase activities a second biochemical property of pten was reported the ability to bind to lipid membranes in vitro through the c2 domain 33 this raised the possibility that some pten mutations may inactivate the physiological function of pten through disruption fig 3 phosphatase activity of pten against ptdins345 p3 two mg of his6pten were assayed for phosphoinositol phosphatase activity against watersoluble dic8 ptdins345p3 the amount of free phosphate released in the reaction was measured at a620 nm and compared with a standard curve the data are representative of three experiments that yielded similar results fig 2 phosphatase activity of pten against ins1345p4 one mg of his6pten protein was assayed for phosphoinositol phosphatase activity against 3 hins1345p4 as described in \\u201cmaterials and methods\\u201d the radioactivity of the dephosphorylated product was counted and the results were normalized to wildtype pten 100 the 42 pten mutants were divided into seven groups a2g for protein extraction and phosphatase assay the data are representative of three independent experiments that yielded similar results 3149 functional evaluation of pten missense mutations downloaded from cancerresaacrjournalsorg on may 28 2017 \\u00a9 2000 american association for cancer research of the membrane binding activity to examine this possibility 15 mutant ptens with s10n l42r k289e d331g v369g and t401i or without y16c y68h r130q r130l g165r s170n r173c g251c and l345q phosphatase activity were subjected to a proteinmembrane binding assay using lmvs that consisted of three different phosphoglycerides pc ps and pe according to procedures published previously 31 32 representative data of the selected mutant ptens are shown in fig 4 when bacterial proteins copurified with pten failed to bind lmvs all mutants tested retained the binding activity of pten irrespective of whether or not they had phosphoinositide phosphatase activity this result suggested that the ability of pten to bind to membranes is not a major target of tumorderived or germlinederived missense mutations even if the activity may be important for physiological pten function discussion since its discovery pten has been thought to be a protein phosphatase because it shares the signature motif hcxxgxxr with the active site of ptps and dualspecific protein phosphatases however recent studies have shown that the biologically relevant targets of pten may not be phosphoproteins but rather a subset of phosphoinositides 20 22 25 this speculation can be supported by several findings for example tumor cell lines with mutant pten retained elevated levels of ptdins345p3 and akt activity and the introduction of wildtype pten reduced the levels of both 34 35 furthermore analysis of the pten crystal structure indicated that the phosphatase active site of pten is larger than that of the ptps and that the coohterminal portion has a structure similar to the c2 domain and actually binds to phospholipid membranes in vitro 33 to elucidate the most important tumor suppressor function of pten the analysis of the effect of the missense mutations on currently known functional properties of pten will be beneficial until all of the functions of pten are clarified thus far only a limited number of tumorderived and germlinederived pten missense mutations have been examined for known pten functions all of the missense mutations that have been tested inactivated phosphoinositide phosphatase activity whereas some of the mutations retained protein phosphatase activity 22 25 although these observations stressed the importance of phosphoinositide phosphatase activity it was necessary to determine whether the reported pten missense mutations inactivate the currently proposed functions of pten as an initial study we constructed 42 missense mutations which covered more than 40 of reported missense mutations mapped throughout from the nh2terminal and coohterminal portions of pten and tested them against the phosphoinositide phosphatase activity and membrane binding activity of wildtype pten we classified them according to the recent report describing pten crystal structure 33 there were 31 mutations s10n y16c g20e y27s l42r h61r y68h c71y h93y c105f d107y l112p l112r a121p c124r g129r g129e r130g r130l r130q v133i m134l c136y y155c g165r s170n s170r r173c r173h r173p and y174n within the nh2terminal phosphatase domain residues 7185 9 mutations s227f g251c k289e d331g f341v k342n v343e l345q and f347l within the coohterminal c2 domain residues 186351 and 2 mutations v369g and t401i close to the coohterminal end among these mutations most of the mutations within the nh2terminal phosphatase domain 27 of 31 mutations 87 inactivated phosphatase activity in contrast a higher percentage of mutations within both the c2 domain five of nine mutations 56 and the coohterminal end two of two mutations 100 retained phosphatase activity overall our results showed that most mutations 38 of 42 mutations 90 eliminated or reduced phosphatase activity the mutations within the phosphatase domain contain eight mutations located at the predicted active site pocket including six mutations c124r g129r g129e r130g r130l and r130q in the hcxxgrxxr signature motif of the p loop residues 123130 at the bottom of the pocket and two mutations h93y and g165r at the walls of the pocket the wpd loop residues 9097 and the ti loop residues 160168 respectively all mutations within the active site pocket eliminated phosphatase activity the mutations in the motif represent 19 27 of 143 of the total missense mutations according to our statistics from our functional analysis and the previous genetic and structural analyses we conclude that pten phosphoinositide phosphatase activity is an important tumorsuppressive function of pten however 26 11 of 42 of the mutants retained some level of the normal pten phosphatase activity in vitro although there may be a polymorphisms among these mutants it is likely that the in vitro phosphatase assays do not always reflect the in vivo phosphatase activity of pten and that there might be other mechanisms of pten inactivation in our study most of the mutations that mapped to the coohterminal region retained some levels of phosphoinositide phosphatase activity according to recent studies there may be regulatory domains in the coohterminal region georgescu et al 30 have shown that two missense mutations l345q and t348i in the predicted bstrand structure residues 342349 seem to destabilize pten in mammalian cells although these mutations are also defective in the phosphatase assay by bacterially expressed proteins lee et al 33 have shown that the predicted c2 domain in the coohterminal portion bound to phospholipid membranes they also speculated that the c2 and the phosphatase domains associate across an extensive interface adjacent to the phosphatase active site furthermore they predicted that the c2 domain may not only recruit pten to the membrane but may also help with the positioning and orientation of the catalytic domain with respect to the membranebound substrate in this study we could not find any tumorderived mutations that affected the membrane binding activity of pten which suggested that the binding of pten to membranes is not a major target of the missense mutations our results do not exclude the possibility of binding activity as an important tumor suppressor function of pten because there are high frequencies of pten frameshift 39 and nonsense 18 mutations that are predicted to eliminate the pten membrane binding domain and perhaps preserve enzyme activity it would be of great interest to determine whether the mutations in the c2 domain which have both in vitro phosphatase and membrane binding activities affect the phosphatase activity under the conditions of pten protein binding to the phospholipid membrane pten also has another domain called the pdz binding domain in the coohterminal end 33 36 the domain binds to the pdz proteins dlg1 and mast205 and this binding is regulated by phosphorylation of the tyrosine residue at codon 401 it would also be fig 4 pten binding activity to lmvs a representative silverstained gel of protein bound to lmvs is shown p the pellet fraction coprecipitated with lmvs s the supernatant fraction without binding to lmvs note that proteins other than ptens exist only in the supernatant fractions 3150 functional evaluation of pten missense mutations downloaded from cancerresaacrjournalsorg on may 28 2017 \\u00a9 2000 american association for cancer research interesting to determine whether tumorderived t401i mutations with in vitro phosphatase activity inactivate binding to the pdz proteins finally we found that some mutants with partial phosphatase activity against ins1345p4 g20e m134l s227f and k342n showed nearly wildtype levels of phosphatase activity against ptdins345p3 it is possible that certain pten mutations impair phosphatase activity only toward one type of inositol phosphate molecule ins1345p4 but not others ie ptdins345p3 and therefore affect distinct cellular signaling pathways because these two molecules have different functions in the cell ins1345p4 is a second messenger involved in regulating intracellular calcium signaling whereas ptdins345p3 is well known as a second messenger molecule generated by pi3k activity 20 taken together pten phosphatase activity could be regulated by subcellular location phosphorylation andor interaction with other cellular proteins the physiological function of pten remains to be clarified in additional experiments   since their discovery protein tyrosine phosphatases have been speculated to play a role in tumor suppression because of their ability to antagonize the growthpromoting protein tyrosine kinases recently a tumor suppressor from human chromosome 10q23 called pten or mmac1 has been identified that shares homology with the protein tyrosine phosphatase family germline mutations in pten give rise to several related neoplastic disorders including cowden disease a key step in understanding the function of pten as a tumor suppressor is to identify its physiological substrates here we report that a missense mutation in pten pteng129e which is observed in two cowden disease kindreds specifically ablates the ability of pten to recognize inositol phospholipids as a substrate suggesting that loss of the lipid phosphatase activity is responsible for the etiology of the disease furthermore expression of wildtype or substratetrapping forms of pten in hek293 cells altered the levels of the phospholipid products of phosphatidylinositol 3kinase and ectopic expression of the phosphatase in ptendeficient tumor cell lines resulted in the inhibition of protein kinase pk bakt and regulation of cell survival  glioblastoma is one of the most common and malignant forms of cancer it is often characterized by the constitutive activation of epidermal growth factor egfdependent signaling pathways caused by the amplification of members of the egf receptor family of protein tyrosine kinases ptks the products of tumorsuppressor genes may attenuate these signaling pathways and therefore their loss through deletion or mutation may contribute to tumor progression the frequent loss of heterozygosity at tumorsuppressor loci is often used to identify tumorsuppressor genes the 10q23 region of human chromosome 10 is frequently deleted or mutated in a wide variety of tumor types most frequently in glioblastoma endometrial cancer and prostate cancer indicating the presence of a tumorsuppressor gene at this locus pten was subsequently identified as this tumor suppressor and was found to contain the catalytic signature motif detected in all members of the protein tyrosine phosphatase ptp family 1 2 importantly pten appears to be lost frequently in advanced cancers suggesting that its deletion may not be the transforming event but that pten may inhibit other cellular functions necessary for tumor progression 3 4  importantly germline transmission of mutations in pten was shown to give rise to a related set of disorders including cowden disease that are characterized by numerous small benign tumors and an increased incidence of other malignant growths 57 the detection of germline mutations in these neoplastic disorders verifies that pten is the tumor suppressor residing on human chromosome 10q23 this conclusion is supported by the recent findings that pten knockout mice develop tumors similar to those found in cowden disease 8  a key step in establishing the cellular function of pten is the identification of its physiological substrates recently two groups have identified different candidate substrates for pten tamura et al 9 report that overexpression of pten results in changes in cell adhesion and spreading and suggest that these effects result from the the dephosphorylation of the ptk focal adhesion kinase fak in contrast maehama and dixon 10 have shown that pten recognizes phosphatidylinositol phosphate ptdinsp as a substrate our initial studies demonstrating catalytic activity in pten indicated that it preferred highly acidic or multiply phosphorylated substrates 11 this may explain why in certain contexts pten will dephosphorylate polyphosphorylated molecules such as fak or ptdinsp 11 12 nevertheless the important question remains as to which multiply phosphorylated molecules protein or otherwise are the physiologically relevant targets of pten  significantly we have demonstrated previously that the majority of missense mutations isolated from tumor and cowden disease samples ablate the protein phosphatase activity of pten 11 one exception is a missense mutation that changes a glycine residue in the catalytic signature motif to a glutamate pteng129e see ref 11 although there was no adverse effect on the protein phosphatase activity of pten this g129e mutation was isolated from two independent cowdendisease kindreds indicating that it abolishes the tumorsuppressor activity of pten 5 12 therefore this mutation can be used as an important indicator to determine whether a proposed function of pten is specific for its role as a tumor suppressor  we analyzed the pteng129e cowden disease mutation and found that this mutation specifically inhibits the recognition of ptdinsps by pten additionally we show that expression of pten in mammalian cells results in changes in ptdins345p3 levels and expression of pten in two independent glioblastoma cell lines results in the disruption of signaling downstream of phospahatidylinositol 3kinase pi 3kinase to pkbakt and bad significantly we also show that expression of pten in lncap cells a prostatetumor cell line abrogates cell survival and that this effect is inhibited by the expression of a constitutively activated form of pkb therefore we believe that the physiological function of pten is to antagonize signaling downstream of pi 3kinase by dephosphorylating ptdinsps  go to methods pten phosphatase assays recombinant wildtype and mutant forms of pten were expressed in eschirichia coli and purified by glutathioneaffinity chromatography 11 the purified proteins were assayed with polyglutyr as described 11 release of 32pi from radiolabeled ptdinsps was determined by performing a modified bligh and dyer extraction 13 the upper phase containing inorganic phosphate was removed dried and resuspended in a 1 m trichloroacetic acid1 ammonium molybdate solution after extraction with 2 vol of tolueneisobutylalchohol 11 volvol the upper phase was removed and counted site selectivity was determined by incubating recombinant pten or ship a gift from c erneux free university brussels with radiolabeled ptdins345p3 and the lipid products of these reactions were analyzed by tlc or hplc 13 the phospholipids were extracted by performing the modified bligh and dyer extraction described and the resulting lower phases were dried resuspended in 20 \\u03bcl of chloroformmethanol 21 volvol and applied to an oxalateactivated silica 60 tlc plate plates were developed in methanolchloroformwaterammonia 100752515  determination of ptdins345p3 levels hek293 cells were transfected via calcium phosphate coprecipitation with 20 \\u03bcg of dna per 10cm dish the calcium phosphate\\u22c5dna coprecipitate was removed by washing with pbs 16 hr after addition and the cells were returned to growth medium for 36 hr before harvesting cotransfection of pten and p110 constructs were performed by using 95 \\u03bcg of each plasmid dna transfection efficiency was determined by including a green fluorescent protein expression plasmid 1 \\u03bcg in all transfections and was shown to be 80 phospholipids were extracted from the cells exactly as described 14 ptdins345p3 levels were assayed by using a ligand displacement assay 14 and were normalized to total protein  expression of pten in glioblastoma cell lines pten retroviral expression vectors were constructed in pbabepuro 15 after transfection into packaging lines the viral supernatants were harvested diluted with growth medium and incubated with u87mg or u373mg cells for 8 hr at 32\\u00b0c in the presence of polybrene infected cells were selected with puromycin 2 \\u03bcgml and drugresistant colonies were expanded to generate clonal cell lines cell lysis and immunoblots were performed essentially as described 16 protein levels were determined by the method of bradford using bsa as a standard and equal protein was loaded in each lane antibodies to pten were generated in rabbits by using a cterminal peptide of pten enepfdedqhtqitkv conjugated to keyhole limpet hemocyanin antibodies to pkbakt phosphopkbakt specific to phosphorylated ser473 and phosphobad were purchased from new england biolabs  expression of pten in lncap cells cells were transfected by using cationic lipids transfast promega at a dnalipid ratio of 1 cells were incubated with the dnalipid complexes for 12 hr and transfected cells then were identified based on the expression of a cotransfected gfp vector or by performing immunoflourescence by using antibodies to the haepitope tag located at the n terminus of pten essentially as described 17  go to results activity of pten toward inositol phospholipids recombinant pten produced in e coli 11 was assayed for its ability to release 32pi from radiolabeled ptdins345p3 or polyglutyr fig fig11a as expected wildtype pten catalyzed the dephosphorylation of both substrates fig fig11a remarkably the activity of pteng129e toward ptdins345p3 was reduced by \\u224890 relative to the wildtype enzyme while still retaining activity toward polyglutyr fig fig11a a catalytically inactive mutant of pten ptenc124s was unable to dephosphorylate polyglutyr or ptdins345p3 indicating that the phosphatase activity was not caused by a bacterial contaminant in addition cdc14 a dualspecificity phosphatase closely related to pten was also unable to dephosphorylate ptdins345p3 demonstrating that recognition of this phospholipid substrate is not a general property of other even closely related dualspecificity phosphatases these data demonstrating that the pteng129e mutation specifically ablates the activity of pten toward phosphorylated inositol lipids demonstrates that the lipid phosphatase rather than the protein phosphatase activity of pten is required for it to function as a tumor suppressor and that loss of the lipid phosphatase activity results in cowden disease  figure 1 figure 1 pten is a ptdins 3phosphatase a recombinant pten was incubated with radiolabeled polyglutyr and ptdins345p3 and the release of 32pi was measured as described b ptdins345p3 labeled at the 3 position was incubated with pten or ship for  site specificity of pten within ptdins345p3 to determine whether pten recognizes specific sites in the inositol ring of ptdins345p3 lipid substrate labeled exclusively in the 3 position with 32p was incubated with pten or ship a well characterized 5phosphatase 18 the products of these reactions were analyzed by tlc incubation with ship yielded a radiolabeled product with the expected mobility of ptdins34p2 fig fig11b however no labeled lipid products were generated following treatment with pten under conditions where >50 of the 32p was lost from the substrate fig fig11b the site selectivity of pten was confirmed by hplc analysis of similar reactions by using 3h and 332plabeled ins1345p4 in which the only products detected were 32pi and 3hins145p3 data not shown in addition to ptdins345p3 pten also hydrolyzed the other potential products of pi 3kinase ptdins3p and ptdins34p2 with the following rank order based on relative percent hydrolysis ptdins345p3  ptdins34p2 > ptdins3p > ins1345p4 the specificity of pten for 3phosphorylated inositol lipids indicates that it may function as a negative regulator of pi 3kinasemediated signaling and distinguishes pten from other known lipid phosphatases  pten inhibits pi 3kinasedependent signaling the potential for pten to antagonize pi 3kinase signaling was further investigated in mammalian cells fig fig22 transient expression of pten in hek293 cells lowered the levels of ptdins345p3 fig fig22 from 1 pmolmg to 07 pmolmg importantly ptenc124s in which the catalytic cysteine has been mutated to serine resulted in an increase in the levels of ptdins345p3 fig fig22 from 1 pmolmg to 2 pmolmg the accumulation of ptdins345p3 in response to expression of ptenc124s is likely because of the ability of this form of pten to behave as a \\u201csubstratetrapping\\u201d mutant resulting in a stable complex with the lipid substrate that protects it from dephosphorylation by endogenous phosphatases 19 the accumulation of ptdins345p3 in the presence of a substratetrapping mutant confirms that ptdins345p3 is a physiological target of pten 19 20 expression of a constitutively activated pi 3kinase p110caax resulted in elevated levels of ptdins345p3 that were reduced \\u224860 following coexpression with pten and were increased \\u224845fold when coexpressed with ptenc124s fig fig22 the ability of pten to decrease ptdins345p3 levels produced by p110caax further indicates that this phosphatase exerts its effects on the products of pi 3kinase rather than by dephosphorylating the tyrosine residues responsible for recruiting the p85p110 pi 3kinase complex to the membrane  figure 2 figure 2 expression of ptenantagonized pi 3kinase hek293 cells were transfected with pten or ptenc124s in combination with the p110 subunit of pi 3kinase or an activated membranebound pi3 kinase p110caax the resulting levels of ptdins345p3 were  importantly we detected defects in pi 3kinase signaling in tumor cell lines that have lost their endogenous pten genes through deletion or mutation signaling downstream of pi 3kinase was assessed in two of these ptendeficient cell lines u87 mg and u373 mg by determining the phosphorylation status of pkbakt a kinase whose activation depends on ptdins345p3 or ptdins34p2 two products of pi 3kinase 21 clonal cell lines were generated in u87 mg and u373 mg in which pten expression was restored by infection with recombinant retroviruses fig fig33 the parental cell lines exhibited high basal levels of phosphorylated pkbakt whereas in every wildtype ptenexpressing clonal cell line the basal levels of activated pkbakt were significantly lowered fig fig33 expression of a catalytically inactive mutant of pten ptenr130m had no effect on basal pkbakt activation fig fig33 similarly no effect on pkbakt activation was observed in a u373 mg ptensrve no b9 clone infected with an unstable mutant of pten fig fig33 after reconstitution of pten expression in the tumor lines activation of pkbakt by insulin or plateletderived growth factor was unaffected fig fig33 suggesting that there are regulatory mechanisms that allow for the generation of ptdins345p3 following growth factor stimulation the activation of pkbakt was completely inhibited by wortmannin in the tumorcell lines fig fig33 indicating that the increase in pkbakt phosphorylation was pi 3kinasedependent  figure 3 figure 3 expression of pten in glioblastoma cell lines decreases the amount of activated pkbakt pten expression was reconstituted in u87mg a or u373mb glioblastoma b cell lines by infection with recombinant retrovirus confluent dishes were either  regulation of cell survival one of the downstream substrates of pkbakt is the deatheffector protein bad 22 phosphorylation of which has been linked to the promotion of cell survival 23 although expression of pten in both u87 mg and u373 mg resulted in the reduction of bad phosphorylation fig fig33b we did not detect evidence of apoptosis in the ptenexpressing clones however expression of wildtype pten in lncap cells a ptendeficient prostatecancer cell line resulted in a decrease in the number of ptenpositive cells recovered relative to controls expressing the catalytically inactive mutant ptenc124s fig fig44 in this assay the cowden disease mutation pteng129e behaved similarly to ptenc124s consistent with the ablation of the tumorsuppressor activity of pten by this mutation fig fig44 significantly coexpression of a constitutively active membranetargeted pkbakt 24 completely reverted the pten phenotype in that ptenpositive cells could now be recovered despite the presence of wildtype pten fig fig44  figure 4 figure 4 expression of pten inhibits cell survival in lncap cells pten ptenc124s or pteng129e were cotransfected with a green fluorescent protein gfp expression vector into lncap cells the cotransfections also included either an expression vector for consitutively  go to discussion pten is a newly defined tumorsuppressor gene that has been implicated in a wide variety of cancers and in a series of related disorders that are characterized by a predispostion to cancer 1 2 2528 although these studies offer many insights into the biology of pten they do not address the molecular mechanism of pten function which requires the identification of its physiological substrates however this has become an area of controversy tamura et al 9 have suggested that pten disrupts cell spreading and migration by dephosphorylating fak in contrast maehema and dixon 10 have shown that in certain contexts pten can dephosphorylate the lipid second messenger ptdinsp however both of these studies fall short of demonstrating that the ability of pten to dephosphorylate fak or ptdinsp is related to its tumorsuppressive function in light of the facts that previous studies have shown that pten prefers highly acidic or multiply phosphorylated substrates 11 and that fak and ptdinsp share these properties one must view these results with caution as they may result from nonspecific interactions between pten and highly charged molecules 12  significantly we have been able to show that a single point mutation in pten pteng129e specifically ablates the ability of pten to recognize phospholipids as substrates in contrast tamura et al 9 found that the pteng129e point mutation behaves essentially like wildtype pten in their assays of cell spreading even though this mutation does not alter the ability of pten to recognize proteinaceous substrates the fact that it has been identified in two independent cowden disease kindreds indicates that this mutation would be expected to impair the tumorsuppressor activity of pten 5 11 therefore these findings demonstrate that the lipid phosphatase activity rather than the protein phosphatase activity of pten is important for its tumorsuppressive activity and that fak at least in this respect is not a relevant target of the phosphatase  characterization of the lipid phosphatase activity of pten demonstrates that it shows specificity for phosphatidylinositols phosphorylated at the 3 position this represents the first phosphatidylinositol phosphatase with specificity for the lipid products of pi 3kinase and suggests that pten may function to antagonize the growthpromoting signals generated by pi 3kinase indeed overexpression of pten in mammalian cells disrupted the pi 3kinasedependent production of ptdins345p3 furthermore expression of a catalytically inactive mutant of pten ptenc124s which may function as a substrate trap results in the accumulation of ptdins345p3 indicating that pten may function in vivo to antagonize pi 3kinasedependent signaling  importantly signaling downstream of pi 3kinase is altered in two different glioblastoma cell lines u87 mg and u373 mg which have lost expression of pten because of mutation or deletion of both alleles both cell lines exhibited elevated levels of active phosphorylated pkbakt a serinethreonine kinase whose activation requires the production of ptdins345p3 or ptdins34p2 21 29 restoration of pten expression by retroviral transduction resulted in the reduction of activated pkbakt as is evident from the observed reduction in the phosphorylation of pkbakt and in reduced levels of phosphorylated bad an apoptosis regulator that is a known in vivo substrate of pkbakt 22 significantly insulin or plateletderived growth factor stimulation of these cell lines resulted in the stimulation of pkbakt regardless of whether pten expression had been reconstituted suggesting that there are mechanisms in place which in response to growth factor stimulation serve to regulate the activity of pten and allow the production of ptdins345p3 this is especially important given that pten is widely expressed including cell lines that produce ptdins345p3 in response to insulin  although the reconstitution of pten decreased the levels of phosphorylated bad we were unable to detect evidence of apoptosis in these cell lines pkbakt is thought to affect cell survival in many cell types by phosphorylating bad or inactivating gsk3\\u03b2 23 30 reconstitution of pten in lncap cells a ptendeficient prostatecancer cell line resulted in the inhibition of cell survival importantly expression of the pteng129e cowden disease point mutation does not result in the abrogation of cell survival this observation indicates that the lipid phosphatase activity of pten is required for these effects on cell survival the properties of the pteng129e mutant emphasize the requirement of the lipid phosphatase activity for the tumorsuppressor function of pten furthermore the ptendependent induction of apoptosis was completely inhibited by coexpressing a constitutively active membranetargeted form of pkbakt indicating that pten by dephosphorylating phosphatidylinositides is an upstream regulator of pkbakt and at least in lncap cells serves to regulate survival signals thus in some cell types pten functions as a tumor suppressor to inhibit the pkbaktdependent survival signals that are activated in response to 3phosphorylated phosphatidylinositols  the failure of pten to regulate cell survival in the glioblastoma cell lines suggests that the generation of phosphatidylinositol 3phosphates plays other roles during tumor progression in fact pi 3kinase has been implicated in the regulation of a wide variety of cellular processes 31 therefore by dephosphorylating the products of pi 3kinase pten is likely to be involved in the regulation of many of these processes significantly both pi 3kinase and pkbakt have been isolated as transforming oncogenes in retroviruses 32 33 these observations indicate that pi 3kinase and pkbakt function in a signaling pathway promoting growth and survival our data demonstrate that pten functions to suppress these growthpromoting and survival signals by dephosphorylating the phospholipid products of pi 3kinase our data provide a molecular basis for the development of cowden disease as loss of the lipid phosphatase activity of pten leads to the accumulation of the products of pi 3kinase and results in the neoplastic pathologies characteristic of the disease\",\n          \"CTNNB1 S37C we screened 75 primary hepatocellular carcinomas for somatic muta tions in the entire coding region of the 0catenin gene we detected somatic mutations in 14 tumors 12 were considered to cause amino acid substitutions and 2 were interstitial deletions of 51 or 195 nucleotides of genomic dna corresponding to exon 3 among the 12 point mutations 6 occurred at potential serinethreonine phosphorvlation residues of codons 33 41 or 45 the remaining six tumors contained a mutation at codon 32 aspartic acid or 34 glycine flanking to the serine residue at codon 33 by western blot analysis we confirmed accumulation of \\u00e3\\u00ffcateninin five tumors for which frozen tissues were available the five included tumors in which amino acid alterations had occurred at codons 32 34 or 45 and one with a 17amino acid deletion our results suggested that accumula tion of \\u00e3\\u00ffcatenin due lo amino acid substitutions at potential serine threoninc phosphorvlation residues or at their neighboring codons or interstitial deletions involving exon 3 could contribute to hepatocellular carcinogencsis introduction continuous destruction and regeneration of hepatocytes as a con sequence of hepatitis b or c virus infection often results in chronic hepatitis and subsequent liver cirrhosis because these infections are widespread in asian populations and liver cirrhosis is considered a precancerous lesion for hcc' the incidence of hcc in asian coun tries is significantly higher than in other parts of the world 1 however as with other solid tumors it is presumed that development and progression of liver carcinomas require an accumulation of ge netic alterations in hepatic cells in addition to the damage occasioned by chronic viral infection activation of cmvc or cyclin dl by dna amplification 2 3 and inactivation of tumor suppressor functions of p53 or rb genes 4 have in fact been observed in some hccs but at relatively low frequencies therefore the genetic mechanism of hepatocellular carcinogenesis remains unsolved reduced expression of ecadherin due to methylations at cpg sites around the promoter region and allelic deletions of the ecadherin gene itself have been frequently recognized in hccs 5 6 further more differences in growth patterns or in differentiation grades of received 3998 accepled 43098 the costs of publication ol' this article were defrayed in part by the payment of page charges this article must therefore he hereby marked advertisement in accordance with 18 usc section 1734 solely to indicale this fact 1this work was supported in part by a special grant for strategic advanced research on cancer from the ministry of kducation culture sports and science of japan and by research lor the kulure program grant lmh02 from the japan society for the promotion of science \\\" to whom requests for reprints should be addressed at laboratory of molecular medicine human genome center the institute of medical science the university of tokyo 461 shirokanedai minatoku tokyo 1088639 japan phone 81354495372 fax 81354495433 email yusuke@imsutokyoacjp 1the abbreviations used are hcc hepatocellular carcinoma feadherin epithelial cudhcrin apc adenomalous polyposis coli gsk3\\u00e3\\u00ffglycogen synthase kinase3\\u00e3\\u00ffrt reverse transcription sscp singlestrand conformation polymorphism hccs can be correlated with altered expression of ecadherin 7 ecadherin forms complexes with catenin molecules such as \\u00e3\\u00ffcatenin the complexes connect to the actin filament network and also to cytoplasmic proteins such as fodrin these interactions play important roles in cell migration cell separation and formation of cell layers processes that polarize hepatocytes and shape hepatic plates in the liver 8 \\u00e3\\u00ffcatenin is involved in the wntwingless signaltransduction pathway 9 oncogenic activation of this protein by amino acid substitutions or interstitial deletions has been noted in colorectal cancer cell lines and melanoma cell lines as well as in sporadic colorectal cancers 1014 intracellular concentrations of \\u00e3\\u00ffcatenin are mainly regulated by degradation which is probably initiated by interaction with apc protein and by phosphorylation at serine andor threonine residues of codons 33 37 41 and 45 through the action of gsk3\\u00e3\\u00ff1517 mutations of apc or \\u00e3\\u00ffcateninresult in stabiliza tion of \\u00e3\\u00ffcateninand a significant increase of this protein within the cell accumulated \\u00e3\\u00ffcateninmay translocate into the nucleus where it could serve as a transcriptional factor through binding with highmobility group box factors of the tcflef family 18 19 and stim ulate tumor formation in addition to decreased levels of ecadherin strong immunohistochemical staining of \\u00e3\\u00ffcateninhas been detected in some hccs 7 these observations have implied a possible asso ciation between disruption of the cadherincatenin system and tumorigenesis in the liver here we report results of screening of genetic alterations in the \\u00e3\\u00ffcateningene in 75 primary hccs and discuss a possible role of \\u00e3\\u00ffcateninactivation in hepatic carcinogenesis materials and methods tissue samples tumors and corresponding noncancerous liver tissues were obtained from patients who underwent hepatectomy at osaka medical center for cancer and cardiovascular diseases osaka japan or municipal ikcda hospital osaka japan with informed consent all tumors were diag nosed histopathologically as hccs preparation of dnas and rnas dnas were extracted from these tu mors and corresponding noncanccrous liver tissues according to methods described elsewhere 14 total rnas were also isolated from each samples with trizol reagent life technologies inc according to the manufactur er's protocol rtpcrsscp three ng of total rna were reversely transcribed for singlestrand cdna using oligodt15 primer and superscript ii life tech nologies inc each singlestrand cdna was used as a template for subse quent pcrs that covered the entire coding regions with 11 segments primer sets used for pcr amplifications were as follows set 1 if 5'ccatacaactgttttgaaaatccvand ir vtgagctcgagtcattgcatac3' set 2 2f 5'cagtccttcactcaagaacaag3' and 2r 5'atagttaatcaagtttacaactgc3' set 3 3f vgtttggctgaaccatcacag3' and 3r vgttatgcaaggtcccagcgg3' set 4 4f s'atcatgcgttctcctcagatgi' and 4r s'caccagcta 2524 downloaded from cancerresaacrjournalsorg on may 26 2017 \\u00a9 1998 american association for cancer research  mutations of the 3catenin gene in hccs a t tcttacctggactctggaat 110 tcttacctgaactctggaat 100 normal tumort1 b g actctgg a at c c att c tg gt g cc ac 120 uvaggcaatcctg agg aag agg atgtg 170  1\\u00e2\\u00bb ggaatccattctggt normal aatcctgaggaaga ggaatcctggaaga ctggactctggaatcctgg aag agg atgtg g at ai  120  430    14 tumort8 fig 1 sequence analysis of aberrant pcr products of \\u00e3\\u00ffcateninfound in hccs a the normal sequence g was altered to a in tumor tl resulting in substitution of aac asn for gac asp b a 51bp interstitial deletion has occurred in tumor t8 with a thymine inserted at the breakpoint instead of adenine an identical fivenucleotide sequence aatcc occurs at both ends of the deletion aacgcactgcc3' set 5 5f 5'gccattacaactctccacaac3' and 5r 5'gatagcaccttcagcactctg3' set 6 6f 5'tcatactggctagtggtggac3' and 6r 5'ccaaggagaccttccatccc3' set 7 7f 5'agatgctgcaactaaacagg3' and 7r 5'gtgaaggcgaactgcattctg3' set 8 8f 5'ggtggtatagaggctcttgtg 3' and 8r 5'ctgaactagtcgtggaatggc3' set 9 9f 5'atcatgcacctttgcgtgag3' and 9r 5'ttcaatgggagaataaagcagc3' set 10 10f 5'atgttcacaaccgaattgttatc3' and 10r 5'gtcagctcaactgaaagccg3 and set 11 11f 5'tgttccgaatgtctgaggac3' and 11r 5'gttcagaacatacagctaaagg3' the pcr was carried out in a volume of 20 j1for 4 min at 94\\u00e2\\u00b0c for initial denaturing followed by 35 cycles of 94\\u00e2\\u00b0cfor 30 s 58\\u00e2\\u00b0cfor 30 s and 72\\u00e2\\u00b0cfor 30 s on the geneamp pcr system 9600 perkinelmer corp each 02 \\u00e2\\u00bfigof pcr product was diluted with formamide dye solution and electrophoresed in 5 polyacrylamide gels containing 10 glycerol at 4\\u00e2\\u00b0c after electrophoresis bands were visualized by means of fluorescent image analyzer fmbio ii multiview fluorescent image analyzer takara with staining by sybr green ii fmc bioproducts genomic pcr we performed pcr amplifications using genomic dna with a pair of primers cf 5'ccagcgtggacaatggctac3' and cr 5'tgagctcgagtcattgcatac3' corresponding to parts of dna sequences of exons 2 and 4 respectively in the same conditions described 2525 downloaded from cancerresaacrjournalsorg on may 26 2017 \\u00a9 1998 american association for cancer research  mutations of the \\u00e3\\u00ffcatenin gene in hccs table 1 summary nf \\u00e3\\u00ffctileninmutatinns amang 75 primary hccs tumorpoint itl linn ni t2tilt50 ta42t70t30t3it65 t67t9 ta99intenititial deletiont8affected codons32323334414545deletion of 17 umino acidsmutationgac asngac sp \\u00e2\\u00bbaac tyrtctasp > tac progga ser \\u00e2\\u00bbcct valaccgly ^gta alatctthr \\u00e2\\u00bbgcc protct ser \\u00e2\\u00bbcct phe5lbp ser \\u00e2\\u00bbttt deletion nucleotides t32 codons 3653 deletion of 50 amino acids codons 554 321372\\\" 195bp deletion 1on the basis of the sequence dala of genbank accession no x87838 above the pcr products were electrophoresed in a 15 agarose gel and visualized by staining with elhidium bromide cloning and sequence analyses aberrant pcr products detected by rtpcrsscp or genomic pcr were recovered from gels and subcloned into pbluescript ii sk \\u00e2\\u20ac\\u201dvector stratagenc the dna sequences of each clone were determined using an applied biosystems model 373 or 377 dna sequencer perkinelmer corp with t3 or t7 primers and a dye terminator cycle sequencing fs ready reaction kit applied biosystems all mutations were confirmed by repealed experiments and southern hybridization with oligonucleotides corresponding to mutated dna sequences western blot analysis for \\u00e3\\u00ffcatenin extraction of proteins from cancer tissues and following western blot analyses were performed as described previously 14 briefly 20 \\u00e2\\u00bf\\u00e2\\u00bfgof total protein extracted from each of the liver cancer tissues tl t2 t8 t9 and t30 were electrophoresed transferred to polyvinylidene ditluoridc membrane and visualized with anti\\u00e3\\u00ffcatenin monoclonal antibody c19220 transduction laboratories results and discussion a total of 75 primary hccs were screened for genetic alterations of the \\u00e3\\u00ffcateningene by means of the pcrsscp technique for an entire coding region using cdnas that were reversely transcribed from tumor mrnas and by pcr amplification of genomic dna of exons 24 that included phosphorylation target residues serinethreonine present in exon 3  1014 sequence analyses of aberrant pcr frag ments detected by sscp or genomic pcr amplification confirmed somatic alterations that were likely to activate \\u00e3\\u00ffcateninin 14 of the 75 cases for example in tumor tl we identified a missense substi tution of aac asn for gac asp fig 1a and in tumor t8 we detected an interstitial deletion of 51bp within exon 3 that would delete 17 amino acids from the gene product fig i\\u00e3\\u00ffthe 14 mutations detected in this study are summarized in table 1 and fig 2 among them 12 were point substitutions at codons 32 n  5 33 n  1 34 n  1 41 n  1 and 45 n  4 interstitial deletions which were detected in ~3 of sporadic colorectal cancers 14 were found in two of our hcc samples tumor t8 had a 51bp interstitial deletion of a part of exon 3 and tumor t32 was found to have a 195bp deletion corresponding to 5' portion of exon 3 and a part of intron 2 as shown in fig 2 an identical series of five nucleotides aatcc was present at each end of the deleted sequence in tumor t8 fig i\\u00e3\\u00ffas expected the rtpcr using rnas derived from those two tumors detected aberrant transcripts that were shorter than the normal transcript dna sequencing of these bands disclosed that the transcript in tumor t8 had lost 51 bases in exon 3 corresponding to the genomic interstitial deletion and that the transcript of t32 caused aberrant splicing resulting in a 150base deletion of 5' portion of exon 3 data not shown all in t32 or three in t8 of the four potential serinethreonine phosphorylation residues were involved in each deletion fig 2 because frozen materials were available for 5 of the 14 tumors in which we found \\u00e3\\u00ffcateninmutations we performed western analysis and confirmed accumulation of normalsize \\u00e3\\u00ffcateninfor ti t2 t9 and t30 or of a slightly smaller molecule corresponding to the predicted product in tumor t8 fig 3 amino acid substitutions involving serine or threonine residues in exon 3 of \\u00e3\\u00ffcateninwhich are essential phosphorylation sites for gsk3\\u00e3\\u00ffhave been reported in colorectal cancers and melanomas 1013 in our study of primary hccs in addition to mutations at those residues we demonstrated six missense substitutions at neighboring sites specifically at codons 32 aspartic acid and 34 glycine flanking the serine at codon 33 furthermore by western blot analyses we confirmed the accumula tion of \\u00e3\\u00ffcateninin tumors tl t2 and t30 in which amino acid substitutions had occurred at codons 32 and 34 others have reported mutations of \\u00e3\\u00ffcateninat codons 32 and 34 in chemically induced colon tumors in rats 20 these three residues dsg codons 3234 highly conserved among species are consensus motifs of the \\u00e3\\u00ffcatet1 n t8 92kd' 90kd fig 3 western bloi analysis of tumors ti and t8 and normal liver n accumulation of \\u00e3\\u00ffcaleninis detected at the normal mr 92000 position in tl in t8 a band of mr 90000 corresponds to the product that lacks 17 amino acids fig 2 schematic representation of 12 amino acid substitutions 5 a! codon 32 i each at codons 33 34 and 41 and 4 at codon 45 and two deleted regions i drawn to scale boldface type residues representing potential phosphorylation siles y  n p n pv ap ttt t r yldsgihsgatttapslsg intron 2444bpexon 3228bpt8 1 1| 4intron txon 3 200bp 51bp t32 i 195bp 2526 downloaded from cancerresaacrjournalsorg on may 26 2017 \\u00a9 1998 american association for cancer research  mutations of the \\u00e3\\u00ffcatenin gene in hccs nin and inhibitor of nuclear factor kb protein families which are supposed to be necessary for ubiquitindependent proteolysis 21 we assume that alterations at codon 32 or 34 may change the protein structure and thereby inhibit the phosphorylation of \\u00e3\\u00ffcateninor its recognition by the ubiquitindependent proteolysis system in a previous study 14 we investigated \\u00e3\\u00ffcateninmutations in human colorectal cancers and demonstrated a novel \\u00e3\\u00ffcateninactivating mechanism ie interstitial deletions involving exon 3 how ever unlike other groups 1113 we found no missense mutations at serinethreonine residues in this exon we have reported here a novel \\u00e3\\u00ffcateninactivating mechanism in human primary hccs represented by missense mutations at codons 32 and 34 which does not involve potential phosphorylation sites serine or threonine residues in exon 3 our results suggest that \\u00e3\\u00ffcateninaccumulation as a result of amino acid substitutions or interstitial deletions within or including parts of exon 3 can contribute to hepatocellular carcinogenesis different types of mutations in the same gene may reflect different etiologies of carcinogenesis in specific tissues which might reflect dietary or environmental differences or different genetic backgrounds however the frequent oncogenic activation of \\u00e3\\u00ffcateninin hccs reported here should contribute to a better understanding of hepatic carcinogenesis   abstract mutations of the \\u00e3\\u00ffcatenin gene ctnnbl have recently been impli cated in the initiation of some colon carcinomas and melanomas in these tumors \\u00e3\\u00ffcateninabnormally accumulates in the cell nuclei in an ongo ing immunohistochemical study of the cadherincatenin complex protein expression in ovarian carcinomas we observed \\u00e3\\u00ffcatenin in tumor cell nuclei in some cases this prompted us to study whether or not this abnormal immunostaining pattern was due to mutation in the \\u00e3\\u00ffcatenin gene itself this study examines \\u00e3\\u00ffcatenin immunohistochemical expres sion in 40 stage i and ii ovarian borderline tumors and carcinomas of the most common histol\\u00e3\\u00b3gica! types membrane expression was heterogeneous in all 40 cases however the cytoplasm and nucleus of five one borderline tumor and four carcinomas of the six endometrioid lesions contained \\u00e3\\u00ffcateninexpression pcr and sequencing analyses of a 200bp fragment of exon 3 of the ctnnbl gene encompassing the sequence for glycogen syn tlift iso kinase3\\u00e3\\u00ff phosphorylation were performed in 11 tumors heterozygous substitution mutations at codon 37 in two cases s37f and s37c and at codon 41 in one case t41a were found in three endometri oid lesions one borderline tumor and two carcinomas with abnormal \\u00e3\\u00ffcateninexpression three endometrioid carcinomas and five tumors of other histol\\u00e3\\u00b3gica! types analyzed showed normal dna sequences these results implicate \\u00e3\\u00ffcateningene mutations in ovarian malignant transfor mation with a characteristic phenotype endometrioid ovarian carcinoma introduction 3catenin the vertebrate homologue of armadillo protein in drosophila is a multifunctional protein involved in two apparently inde pendent processes cellcell adhesion and signal transduction transcriptional activation refs 13 the role of \\u00e3\\u00ffcateninin adhesion is well established in the adherens junctions of epithelial cells the cytoplasmic domain of ecadherin organizes a peripheral protein complex including acatenin \\u00e3\\u00ffcateninand ycatenin plakoglobin that is necessary for adhesion to occur the role of \\u00e3\\u00ffcateninin the linkage between acatenin and ecadherin in this complex is probably regulated by epidermal growth factormediated tyrosine phosphoryl ation of the \\u00e3\\u00ffcatenin4 recently it has been demonstrated that cytoplasmic \\u00e3\\u00ffcatenin participates in the transduction of winglesswnt signals and activates transcription by forming complexes with dnabinding proteins which are members of the tcf'lymphoid enhancer factor family 57 the free \\u00e3\\u00ffcateninlevel is low in normal cells because the protein is targeted for destruction by apc protein together with gsk3\\u00e3\\u00ff8 thus wt apc plays an essential role in clearance of unnecessary \\u00e3\\u00ffcateninfrom the cytoplasm 13 the presence of a winglesswnt signal in normal embryonic cells stabilizes \\u00e3\\u00ffcatenin which accumulates in the cytoplasm where it binds to tcflymphoid received 112097 accepted 21898 the costs of publication of this article were defrayed in part by the payment of page charges this article must therefore be hereby marked advertisement in accordance with 18 usc section 1734 solely to indicate this fact 1this work was supported in part by fondo de investigaci\\u00e3\\u00b3n sanitaria grant 950849 2 to whom requests for reprints should be addressed at departamento de anatom\\u00e3a patol\\u00e3\\u00b3gica hospital universilario la paz universidad aut\\u00e3\\u00b3noma de madrid paseo de la castellana 261 28046 madrid spain 3 the abbreviations used are tcf tcell factor apc adenomatous polyposis coli gsk33 glycogen synthetase kinase3\\u00e3\\u00ffwt wildtype enhancer factor and triggers gene expression 13 disruption of the apcmediated regulation of the \\u00e3\\u00ffcatenintcf pathway is critical for the development of colon cancer and melanoma in both types of neoplasms the mutations that inactivate apc increase cytoplasmic levels of free \\u00e3\\u00ffcateninwhich may then act as an oncoprotein through constitutive \\u00e3\\u00ffcatenintcfregulated transcription 9 10 in the same manner activating \\u00e3\\u00ffcateninmutations render this pathway insensiuve to the effect of wt apc 9 10 in this study on expression of cadherincatenin complex proteins in ovarian carcinomas we observed localization of \\u00e3\\u00ffcatenin in the nuclei of a subset of tumors and questioned whether this abnormal immunostaining pattern might be due to mutations in the \\u00e3\\u00ffcatenin gene ctnnbl itself our results indicate that \\u00e3\\u00ffcateningene muta tions are implicated in malignant ovarian transformation and that the resulting tumors have a characteristic phenotype endometrioid materials and methods samples this study was conducted on 40 primary epithelial ovarian tu mors diagnosed at the department of pathology of la paz hospital madrid spain we examined 10 stage i borderline ovarian tumors 4 serous tumors 5 mucinous tumors and 1 endometrioid tumor in addition 30 stage i and ii ovarian carcinomas were studied 8 clear cell carcinomas 7 serous carcinomas 7 mucinous carcinomas 5 endometrioid carcinomas and 3 mixed serousclear cell carcinomas histological immunohistochemical and dna studies were performed on formalinfixed paraffinembedded tissue samples immunohistochemistry immunostaining for \\u00e3\\u00ffcateninwas performed by the avidinbiotinalkaline phosphatase method as reported previously 11 except for the antibody and the performance of a heatinduced antigen retrieval step deparaffmized sections were immersed in 001 m sodium citrate buffer ph 60 and incubated in a pressure cooker for 3 min the mouse antihuman \\u00e3\\u00ffcateninmonoclonal antibody transduction laboratories lexington ky was applied overnight at a dilution of 1200 in negative controls the primary antibody was omitted or replaced by an irrelevant antibody dna analysis pcr and sequence analysis were performed on 11 tumors all 5 cases with nuclear \\u00e3\\u00ffcateninexpression and 6 tumors with only mem brane immunostaining 2 clear cell carcinomas 2 serous carcinomas 1 muci nous carcinoma and 1 endometrioid carcinoma normal tissue samples miometrium or fallopian tube from each of the 11 patients were also sub jected to pcr and sequencing dna was extracted from 10pim paraffin sections by proteinase k digestion without further purification five to 10 xl of the crude extract were amplified by pcr in a loo^il reaction containing 10 mm trishcl ph 83 50 min kc1 2 mm mgcl2 001 wv gelatin 200 pm dntp 05 jim each primer and 25 units of dinazyme taq polymerase finnzymes helsinki finland the primers used were 240f 5'atg gaa cca gac aga aaa gc and 439f 5'gct act tgt tct gag tga ag these amplified a 200bp fragment of exon 3 of the ctnnbl gene encompassing the sequence for gsk3\\u00e3\\u00ffphosphoryl ation 9 the pcr conditions consisted of 1 cycle at 95\\u00e2\\u00b0cfor 5 min 40 cycles at 95\\u00e2\\u00b0cfor 1 min 30 s 56\\u00e2\\u00b0cfor 1 min 30 s and 72\\u00e2\\u00b0cfor 2 min and a final extension cycle at 72\\u00e2\\u00b0cfor 5 min because most of the tumors were over 5 years old and extensive dna degradation is a common event in old paraffinembedded samples 40 pcr cycles were necessary to create enough amplifi cation product for subsequent restriction analysis and sequencing pcr was performed in a gene amp pcr system 2400 perkinelmer foster city ca reaction products were separated in 3 agarose or 5 polyacrylamide gels and stained in ethidium bromide for visualization pcr products were purified using centricon100 columns amicon bev 1344 downloaded from cancerresaacrjournalsorg on june 2 2017 \\u00a9 1998 american association for cancer research  bcatenin mutations in ovarian cancer erly ma treated with shrimp alkaline phosphatase and exonuclease i amersham buckinghamshire united kingdom and sequenced in a 373 auto mated sequencer applied biosystems foster city ca using the abi prism dye terminator cycle sequencing ready reaction kit applied biosystems the pcr primers were used as the primers for sequencing the two strands of the amplified products cases with abnormal dna sequences were subjected to dna extraction from additional tissue sections pcr and sequencing to confirm the presence of true mutations results immunohistochemistry showed that all 40 ovarian borderline tu mors and carcinomas of the common histopathological subtypes that were examined showed \\u00e3\\u00ffcateninexpression localized at the cellular membrane frequently with a heterogeneous pattern areas of intense and weak immunoreactivity were admixed in the same tumor fig 1 a b and d in addition 5 cases 1 endometrioid borderline tumor fig 14 and 4 endometrioid carcinomas fig l b and c also showed immunoexpression in the tumor cell nuclei in these cases nuclear immunostaining \\\\uis present in 25100 of neoplastic cell nuclei in some of these cells \\u00e3\\u00ffcateninwas also expressed in the cytoplasm fig 1 ac the only endometrioid carcinoma that did not show nuclear \\u00e3\\u00ffcatenin immunostaining had a growth pattern resembling a sexcord stromal tumor borderline tumors and carcino mas of histol\\u00e3\\u00b3gica! types \\u00e2\\u00bbilicithan endometrioid showed neither nuclear nor cytoplasmic \\u00e3\\u00ffcateninexpression sequence analysis of the k'r amplification products showed het erozygous substitution mutations at codon 37 of ctnnbi gene exon 3 in two cases fig 1 \\u00e2\\u00a3and  i and at codon 41 in one case fig 1g at codon 37 a tct to ttt change s37f and a tct to tgt change s37c were detected in one borderline endometrioid tumor and one endometrioid carcinoma respectively an acc to gcc change at codon 41 t41a was observed in another endometrioid carcinoma all three cases showed both nuclear and cytoplasmic \\u00e3\\u00ffcateninim munoexpression to further confirm sequencing data in the two first cases pcr products were digested with xmn\\\\ restriction endonuclefig 1 immunohistocemistry for \\u00e3\\u00ffcatenindem onstrates prominent nuclear and cytoplasmic but weak and heterogeneous membrane immuno staining in an endometrioid borderline tumor ai and an endometrioid carcinoma b and ci a note nuclear expression in areas of squamous metapla sia d a clear cell carcinoma showing intense membranous \\u00e3\\u00ffcateninexpression eg sequenc ing data for dna from ovarian tumors t and normal tissue from the same patient a7e a tct to tgt change at codon 37 s37c in an endo metrioid ovarian carcinoma f an aga to aaa change at codon 37 reverse strand s37f in an endometrioid borderline tumor g an acc to gcc change at codon 41 t41a in another endometri oid carcinoma h analysis with the xmnl restric tion endonuclease of the 200bp pcr amplification products from e digestion of the normal products lanes ni resulted in two fragments of 139 and 61 bp whereas heterozygous mutation at codon 37 eliminated the xmnl restriction site in one al\\u00e3ele and the mutant product continued to consist of 200 bp attcigg cca g a a t ctaccac n pbb322h\\u00e2\\u00bb\\u00e2\\u00abi t |200 bp |139 bp 161 bp downloaded from cancerresaacrjournalsorg on june 2 2017 \\u00a9 1998 american association for cancer research  bcatenin mutations in ovarian cancer ase because mutations at codon 37 had eliminated a xmnl restriction site digestion of the normal products resulted in two fragments of 139 and 61 bp whereas the mutant product continued to present a 200bp product fig itf discussion this study demonstrates the presence of ctnnbi mutations in some ovarian carcinomas for the first time in addition this genetic alteration is correlated with a definite phenotype endometrioid and is identified in the initiation stage borderline tumors and stage i carci nomas of this histological subtype these results are in accordance with the heterogeneity of genetic disorders previously observed in different types of epithelial ovarian tumors although in most cases epithelial tumors of the ovary seem to have a common origin in the surface epithelium lining the ovary different pathways and genetic alterations have been implicated in the development of cystadenomas borderline tumors and carcinomas of different histological subtypes and stages of progression for example loss of heterozygosity and a complete loss of chromosome 17 is characteristic of the advanced stages of serous carcinomas whereas ras mutation participates in the development of mucinous tumors but not in other histological types 12 to date no specific genetic alterations had been described in endometrioid tumors these data strongly suggest that \\u00e3\\u00ffcateninmay function as an oncogene in the initiation of some ovarian endometri oid carcinomas in up to 50 of the cases in our series as has previously been reported in colon carcinomas and melanomas 9 10 13 14 in colon carcinomas ctnnbi mutations have been found in 2 of primary tumors in which only exon 3 was analyzed 14 and in up to 26 of the cell lines derived from human colorectal cancers in which the entire coding sequence was examined 13 future studies should establish the frequency of \\u00e3\\u00ffcateningene mutations in the different types and histological subtypes of human malignant tumors in this sense candidus et al 15 have not found evidence for mutation in this gene in gastric diffuse and intestinal subtypes and breast ductal and lobular subtypes carcinomas although their sample is small \\u00e3\\u00ffcateningene mutations in ovarian carcinomas s37c s37f and t41a are similar to those previously reported in human colon car cinomas 9 13 14 and human melanoma cell lines 10 mutations at s33 t41 and s45 have been reported in colon carcinomas whereas point mutations at s37 and s45 have been detected in melanoma cell lines these mutations in ctnnbi gene exon 3 affect the sequence for gsk3\\u00e3\\u00ffphosphorylation mainly for those amino acids implicated in the downregulation of \\u00e3\\u00ffcatenin through phosphorylation by this serinethreonine kinase 9 the mutations probably render a fraction of cellular \\u00e3\\u00ffcatenin insensitive to apcmediated downregulation and are responsible for the upregulation of cytoplasmic \\u00e3\\u00ffcateninand its distribution in the nuclei of tumor cells as was immunohistochemically detected in this series it is of interest to note that two of the colon cancers reported by kitaeva el al  14 and two colon cancer cell lines hct 116 and sw48 that had \\u00e3\\u00ffcateningene mutations also showed a replication errorpositive phenotype suggesting that the activating mutations in \\u00e3\\u00ffcateninthat occur in tumors may be asso ciated with an underlying defect in mismatch repair 14 we did not find \\u00e3\\u00ffcatenin gene mutations in two of the five endometrioid carcinomas with nuclear and cytoplasmic \\u00e3\\u00ffcatenin expression however our dna analysis was very limited and genetic alterations other than mutations in the sequence for gsk3\\u00e3\\u00ffphos phorylation have been described by others 10 13 in addition increased \\u00e3\\u00ffcatenincytoplasmic and nuclear levels can also be ob served in cases of apc inactivation apc contains \\u00e3\\u00ffcateninbinding and degradation sites as well as binding sites for gsk3\\u00e3\\u00ffin the steady state of normal cells an equilibrium exists between apc and \\u00e3\\u00ffcateninbecause one of the most important tumor suppressor activ ities of apc is the negative regulation of the cytoplasmic levels of \\u00e3\\u00ffcatenininomata et al 16 have reported alteration of \\u00e3\\u00ffcatenin expression in colonie epithelial cells of familial apc patients these authors observed that \\u00e3\\u00ffcateninwas immunostained at the membranes of the cellcell borders in normal epithelial cells whereas the nuclei and cytoplasms stained intensely in addition to membranes in both adenomas and cancer cells in these patients apc protein was de tected and shown to have lost its central region including a known \\u00e3\\u00ffcateninbinding domain nuclear and cytoplasmic accumulation of \\u00e3\\u00ffcateninhas also been reported in adenomatous polyps from patients without familial apc 17 moreover \\u00e3\\u00ffcateninprotein has also been observed in sporadic aggressive fibromatosis associated to somatic apc mutations 18 in relation to apc in ovarian carcinomas apc expression has been observed in the surface epithelium of the ovary 19 and loss of heterozygosity at the apc locus occurs with a frequency of 50 20 however this seems to be a late event in tumor progression and does not correlate with mutation in the apc gene 20 this and previous reports suggest that both the absence of wt apc protein and ctnnbi mutations affect the subcellular localization and expression levels of \\u00e3\\u00ffcateninin human tumors in accordance with the proposed model of an apc\\u00e3\\u00ffcatenintcfregulated transcription pathway however upregulation or nuclear distribution of \\u00e3\\u00ffcatenin may be a common feature of tumorigenesis that could hypothetically be achieved through mutation not only in the apc or \\u00e3\\u00ffcateningenes but also by alterations in other genes that function in this pathway for example it has been reported that the wntl oncogene promotes the accumulation of \\u00e3\\u00ffand ycatenin 21  abstract inactivation of the adenomatous polyposis coli apc tumor suppressor gene initiates colorectal neoplasia one of the biochemical activities associated with the apc protein is downregulation of transcriptional activation mediated by \\u03b2catenin and t cell transcription factor 4 tcf4 the protein products of mutant apc genes present in colorectal tumors were found to be defective in this activity furthermore colorectal tumors with intact apc genes were found to contain activating mutations of \\u03b2catenin that altered functionally significant phosphorylation sites these results indicate that regulation of \\u03b2catenin is critical to apc's tumor suppressive effect and that this regulation can be circumvented by mutations in either apc or \\u03b2catenin mutations of the apc gene are the most common diseasecausing genetic events in humans about 50 of the population will develop colorectal polyps initiated by such mutations during a normal lifespan 1 individuals who inherit apc mutations develop thousands of colorectal tumors consistent with the tumor suppressor or \\u201cgatekeeping\\u201d role of apc protein in colorectal tumorigenesis 2 3 apc homodimerizes through its nh2terminus 4 and interacts with at least six other proteins \\u03b2catenin 5 \\u03b3catenin plakoglobin 6 tubulin 7 eb1 8 hdlg a homolog of the drosophila discs large tumor suppressor protein 9 and glycogen synthase kinase3\\u03b2 gsk3\\u03b2 10 a mammalian homolog of zw3 kinase whether any of these interacting proteins communicate apc growthcontrolling signals is unknown here we used a genetic approach to investigate the role of \\u03b2catenin in apc's tumor suppressor function  although \\u03b2catenin was originally discovered as a cadherinbinding protein it has recently been shown to function as a transcriptional activator when complexed with members of the tcf family of dna binding proteins 11 one family member htcf4 is expressed in normal and neoplastic colorectal epithelium and wildtype wt apc can suppress signaling by the \\u03b2catenintcf complex 12 if this inhibitory activity is critical for apc's tumor suppressor function then mutant apc proteins should be defective in this activity  to evaluate this hypothesis we tested four apc mutants fig 1a for their ability to inhibit \\u03b2catenintcfregulated transcription crt in transfection assays the first mutant apc331\\u03b4 represents a type of mutation commonly found in the germ line of familial adenomatous polyposis patients as well as in sporadic tumors 2 the apc331\\u03b4 protein is truncated at codon 331 nh2terminal to the three 15amino acid aa \\u03b2cateninbinding repeats between codons 1020 and 1169 the second mutant apc1309\\u03b4 is the most common germline apc mutation 2 a 5base pair bp deletion that produces a frameshift at codon 1309 and truncation of the protein the apc1309\\u03b4 protein retains the 15aa \\u03b2cateninbinding repeats but lacks the seven 20aa repeats between codons 1323 and 2075 that have been implicated in binding and degradation of \\u03b2catenin 5 the third mutant apc1941\\u03b4 represents one of the most distal somatic mutations observed in colorectal tumors 13 the apc1941\\u03b4 protein is truncated at codon 1941 and therefore contains the 15aa repeats and all but the last two 20aa repeats finally apc2644\\u03b4 represents a germline mutation resulting from a 4bp deletion in codon 2644 patients with this type of unusual coohterminal mutation develop few polyps attenuated polyposis but have pronounced extracolonic disease particularly desmoid tumors 14    download figure  open in new tab  download powerpoint   download figure  open in new tab  download powerpoint fig 1 effects of apc mutations on crt a schematic of wildtype wt and mutant apc apc is a 2843aa protein 23 that contains armadillo arm repeats in the nh2terminus 24 15 and 20aa \\u03b2cateninbinding repeats in the central region 5 6 and a basic region in the coohterminus 23 the coohterminus also contains a txv sequence which mediates dlg binding 9 b effects of wt and mutant apc on crt sw480 cells containing endogenous mutant apc were transfected with the apc expression vectors shown in a and crt was measured 25 cells were transfected with increasing amounts of wt apc 0 015 and 05 \\u03bcg indicated by the black wedge or 05 \\u03bcg of mutant apc crt reporter activities are expressed relative to assays containing no wt apc and are the means of three replicates error bars represent standard deviations  each of the apc mutants was cotransfected with a crt reporter into the sw480 colorectal cancer cell line sw480 cells have truncated apc and constitutively active crt which can be suppressed by exogenous wt apc 12 although all four mutants produced comparable amounts of apc protein after transfection 15 they varied in their crt inhibitory activity the three mutants found in patients with typical polyposis or cancer were markedly deficient in inhibition of crt fig 1b the reduced activity of apc1309\\u03b4 and apc1941\\u03b4 suggests that \\u03b2catenin binding is not sufficient for apcmediated inhibition of crt and that the complete set of 20aa repeats is required the inhibitory activity of the apc2644\\u03b4 mutant associated with attenuated polyposis was comparable with that of wt apc fig 1b suggesting that the dlgbinding domain at the coohterminus of apc is not required for downregulation of crt  if apc's inhibition of crt is critical to suppression of colorectal tumorigenesis cancers with wt apc must escape this inhibition through alternative mechanisms to investigate this possibility we evaluated crt in two colorectal tumor cell lines hct116 and sw48 that express fulllength apc fig 2a both hct116 and sw48 displayed constitutively active crt and in contrast to cell lines with truncated apc dld1 and sw480 this activity was not inhibited by exogenous wt apc figs 1b and 2b other noncolorectal cancer cell lines expressing wt apc do not display constitutive crt activity 12    download figure  open in new tab  download powerpoint   download figure  open in new tab  download powerpoint fig 2 evaluation of crt in colorectal cancer cell lines with wt apc a immunoblot of endogenous apc in the sw480 dld1 hct116 sw48 and 293 cell lines developed with apc monoclonal antibody fe9 26 b effects of exogenous wt apc on crt in cell lines with endogenous mutated or wt apc cells were transfected with increasing amounts 0 015 and 05 \\u03bcg for dld1 and sw48 0 05 and 5 \\u03bcg for hct116 of wt apc or apc1309\\u03b4 mutant 05 \\u03bcg for dld1 and sw48 5 \\u03bcg for hct116 and crt was assessed as in fig 1 crt reporter activities are expressed relative to activity in extracts without exogenous apc and are the means of three replicates error bars represent standard deviations  these transfection results suggested that the constitutive crt in hct116 and sw48 might be due to an altered downstream component of the apc tumor suppressor pathway we therefore evaluated the status of a likely candidate \\u03b2catenin in the same four lines all four lines expressed similar amounts of apparently intact \\u03b2catenin as assessed by immunoblots fig 3a however sequence analysis revealed that both hct116 and sw48 harbored mutations in the \\u03b2catenin gene ctnnb1 fig 3b hct116 had a 3bp deletion that removed one amino acid ser45 and sw48 had a ctoa missense mutation that changed ser33 to tyr analysis of paraffinembedded archival tissue from the hct116 patient confirmed the somatic nature of this mutation and its presence in the primary tumor before culture 16 both mutations affect serines that have been implicated in the downregulation of \\u03b2catenin through phosphorylation by the gsk3\\u03b2 kinase in xenopus embryos fig 3c 17 18    download figure  open in new tab  download powerpoint   download figure  open in new tab  download powerpoint   download figure  open in new tab  download powerpoint fig 3 evaluation of \\u03b2catenin in colorectal cancer cell lines with wt apc a immunoblot of the cell lines used in this study developed with \\u03b2catenin monoclonal antibody c19220 transduction laboratories lexington kentucky 22 b sequence of ctnnb1 in hct116 and sw48 27 the left panel nucleotides 121 to 143 from hct116 reveals the presence of a deletion in addition to the wt sequence the middle panel antisense strand of nucleotides 156 to 113 of the wt and deleted alleles of hct116 reveals the 3bp deletion \\u03b4tct that removed codon 45 in half the clones the right panel nucleotides 80 to 113 from sw48 reveals a ctoa transition affecting codon 33 tct to tat c schematic of \\u03b2catenin illustrating the armadillo repeats 24 and negative regulatory domain the residues in larger type fit the consensus sequence for gsk3\\u03b2 phosphorylation 20 and those in bold have been demonstrated to affect downregulation of \\u03b2catenin through gsk3\\u03b2 phosphorylation in xenopus embryos 17 the five mutations found in human colon cancers are indicated at the top singleletter abbreviations for the amino acid residues are as follows a ala c cys d asp e glu f phe g gly h his i ile k lys l leu m met n asn p pro q gln r arg s ser t thr v val w trp and y tyr  to test the generality of this mutational mechanism we evaluated five primary colorectal cancers in which sequencing of the entire coding region of apc had revealed no mutations 13 three of these five tumors were found to contain ctnnb1 mutations s45f s45f and t44a that altered potential gsk3\\u03b2 phosphorylation sites 19 fig 3c each mutation was somatic and appeared to affect only one of the two ctnnb1 alleles  because the \\u03b2catenin mutations were heterozygous we hypothesized that the mutations might exert a dominant effect rendering a fraction of cellular \\u03b2catenin insensitive to apcmediated downregulation to test this notion we performed gel shift analyses with nuclear extracts from untransfected hct116 cells in contrast to noncolorectal cancer cell lines with intact apc hct116 cells contained a \\u03b2catenintcf complex that gelshifted an optimized tcfbinding oligonucleotide and this complex supershifted with antibody to \\u03b2catenin fig 4a we also constructed \\u03b2catenin expression vectors and compared the biologic activity of the mutant \\u03b2catenin from hct116 \\u03b2cat \\u03b445 and sw48 \\u03b2cat s33y with that of their wt counterpart for these experiments we used the 293 kidney epithelial cell line because it is highly transfectable exhibits low endogenous crt and contains a large amount of endogenous apc fig 2a in the presence of endogenous apc both mutant \\u03b2catenins were at least six times as active as the wt protein and this activity was inhibited by dominantnegative htcf4 fig 4b    download figure  open in new tab  download powerpoint   download figure  open in new tab  download powerpoint fig 4 functional evaluation of \\u03b2catenin mutants a constitutive nuclear complex of \\u03b2catenin and tcf in hct116 cells the presence of nuclear \\u03b2catenintcf complexes was assessed by gel shift assays 12 lanes 1 to 3 the optimal tcf retardation probe shifted with nuclear extract from hct116 cells when no antibody was added lane 1 or with addition of antibody to \\u03b2catenin 025 \\u03bcg lane 2 or an irrelevant antibody 025 \\u03bcg lane 3 lane 4 mutant tcf retardation probe shifted with nuclear extract from hct116 cells ns nonspecific shifting seen with the mutant probe b effects of the \\u03b2catenin mutations on crt human 293 cells were transfected with wt or mutant \\u03b445 s33y \\u03b2catenin and crt was assessed 28 crt reporter activities are expressed relative to wt \\u03b2catenin and are the means of three replicates error bars represent standard deviations dn tcf refers to dominantnegative htcf4  together these results indicate that disruption of apcmediated regulation of crt is critical for colorectal tumorigenesis this is most commonly achieved by recessive inactivating mutations of both apc alleles but as shown here can also be achieved by dominant mutations of ctnnb1 that render crt insensitive to the effects of wt apc our results suggest that apc inhibition of crt requires phosphorylation of \\u03b2catenin at multiple sites these potential phosphorylation sites are consistent with the known specificity of gsk3\\u03b2 20 a serinethreonine kinase that negatively regulates \\u03b2catenin in xenopus and drosophila cells 17 and that interacts with apc and \\u03b2catenin in mammalian cells 10 our results also suggest a functional basis for the occasional ctnnb1 mutations observed in other tumor types 21 and illustrate how a critical pathway in human disease can be illuminated by the discovery of mutations in different components of the pathway the next step in understanding apc function will be the identification of the genes that are activated by htcf4\\u03b2catenin complexes and inhibited by wt apc these genes are likely to be related to apc's ability to induce apoptosis in colorectal cancer cells 22\",\n          \"EGFR E746_T751delinsVA abstract  malignant pleural effusions mpes are often observed in lung cancer especially adenocarcinoma epidermal growth factor receptor egfr gene mutations are usually detected in lung adenocarcinoma the purpose of the present study was to investigate the egfr mutation rate in mpes of lung adenocarcinoma  between june 2005 and december 2006 136 mpes from lung adenocarcinoma were collected for egfr mutation detection in addition between april 2001 and november 2004 91 surgically resected specimens of lung adenocarcinoma from patients without mpes were assessed for egfr mutation  the egfr mutation rate was significantly higher in the patients with mpes than in the patients without 684 versus 505 the egfr mutation rate in patients with mpes was not associated with sex smoking history age or cancer stage by multivariate analysis an age of <65\\u2005yrs never smoking eastern cooperative oncology group performance status 01 and egfr mutation were significantly associated with a longer overall survival for lung adenocarcinoma patients with mpes  the patients with malignant pleural effusions related to lung adenocarcinoma had a higher epidermal growth factor receptor gene mutation rate than the patients from whom surgically resected specimens were taken epidermal growth factor receptor tyrosine kinase inhibitors may be the treatment of choice for lung adenocarcinoma with malignant pleural effusions in east asia  epidermal growth factor receptorlung cancermutationpleural effusion annually there are \\u223c12\\u2005million new cases of lung cancer globally with \\u223c11\\u2005million patients dying due to the disease 1 platinumbased chemotherapy has a partial response in only 30 of patients with advanced nonsmall cell lung cancer nsclc 2  epidermal growth factor receptor egfr tyrosine kinase inhibitors tkis are used in the treatment of nsclc egfr tkis have a higher response in specific subgroups including females never smokers adenocarcinoma histology and east asians 35 these subgroups also have higher egfr mutation rates 6 a better response to egfr tkis and prolonged survival are related to egfr mutations including inframe deletions and point mutations 79  malignant pleural effusions mpes are often observed in lung cancer especially adenocarcinoma because it is a tumour that grows in the periphery of the lung and easily invades the pleural cavity 10 mpes also indicate an advanced stage of disease or disease progression in total \\u223c15 of patients have pleural effusion at the initial diagnosis of lung cancer 10 thoracentesis is necessary for diagnosis and treatment mpes can develop as a direct consequence of cancer cell dissemination into the pleural space however the exact mechanisms are not fully understood 11 mpes may result from the combination of both decreased capacity of the lymphatics to remove fluid and increased pleural fluid formation 11 the latter mechanism may be related to an increase in the vascular permeability and vascular endothelial growth factor vegf plays an important role in increasing the permeability of the vasculature 12 median vegf levels in pleural fluid are higher in mpes than nonmalignant pleural effusions 13 in addition to vegf other molecular factors or tumour markers can also be detected in mpes hsieh et al 14 demonstrated that pigment epitheliumderived factor and fibrinogen precursors are expressed at lower levels in mpes than in transudates dai et al 15 showed that p53 and kras gene mutation patterns are effective markers for the detection of recurrent lung carcinoma in cytological specimens of pleural effusions  mpe is collected via very easy techniques in contrast to other invasive techniques such as biopsy and surgery which are necessary to obtain tissue of other metastatic sites mutations of egfr can also be detected in pleural effusion samples 16 egfr mutation screening in pleural effusion is useful for the prediction of the clinical outcome of lung cancer patients treated with gefitinib 16 17 in order to understand how frequently egfr mutations occur in mpes of lung adenocarcinoma the present authors examined the sequences of exons 1821 of egfrs in mpes of lung adenocarcinoma the mutation rate of mpes and the mutation rate of surgically resected specimens were then compared in addition clinical information was collected to analyse the prognostic factors of overall survival in the patients with lung adenocarcinoma with mpes  materials and methods  patients and tissue procurement  between june 2005 and december 2006 383 consecutive samples of pleural effusion were collected from 273 patients who had received thoracentesis in the chest ultrasonography examination room of the national taiwan university hospital taipei taiwan this study was approved by the institutional review board of the national taiwan university hospital and all patients had signed an informed consent form before the thoracentesis was performed the primary malignant tumours or mpes were confirmed by pathology or cytology reports of the 273 patients 164 had lung adenocarcinoma and the other 109 patients had malignancy other than lung adenocarcinoma or nonmalignancyrelated pleural effusions among the 164 patients with lung adenocarcinoma the cytology of pleural effusions was negative in 28 patients therefore 136 mpes of lung adenocarcinoma were assessed for egfr mutations these 136 mpes were obtained before treatment with gefitinib or erlotinib if the patients received egfr tki therapy for comparison 91 surgically resected specimens of lung adenocarcinoma archived from april 2001 to november 2004 were also retrived informed consent about the use of these specimens for future molecular studies with approval of the institutional review board was obtained before surgery the retrospective use of archival tissue for egfr gene analysis was approved by the institutional review board of the national taiwan university hospital the clinical information of all the patients was recorded including age sex smoking history lung cancer stage performance status eastern cooperative oncology group performance status ecog ps treatment regimens and maximal response patients who had smoked <100 cigarettes in their lifetime were categorised as never smokers those who smoked cigarettes within 1\\u2005yr of the diagnosis were categorised as current smokers the others were categorised as former smokers  collection of pleural effusion fluid and surgically resected specimens  the pleural effusion fluid was collected into heparinised tubes a 1ml sample of the fluid was centrifuged at 250\\u00d7g for 10\\u2005min at room temperature and the cell pellet frozen specimens of lung adenocarcinoma tissue obtained at surgery were immediately snapfrozen in liquid nitrogen and stored for later use total rna was isolated using trireagent molecular research center inc cincinnati oh usa and stored at 80\\u00b0c until rna extraction was performed rna was extracted from cell lysates with a qiamp rna mini kit qiagen hilden germany according to the protocol in the manufacturer's instructions the rna obtained was eluted in 50\\u2005\\u03bcl of sterile bidistilled buffer the amounts of rna extracted were measured with spectrophotometry  pcr amplification and direct sequencing  the four exons exons 1821 that code for the tyrosine kinase domain of the egfr gene were amplified with a forward primer 5\\u2032agcttgtggagcctcttacacc3\\u2032 and reverse primer 5\\u2032taaaattgattccaatgccatcc3\\u2032 as reported in a prior study 8 the rtpcr was performed previously as reported 8 using a qiagen onestep rtpcr kit qiagen the rtpcr conditions were as previously described 8 rtpcr amplicons were purified and sequenced using the big dye terminator sequencing kit applied biosystems foster city ca usa according to the manufacturer's instructions sequencing products underwent electrophoresis on an abi prism 3100 applied biosystems both the forward and reverse sequences obtained were analysed by basic local alignment search tool against the cdna sequence of the egfr gene  accession number nm005228 and chromatograms were examined manually pcr amplicons were sequenced in both sense and antisense directions egfr mutations detected in the initial round of sequencing were confirmed by subsequent rounds of independent rtpcr and sequencing reactions only specimens in which a mutation was identified in both rounds were recorded as mutation positive mutations were also checked against the single nucleotide polymorphism database  antitumour therapy and response evaluation of the patients with mpes of lung adenocarcinoma  the antitumour response of the patients was evaluated by chest radiography every 24\\u2005weeks and by computed tomography of the disease sites every 812\\u2005weeks after treatment this is routine practice in the present authors institution for the measurable solid tumours the unidimension method was used according to the response evaluation criteria in solid tumours recist guidelines 18 partial response and stable disease were confirmed by a sustained 4week followup overall survival after antitumour therapy was calculated from the date of initiation of treatment to the date of death last followup or the final followup day of the study which was september 1 2007  toxicity of egfr tkis was also recorded and was graded according to the national cancer institute common toxicity criteria version 30  statistical analysis  all categorical variables were analysed with pearsons chisquared test except where a small size required the use of fisher's exact test overall survival was compared by the logrank test and multivariate analysis for overall survival was performed using the cox linear regression method twosided pvalues <005 were considered significant  results  clinical characteristics and egfr mutations of the surgically resected lung adenocarcinoma patients  in total 91 surgically resected specimens were assessed for the egfr mutation table\\u20051 the median range age of the 91 patients was 634 375854\\u2005yrs three stageiiib patients had satellite tumour nodules within the same lobe as the primary tumour and they received lobectomies two stageiv patients who both had two lung lesions over the right middle lobe and right lower lobe without other distant metastasis lesions received bilobectomies  view inlineview popup table 1\\u2014 patients' characteristics and epidermal growth factor receptor egfr mutation status of the surgically resected specimens in total 46 out of 91 surgically resected lung adenocarcinoma had egfr mutations the egfr mutation rate was 505 the egfr mutations included 19 l858r 23 deletions in exon 19 and four other types 767769 duplication asv p771h773insynp+h773y k860i+l861q and l861q egfr mutation rates of the surgically resected lung adenocarcinoma were not significantly different by sex 49 of males versus 525 of females p\\u200a\\u200a0742 age p\\u200a\\u200a0746 or smoking status 547 of never smokers versus 447 of former or current smokers p\\u200a\\u200a0348 table\\u20051\\u21d1  of the 91 patients 14 took egfr tkis after tumour recurrence of these seven patients had egfr mutations and the other seven patients were wild type five patients with a positive egfr mutation and three patients of wild type responded to egfr tki treatment however the patient numbers were too small for further analysis  clinical characteristics and egfr mutations of the lung adenocarcinoma patients with mpe  mpes were assessed for egfr mutations in 136 patients table\\u20052 the median range age of the 136 patients was 661 287906\\u2005yrs two 15 patients did not complete staging workup of lung cancer and they were lost to followup  view inlineview popup table 2\\u2014 patient characteristics and epidermal growth factor receptoregfr mutation status of malignant pleural effusions in total 93 out of 136 patients with mpes of lung adenocarcinoma had positive egfr mutations the mutation rate was 684 the egfr mutations included 50 l858r 32 deletions in exon 19 and 11 other types the 11 other types included three g719a one g719a+s720f one g719a+s768i one e746v+l747p one l747p one 767769 duplication asv one 768770 duplication svd one l861q and one r776h+l861q egfr mutation rates of the adenocarcinoma patients with mpes were not significantly different by sex 642 of males versus 711 of females p\\u200a\\u200a0396 age p\\u200a\\u200a0734 or smoking status 705 of never smokers versus 634 of former or current smokers p\\u200a\\u200a0413 table\\u20052\\u21d1  of the 136 patients with mpes 111 patients initially had pleural effusions at the diagnosis of lung adenocarcinoma of the 111 patients 77 685 had egfr mutations the mutation rate of initial pleural effusions related to lung adenocarcinoma was similar to the total 136 mpes 685 versus 684  the patients with mpes of lung adenocarcinoma had a higher egfr mutation rate than the patients with surgically resected lung adenocarcinoma 684 versus 505 p\\u200a\\u200a0007 table\\u20053 interestingly the mutation rates of deletion in exon 19 in mpes and surgically resected specimens were not significantly different 235 versus 253 however the patients with mpes of lung adenocarcinoma had a higher l858r mutation rate than the patients with surgically resected lung adenocarcinoma 368 versus 209 p\\u200a\\u200a0011 table\\u20053  view inlineview popup table 3\\u2014 the difference in epidermal growth factor receptoregfr mutations between adenocarcinoma patients with malignant pleural effusions and surgically resected specimens from other adenocarcinoma patients response of adenocarcinoma with mpes treated with egfr tki  of the 136 patients with mpes of lung adenocarcinoma 71 had received egfr tki treatment gefitinib 250\\u2005mg\\u00b7day\\u22121 or erlotinib 150\\u2005mg\\u00b7day\\u22121 two patients were lost to followup before treatment and the other 63 patients received other antitumour therapy of the 71 patients treated with egfr tki 38 had partial response those included one wild type 23 l858r 11 deletions in exon 19 one g719a one l861q and one combined egfr mutation g719a+s720f table\\u20054 for the overall survival analysis of the 136 patients with lung adenocarcinoma and mpes the median overall survival was longer for patients with egfr mutations than for patients with wildtype egfr median 214\\u2005months 95 confidence interval ci 179249\\u2005months versus 115\\u2005months 95 ci 74156\\u2005months p\\u200a\\u200a0005 by logrank test multivariate analysis was performed by the cox regression model for the potential prognostic factors including age smoking status ecog ps egfr mutation table\\u20055 lung adenocarcinoma patients with mpes who were aged <65\\u2005yrs p\\u200a\\u200a0011 who never smoked p\\u200a\\u200a0027 with ecog ps 01 p<0001 and egfr mutation p\\u200a\\u200a0001 were found to be associated with a longer overall survival  view inlineview popup table 4\\u2014 epidermal growth factor receptoregfr mutation status and response of egfr tyrosine kinase inhibitor in patients of malignant pleural effusions view inlineview popup table 5\\u2014 multivariate analysis of prognostic factors for overall survival of the 136 adenocarcinoma patients with malignant pleural effusions the toxicity of egfr tkis were distributed from grade 1 to grade 3 and no patients had grade 4 toxicity the most common toxicity included diarrhoea 24 skin rash 38 acne 31 dry skin 22 pruritus 22 and anorexia 20 grade 3 events included one diarrhea 1 one skin rash 1 and one acne 1 neither grade 4 toxicity nor interstitial pneumonitis was recorded  discussion  in the present study patients with lung adenocarcinoma with mpes had a higher egfr mutation rate than patients with surgically resected adenocarcinoma the egfr mutation rate in mpes of lung adenocarcinoma was \\u223c70 regardless of the patients' sex smoking status or age  mpes are a common clinical problem for patients with lung adenocarcinoma thoracentesis may not only relieve dyspnoea 19 but is also an easy way to collect malignant cells from pleural effusion for molecular studies in contrast to other invasive techniques used to biopsy the primary tumour or its metastases in clinical practice egfr mutations could be detected in mpes and are useful for the prediction of response to gefitinib as shown in the present study and others 16 17  the present study revealed a 684 egfr mutation rate in lung adenocarcinoma with mpes which is higher than in prior published research soh et al 16 reported an egfr mutation rate of 245 13 out of 53 in mpes related to lung adenocarcinoma however only deletions in exon 19 and l858r were analysed 16 kimura et al 20 demonstrated a 282 11 out of 39 egfr mutation rate and considered that the lower frequency of egfr mutation was related to enrolling patients with falsenegative results kimura et al 21 showed a 13 3 out of 23 egfr mutation rate by direct sequence of egfr mutations in mpes related to lung adenocarcinoma in another study with 91 1 out of 11 in females and 10 1 out of 10 in never smokers it should be noted that more current or former smoking patients were enrolled into the kimura et al 21 study the egfr mutation rate of mpes ranged 91684 this variability probably reflects methodology selection of patients and geographic differences besides the pleural effusion samples obtained in the present study were all positive for malignant cells by cytological examination and this increases the detection rate of egfr mutations  breast and colon cancers have been established as the models for a multistep oncogenesis process during which the accumulation of genetic mutations results in the development of an invasive phenotype 22 23 the egfr mutation might be involved in the pathogenesis of lung adenocarcinoma egfr mutations were more frequent in patients with advanced stage disease mpe when compared with more early stages surgically resectable the difference of the egfr mutation rate between the mpes and the surgically resected lung adenocarcinoma might reflect the process of carcinogenesis tang et al 24 found identical egfr mutations detected in the normal respiratory epithelium in 43 of patients with egfr mutant adenocarcinomas tomizawa et al 25 also demonstrated that egfr mutations were more frequently observed in the advanced stage of nsclc matsumoto et al 26 reported frequent egfr mutations in brain metastasis of lung adenocarcinoma in a case series of 19 patients matsumoto et al 26 study showed a 63 egfr mutation rate which is similar to the present study in mpes the higher incidence of egfr mutations in mpes and brain metastasis than in surgically resected lung tumours implies that egfr mutation is more frequent in patients with advanced stages when compared with earlier stages further studies are necessary to address this issue  the incidence of egfr mutations in nsclc ranges from 10 to >50 17 2731 egfr mutations of nsclc are more frequent in females nonsmokers and in asian countries 6 25 geographic ethnic differences and methodology in detection of egfr mutations may result in variability the egfr mutation rate in surgically resected adenocarcinoma was 36 in korea 42556 in japan and 55 in taiwan 27 31 32 reports from korea and japan show that the egfr mutation rates of lung adenocarcinomas are higher in females females versus males 531763 versus 2236 and never smokers never smokers versus smokers 57483 versus 2132 2730 32 33 however huang et al 31 showed that the egfr mutation rate in resected lung adenocarcinoma was not significantly associated with sex females versus males 545 versus 555 and smoking history in taiwan the present study showed that the egfr mutation rate was 505 in surgically resected specimens of lung adenocarcinoma and the difference of egfr mutations between sexes was also not significant these were consistent with another report from taiwan 31 this implies that the asian population is not so homogeneous with regard to the mutation rate of egfr and as regards the influence of sex and smoking on the mutation rate  among the egfr tkitreated patients the present study showed several different egfr mutation types other than l858r and deletion in exon 19 the various egfr mutations did not all have good response to egfr tkis in particular t790m was found to confer primary and acquired resistance to egfr tki 34 the functional values of these different egfr mutations are still unclear  there was one limitation of the present study that the egfr copy numbers in the cancer cells of mpes were not examined egfr amplification is also an important predictive factor for nsclc patients treated with egfr tkis 35 36 although studies have shown that the egfr mutation rather than egfr copy numbers was a determinant of favourable clinical outcomes in gefitinibtreated patients with nsclc 3739 another interesting issue is that further studies are necessary to elucidate whether egfr gene amplification is also a progressive event like the egfr mutation as shown in the present study  there have been studies evaluating the high level of vegf in mpe 13 40 41 and vegf may also have a potential role in the formation of mpe 13 a case report has demonstrated that bevacizumab is active in malignant effusion 42 however treating malignant effusion with bevacizumab might require higher doses than simply treating the underlying cancer 42 therefore an antiangiogenic therapy for the treatment of mpe may play a important role for future studies  in the present study 54 of patients with lung adenocarcinoma with mpes responded to egfr tki treatment and 74 of patients with egfr mutations responded to egfr tki furthermore egfr mutation analysis from pleural effusion is feasible in mpes 8 39 43 the present authors suggest performing egfr mutation analysis in lung adenocarcinoma with mpes to select those with the highest response rate to treatment with egfr tki however future prospective studies are needed to clarify these  in conclusion pleural effusion is easy to collect and epidermal growth factor receptor gene mutation analysis from malignant pleural effusions is feasible the patients with malignant pleural effusions related to lung adenocarcinoma had a higher epidermal growth factor receptor gene mutation rate than the surgically resected specimens epidermal growth factor receptor tyrosine kinase inhibitor may be the treatment of choice for lung adenocarcinoma with malignant pleural effusions in east asia nonsmallcell lung cancer is the leading cause of death from cancer in the united states chemotherapy slightly prolongs survival among patients with advanced disease but at the cost of clinically significant adverse effects1 the success of the abl tyrosine kinase inhibitor imatinib in the treatment of chronic myeloid leukemia cml has demonstrated the effectiveness of targeting the critical genetic lesion that promotes proliferative signals in cancer cells2 gefitinib targets the atp cleft within the tyrosine kinase epidermal growth factor receptor egfr3 which is overexpressed in 40 to 80 percent of nonsmallcell lung cancers and many other epithelial cancers4 egfr signaling is triggered by the binding of growth factors such as epidermal growth factor egf resulting in the dimerization of egfr molecules or heterodimerization with other closely related receptors such as her2neu autophosphorylation and transphosphorylation of the receptors through their tyrosine kinase domains leads to the recruitment of downstream effectors and the activation of proliferative and cellsurvival signals5 despite its ubiquitous expression inactivation of the egfr gene in the mouse causes minimal defects67 suggesting that pharmacologic inhibition of egfr by gefitinib should have few adverse effects gefitinib inhibits the growth of some cancerderived cell lines and tumor xenografts although this effect is not well correlated with the level of expression of egfr or related members of the erbb family of receptors3 in initial clinical studies gefitinib had minimal adverse effects810 but tumor responses were observed in only 10 to 19 percent of patients with chemotherapyrefractory advanced nonsmallcell lung cancer1112 the addition of gefitinib to traditional chemotherapy provided no benefit1314 even in gliomas in which the finding of frequent amplification and rearrangements of the egfr gene suggests that egfr plays an important role gefitinib failed to induce clinically significant responses1516 despite these discouraging results the remarkably rapid and often profound response to gefitinib in a subgroup of patients with nonsmallcell lung cancer led to its approval as singledrug therapy for refractory lung cancer17 we evaluated tumors from patients with these dramatic responses to determine the underlying mechanisms methods nucleotidesequence analysis of tumor specimens tumor specimens were obtained during diagnostic or surgical procedures from patients with nonsmallcell lung cancer who were subsequently treated with gefitinib according to a protocol approved by the institutional review board of massachusetts general hospital in boston frozen tumor specimens along with matched normal tissue were available from four patients and paraffinembedded material was used from the other patients in addition specimens from 25 patients with primary nonsmallcell lung cancer who had not been exposed to gefitinib 15 with bronchoalveolar cancer 7 with adenocarcinoma and 3 with largecell lung cancer with matched normal tissues were obtained from the massachusetts general hospital tumor bank for mutational analysis of the entire egfr coding sequence dna was extracted from specimens all 28 exons were amplified and uncloned polymerasechainreaction pcr fragments were sequenced and analyzed in both sense and antisense directions for the presence of heterozygous mutations all sequence variants were confirmed by multiple independent pcr amplifications primer sequences and amplification conditions are explained in the supplementary appendix egfr mutations in exons 19 and 21 were also sought in primary tumors of the breast 15 specimens colon 20 specimens kidney 16 specimens pancreas 40 specimens and brain 4 specimens along with a panel of 108 cancerderived cell lines representing diverse histologic types listed in the supplementary appendix functional analysis of mutant egfr constructs the l858r and dell747p753inss mutations were introduced into the fulllength egfr coding sequence with the use of sitedirected mutagenesis and inserted into a cytomegalovirus promoterdriven expression construct puse upstate cos7 cells were transfected lipofectamine 2000 invitrogen with 1 \\u03bcg of the expression constructs and then replated 18 hours later at a concentration of 5\\u00d7104 cells per well in 12well plates costar with dulbecco's minimal essential medium without fetalcalf serum after 16 hours of serum starvation cells were stimulated with 10 ng of egf per milliliter sigma to determine whether the mutant receptors were inhibited by gefitinib the drug was added to the culture medium three hours before the addition of 100 ng of egf per milliliter cells were exposed to egf for 30 minutes cell lysates were prepared in 100 \\u03bcl of laemmli lysis buffer followed by the resolution of proteins on 10 percent sodium dodecyl sulfatepolyacrylamidegel electrophoresis transfer to membranes and western blot analysis with the use of an enhanced chemiluminescence reagent amersham autophosphorylation of egfr was measured with antibody against phosphotyrosine at position 1068 and standardized to total protein expression shown with the use of antibody against egfr working concentration 11000 cell signaling technology results clinical characteristics of patients with a response to gefitinib patients with advanced chemotherapyrefractory nonsmallcell lung cancer have been treated with gefitinib as a single agent since 2000 at massachusetts general hospital a total of 275 patients were treated both before its approval on may 2003 by the food and drug administration fda as part of a compassionateuse expandedaccess program and subsequently with the use of a commercial supply during this period 25 patients were identified by physicians as having clinically significant responses to the drug a clinically significant response was defined as a partial response according to the response evaluation criteria in solid tumors18 for patients with measurable disease for patients whose tumor burden could not be quantified with the use of these criteria the response was assessed by two physicians table 1table 1 characteristics of nine patients with nonsmallcell lung cancer and a response to gefitinib shows the clinical characteristics of nine patients for whom tumor specimens obtained at the time of diagnosis were available tissue was not available from the other patients with a response to gefitinib most commonly because diagnostic specimens were limited to needle aspirates as a group the nine patients derived a substantial benefit from gefitinib therapy the median duration of survival from the start of drug treatment exceeded 18 months and the median duration of therapy was greater than 16 months consistent with previous reports we found that most patients with a response to gefitinib were women had never smoked and had bronchoalveolar tumors1112 patient 6 was representative of the cohort this patient a 32yearold man with no history of smoking presented with multiple brain lesions and bronchoalveolar carcinoma in the right lung he was treated with wholebrain radiotherapy followed by a series of chemotherapy regimens carboplatin and gemcitabine docetaxel and vinorelbine to which his tumor did not respond with a declining functional status and progressive lungtumor burden he started therapy with 250 mg of gefitinib per day his dyspnea promptly improved and computed tomography of the lung six weeks after the initiation of treatment revealed a dramatic improvement figure 1figure 1 example of the response to gefitinib in a patient with refractory nonsmallcell lung cancer egfr mutations in patients with a response to gefitinib we hypothesized that patients with nonsmallcell lung cancer who had striking responses to gefitinib had somatic mutations in the egfr gene that would indicate the essential role of the egfr signaling pathway in the tumor to search for such mutations we first looked for rearrangements within the extracellular domain of egfr that are characteristic of gliomas15 none were detected we therefore sequenced the entire coding region of the gene using pcr amplification of individual exons heterozygous mutations were observed in eight of nine patients all of which were clustered within the tyrosine kinase domain of egfr table 2table 2 somatic mutations in the tyrosine kinase domain of egfr in patients with nonsmallcell lung cancer and figure 2figure 2 mutations in the egfr gene in gefitinibresponsive tumors four tumors had inframe deletions removing amino acids 746 through 750 dele746a750 in patient 1 747 through 751 dell747t751inss in patient 2 and 747 through 753 dell747p753inss in patients 3 and 4 the second and third deletions were associated with the insertion of a serine residue resulting from the generation of a novel codon at the deletion breakpoint remarkably all these deletions overlapped sharing the deletion of four amino acids leucine arginine glutamic acid and alanine at codons 747 through 750 within exon 19 another three tumors had amino acid substitutions within exon 21 leucine to arginine at codon 858 l858r in patients 5 and 6 and leucine to glutamine at codon 861 l861q in patient 7 the l861q mutation is of particular interest since the same amino acid change in the mouse egfr gene is responsible for the dark skin dsk5 trait associated with altered egfr signaling19 a fourth missense mutation in the tyrosine kinase domain resulted in the substitution of cysteine for glycine at codon 719 within exon 18 g719c in patient 8 matched normal tissue was available for patients 1 4 5 and 6 and showed only the wildtype sequence indicating that the mutations had arisen somatically during tumor formation by comparison no mutations were observed in seven patients with nonsmallcell lung cancer who had had no response to gefitinib p<0001 by a twosided fisher's exact test prevalence of specific egfr mutations in nonsmallcell lung cancer and other types of cancer unlike gliomas in which rearrangements affecting the egfr extracellular domain have been extensively studied15 the frequency of egfr mutations in nonsmallcell lung cancer has not been defined we therefore sequenced the entire coding region of the gene in tumors from 25 patients with primary nonsmallcell lung cancer who were not involved in the gefitinib study including 15 with bronchoalveolar lung cancer which has been associated with responsiveness to gefitinib in previous clinical trials1112 heterozygous mutations were detected in two patients with bronchoalveolar cancers both had inframe deletions in the kinase domain that were identical to those found in the patients with a response to gefitinib \\u2014 namely dell747p753inss and dele746a750 table 2 given the apparent clustering of egfr mutations we sequenced exons 19 and 21 in a total of 95 primary tumors and 108 cancerderived cell lines representing diverse tumor types see the supplementary appendix no mutations were detected suggesting that only a subgroup of cancers in which egfr signaling may play a critical role in tumorigenesis harbor egfr mutations increase in egfinduced activation and gefitinibinduced inhibition of mutant egfr proteins to study the functional properties encoded by these mutations we expressed the receptor with the l747p753inss deletion and the receptor with the l858r missense mutation in cultured cells transient transfection of wildtype and mutant constructs into cos7 cells demonstrated equivalent expression levels indicating that the mutations do not affect the stability of the protein egfr activation was quantified by measuring phosphorylation of the tyrosine1068 residue commonly used as a marker of the autophosphorylation of egfr20 in the absence of serum and associated growth factors neither wildtype nor mutant egfr demonstrated autophosphorylation figure 3afigure 3 enhanced egfdependent activation of mutant egfr and increased sensitivity of mutant egfr to gefitinib and figure 3b however the addition of egf doubled or tripled the activation of both mutant egfrs as compared with the activation of the wildtype receptor moreover whereas the activation of normal egfr was downregulated after 15 minutes consistent with the internalization of the receptor the two mutant receptors demonstrated continued activation for up to three hours figure 3a similar results were obtained with the use of antibodies to measure the total phosphorylation of egfr after the addition of egf data not shown since seven of the eight egfr tyrosine kinase mutations reside near the atp cleft which is targeted by gefitinib we assessed whether the mutant proteins have altered sensitivity to the inhibitor egfinduced autophosphorylation of egfr was measured in cells pretreated with various concentrations of gefitinib remarkably both mutant receptors were more sensitive than the wildtype receptor to inhibition by gefitinib wildtype egfr was inhibited by 50 percent at a gefitinib concentration of 01 \\u03bcm and was completely inhibited by a concentration of 20 \\u03bcm whereas the respective values for the two mutant proteins were 0015 \\u03bcm and 02 \\u03bcm figure 3c and figure 3d this difference in drug sensitivity may be clinically relevant since pharmacokinetic studies indicate that daily oral administration of 400 to 600 mg of gefitinib results in a mean steadystate trough plasma concentration of 11 to 14 \\u03bcm whereas the currently recommended daily dose of 250 mg leads to a mean trough concentration of 04 \\u03bcm21 discussion gefitinib is the first agent designed with a known molecular target to receive fda approval for the treatment of lung cancer yet its activity is limited to a subgroup of patients with nonsmallcell lung cancer we have identified specific activating mutations within the tyrosine kinase domain of egfr as the molecular correlate of the dramatic responses to gefitinib in this subgroup these somatic mutations were identified in eight of nine patients with a response to gefitinib the ninth patient may have had an undetected mutation or a mutation in a heterodimerization partner of egfr these results together with the finding of egfr mutations in tumors from 2 of 25 patients with nonsmallcell lung cancer who had not received gefitinib 8 percent suggest that such mutations account for the majority of responses to gefitinib reported in clinical studies1112 the heterozygous nature of egfr mutations suggests that they exert a dominant oncogenic effect which is evident despite the presence of the second wildtype allele the presence of an additive specific gain of function is further supported by the observation of identical somatic mutations in different tumors these mutations are clustered near the atp cleft of the tyrosine kinase domain where they flank amino acids shown in crystallographic studies to mediate binding of 4anilinoquinazoline compounds such as gefitinib22 figure 4figure 4 clustering of mutations in the egfr gene at critical sites within the atpbinding pocket we postulate that the mutations result in repositioning of these critical residues stabilizing their interaction with both atp and its competitive inhibitor gefitinib such a mechanism would explain both the increased receptor activation after ligand binding and the enhanced inhibition induced by gefitinib structural analysis of the mutant receptors will therefore provide important insight into the mechanisms that regulate the activation of egfr and the design of more potent inhibitors targeting the mutant receptors our observations have implications for the identification of molecular targets for cancer therapy using smallmolecule kinase inhibitors the effectiveness of imatinib in cml is based on its ability to target the abl tyrosine kinase which is activated by the bcrabl translocation ie the philadelphia chromosome in all patients with this disease and can transform hematopoietic cells223 similar evidence designating a protein as an optimal therapeutic target is not available for most epithelial cancers our data suggest that egfr tyrosine kinase mutations can be used to identify the subgroup of patients with nonsmallcell lung cancer in whom this growth factor receptor may be essential to tumor growth whereas the overexpression of egfr in the absence of mutations may reflect the less critical role played by this factor in the majority of cases this emphasis on genetic alterations is consistent with the observation that the amplification of the her2neu gene is a more reliable predictor than protein overexpression of the responsiveness of breast cancer to the targeting antibody trastuzumab and that ckit mutations can be used to determine the response of gastrointestinal stromal tumors to imatinib2425 ongoing largescale sequencing efforts may reveal additional mutations in other kinases linking different cancers to potential therapeutic targets2627 gefitinib has not elicited clinical responses in patients with gliomas despite the high frequency of amplification and rearrangements of the egfr gene in such patients1516 however the egfr tyrosine kinase mutations in patients with nonsmallcell lung cancer are fundamentally different from the gliomaassociated deletions within the extracellular domain of egfr these truncated egfr proteins resemble the avian erythroblastosis viral oncogene verbb in mediating constitutive ligandindependent activation of the receptor but they do not alter the atp cleft of the tyrosine kinase that is bound by gefitinib the enhanced sensitivity to gefitinib associated with tyrosine kinase mutations may therefore contribute substantially to the clinical responses of certain patients with nonsmallcell lung cancer the plasma concentrations of gefitinib that can be achieved with the use of current dosage recommendations21 exceed the drug concentration that suppressed autophosphorylation of the mutant egfr tyrosine kinase in our assays but are below those required to suppress the wildtype receptor in vitro analysis of wildtype egfr has also suggested that low concentrations of gefitinib may be sufficient to suppress autophosphorylation at some tyrosine residues but that abrogation of downstream signaling requires a higher dose28 thus in patients with gliomas in whom biologic dependence on egfr signaling is identified by the presence of gene amplification or deletions within the extracellular domain a clinical response may require plasma concentrations of an egfr tyrosine kinase inhibitor that are sufficient to abrogate downstream signaling understanding the molecular basis of responsiveness to gefitinib has immediate clinical implications with respect to patients with nonsmallcell lung cancer the clustering of mutations within specific regions of the egfr tyrosine kinase domain makes possible the potential development of rapid and reliable diagnostic testing to guide the clinical use of gefitinib for patients whose tumors have activating mutations of egfr the dramatic responses to gefitinib of patients whose disease has been refractory to all other therapies suggest that this agent may be more effective if used earlier in the course of treatment prospective validation of egfr tyrosine kinase mutations as predictors of the responsiveness to gefitinib is warranted and genotypedirected clinical trials of this tyrosine kinase inhibitor in the initial treatment of advanced nonsmallcell lung cancer \\u2014 and even in the adjuvant setting after surgical resection \\u2014 should now be considered  somatic mutations in the tyrosine kinase tk domain of the epidermal growth factor receptor egfr gene are reportedly associated with sensitivity of lung cancers to gefitinib iressa kinase inhibitor inframe deletions occur in exon 19 whereas point mutations occur frequently in codon 858 exon 21 we found from sequencing the egfr tk domain that 7 of 10 gefitinibsensitive tumors had similar types of alterations no mutations were found in eight gefitinibrefractory tumors p  0004 five of seven tumors sensitive to erlotinib tarceva a related kinase inhibitor for which the clinically relevant target is undocumented had analogous somatic mutations as opposed to none of 10 erlotinibrefractory tumors p  0003 because most mutationpositive tumors were adenocarcinomas from patients who smoked <100 cigarettes in a lifetime \\u201cnever smokers\\u201d we screened egfr exons 228 in 15 adenocarcinomas resected from untreated never smokers seven tumors had tk domain mutations in contrast to 4 of 81 nonsmall cell lung cancers resected from untreated former or current smokers p  00001 immunoblotting of lysates from cells transiently transfected with various egfr constructs demonstrated that compared to wildtype protein an exon 19 deletion mutant induced diminished levels of phosphotyrosine whereas the phosphorylation at tyrosine 1092 of an exon 21 point mutant was inhibited at 10fold lower concentrations of drug collectively these data show that adenocarcinomas from never smokers comprise a distinct subset of lung cancers frequently containing mutations within the tk domain of egfr that are associated with gefitinib and erlotinib sensitivity  tyrosine kinases tks regulate signaling pathways that control critical cellular activities 1 when overexpressed or activated by mutations tks can contribute to the development of cancers if tumor cells depend on a mutant tk for survival as illustrated by certain mouse models of cancer 2 3 the mutated enzyme can fortuitously serve as an achilles' heel for cancer therapy 4 human examples include bcrabldependent chronic myelogenous and acute lymphoblastic leukemias 5 kit and pdgfradependent gastrointestinal stromal tumors 6 and pdgfradependent hypereosinophilic syndrome 7 in each disease activated oncogenes encode tks inhibition by imatinib mesylate gleevec leads to rapid and durable clinical responses  egfr is a tk of the erbb family that is the presumptive target of the tk inhibitor tki gefitinib this drug is an anilinoquinazoline fig 5 which is published as supporting information on the pnas web site that reversibly competes with atp at a critical atpbinding site lysine 745 k745 within the epidermal growth factor egf receptor egfr protein 8 9 in vitro gefitinib selectively inhibits the kinase activity of egfr versus a handful of other kinases 10 in two phase ii trials radiographic regressions of tumors were observed in 28 of patients treated in japan and 10 of those studied in europe and the us 11 12 dramatic responses occurred within the first two weeks of initiating therapy eg ref 13 similar to those seen in the murine and human examples noted above assuming that the drug did affect a kinase these kinds of responses suggested that at least some lung tumors depended on a specific genetic lesion for tumor survival however when gefitinib was approved as second or thirdline treatment for patients with nonsmall cell lung cancer nsclc the clinically relevant targets of the drug in human tumors were unknown analyses of both preclinical xenograft models 14 and specimens from gefitinibsensitive and refractory tumors 15 did not reveal any obvious relationship between egfr expression levels and tumor sensitivity retrospective epidemiologic analyses suggested that gefitinib is more likely to be effective in japanese patients 11 individuals with adenocarcinomas of the bronchioloalveolar carcinoma bac subtype and \\u201cnever smokers\\u201d 16  recently two groups have shown that mutations in the tk domain of egfr are associated with sensitivity of nsclc to gefitinib 17 18 in total deletions or amino acid substitutions in exons 18 19 and 21 of egfr were found in 13 of 14 tumors sensitive to the drug but in none of 11 tumors with no response lynch and colleagues 17 found mutations in another 2 of 25 primary nsclcs and paez et al 18 found egfr mutations in 16 of 119 unselected tumors with a striking predominance of mutations found in 15 of 58 28 specimens from japan as compared to 1 of 61 from the us 2  to confirm and extend data on gefitinib sensitivity we examined the status of the tk domain of egfr in tumors that were sensitive and refractory to the drug to determine whether a related but distinct tki erlotinib fig 5 \\u201ctargets\\u201d a similar subset of nsclcs we also profiled erlotinibsensitive and refractory tumors the clinically relevant target of erlotinib has not yet been documented to examine whether smoking history is predictive of the likelihood of egfr mutations we determined the incidence of egfr tk domain mutations in 96 resected nsclcs from never smokers as well as former and current smokers who had never received a tki finally in an effort to explain the selective advantage of cells with mutant egfr and the drug sensitivity conferred upon mutantbearing tumors we began to characterize some biochemical properties of egfr mutants in vitro  go to methods tissue procurement tumor specimens were obtained on protocols approved by the institutional review board and the human tissue utilization committee of memorial sloankettering cancer center paraffin blocks of tumor material obtained from patients before systemic treatment for lung cancer were collected retrospectively for patients on gefitinib n  18 and prospectively for patients on erlotinib n  17 frozen tumor specimens from untreated patients with stage iiiia nsclc n  96 were prospectively collected at the time of surgical resection supporting text and table 3 which are published as supporting information on the pnas web site  mutational analyses of egfr in lung tumors genomic dna was derived from either tumors embedded in paraffin blocks or from fresh frozen tumors see supporting text and tables 4 and 5 which are published as supporting information on the pnas web site for pcr primers all sequencing reactions were performed in both forward and reverse directions and all mutations were confirmed by pcr amplification of an independent dna isolate for drugsensitive tumors that did not have mutations sequences from exons 19 and 21 were also determined from at least two independently derived pcr products fisher's exact test was used to calculate p values  functional analyses of mutant egfrs egfr has two numbering systems the first denotes the initiating methionine in the signal sequence as amino acid 24 the second used here denotes the methionine as amino acid +1 commercial antibodies such as the y1068specific antiphosphoegfr use the first nomenclature to be consistent we consider y1068 as y1092  mutations were introduced into fulllength egfr by using a quikchange sitedirected mutagenesis kit stratagene see supporting text all mutant clones were fully resequenced to ensure that no additional mutations were introduced cdnas were cloned into pcdna31 expression vectors invitrogen for transient transfections 293t human embryonic kidney cells were transfected 2 \\u00d7 105 cells per well in sixwell plates by using fugene roche applied science and 08 \\u03bcg of plasmid dna cells were grown in dmem with high glucose 10 fcs 2 mm lglutamine 10 unitsml penicillin and 10 \\u03bcgml streptomycin at 37\\u00b0c and 5 co2 cells were serumstarved in media containing 01 serum egf cell signaling technology beverly ma was used at 100 ngml cells were treated with various concentrations of gefitinib or erlotinib which were provided by astrazeneca and genentech respectively at least three independent experiments were performed for all analyses  immunoblotting see supporting text for details on cell lysis and immunoblotting specific proteins were detected by using enhanced chemiluminescence amersham pharmacia and the following antibodies horseradish peroxidase hrpconjugated antiphosphotyrosine rc20 13333 and antitotal egfr 12500 bd transduction laboratories antiactin 12000 sigma antiphosphoegfr tyr1068 y1092 11000 cell signaling technology hrpconjugated antirabbit ig 15000 amersham pharmacia and hrpconjugated antimouse igg 12000 roche applied science blots were stripped with restore stripping buffer pierce at 37\\u00b0c for 15 min stripping was verified by reexposure to film before reprobing densitometry was performed by using imagequant v12 molecular dynamics at least three independent experiments were performed for all analyses  go to results mutations in egfr are commonly found in lung tumors sensitive to gefitinib to ascertain whether mutations within the tk domain of egfr are associated with sensitivity to the tki gefitinib 17 18 we performed mutational profiling of exons 1824 of egfr in tumors from 10 patients who demonstrated a partial response or marked clinical improvement defined in supporting text when treated with the drug as a singleagent at the memorial sloankettering cancer center all patients were selected retrospectively seven tumors 70 had mutations six of which had inframe nucleotide deletions in exon 19 that occur adjacent to k745 encoded by nucleotides 22332235 k745 has been shown to be critical for binding atp 19 a seventh patient had a nonsynonymous mutation at nucleotide 2573 t \\u2192 g in exon 21 resulting in a substitution of arginine for leucine at position 858 l858r this change occurs adjacent to the highly conserved dfg motif amino acids 855857 in the activation loop of the kinase 20 and was previously detected in nsclcs 17 18 table 1 and fig 6 ad which is published as supporting information on the pnas web site mutations were not found in dna from peripheral blood available from four patients g2 g3 g5 and g6 implying that the lesions arose in somatic cells among the seven tumors with mutations five arose in never smokers six had adenocarcinoma histology with features of bac and none were from patients of east asian origin  four of the exon 19 deletion mutations were the same as the previously described del e746a750 17 18 and two were not previously described del l747s752 and del e746t751insi the del e746t751insi mutation may have resulted from a double deletion of nucleotides 22292236 and 22452252 and an insertion of a single t nucleotide fig 1 a and b remarkably in this mutant despite elimination of the codon for k745 the remaining nucleotide sequence reencoded a lysine but eliminated amino acids lrea as in the other exon 19 deletions the lrea motif is completely conserved among egfrs in vertebrates fig 1c  fig 1 fig 1 deletion mutations in exon 19 of egfr from nsclcs sensitive to tkis gefitinib g or erlotinib e all lack four amino acids lrea which are conserved among vertebrate species a nucleotide alignments b amino acid alignments c amino acid alignments  most somatic egfr mutations in nsclcs have been reported to be heterozygous however in the analysis of two of seven tumors g1 and g3 no wildtype sequence was detected in the region encompassing the deletion this result implies that i the mutations in tumors from patients g1 and g3 were hemi or homozygous ii the mutant gene was selectively amplified in these tumors or iii mutations in general may be homozygous with wildtype sequence originating from contaminating \\u201cnormal\\u201d dna table 1 and fig 6a see discussion  table 1 table 1 characteristics of patients sensitive to gefitinib g and erlotinib e none of eight gefitinibrefractory tumors contained mutations within exons 1824 of egfr p  0004 thus egfr mutations appear to be associated with sensitivity of nsclcs to gefitinib  lung tumors sensitive to erlotinib also harbor mutations in egfr erlotinib like gefitinib is an atpcompetitive inhibitor of the egfr tk fig 5 erlotinib also appears to be particularly effective in lung cancers with bac histology and in patients who are never smokers\\u2021\\u2021 but the drug's clinically relevant target in tumors has not yet been documented to see whether lung tumors sensitive to erlotinib also contained mutations within the tk domain of egfr we analyzed exons 1824 in seven tumors from patients who demonstrated a partial response while on a phase ii trial of this agent in bac  five tumors contained mutations two of which were multinucleotide inframe deletions within exon 19 near the atpbinding site k745 similar to those found in gefitinibsensitive tumors these deletions all eliminated amino acids lrea amino acids 747750 fig 1 a and b one mutation del l747s752insq has not been previously reported three tumors contained the l858r mutation at nucleotide 2573 that is also found in gefitinibresponsive tumors one specimen also had a second mutation that was previously unreported a c \\u2192 t change at nucleotide 2326 in exon 20 substituting cysteine for arginine at position 776 r776c carboxyterminal to the ploop fig 6e four of five mutationpositive tumors were obtained from never smokers and none were from patients of east asian origin  matched normal tissue available from four patients e1e3 and e5 showed only the wildtype sequence indicating that mutations arose in somatic cells no mutations were observed in exons 1824 in tumors from 10 patients on the trial who did not have a partial response p  0003 thus similar to results with gefitinib mutations in the egfr tk domain are also associated with sensitivity of nsclcs to erlotinib  egfr mutations are commonly found in lung adenocarcinomas from never smokers nine of 12 75 mutationpositive tumors in this study had adenocarcinoma histology and were derived from never smokers table 1 moreover \\u224810 of all cases of lung cancer arise in patients who have no history of tobacco use most of which have adenocarcinoma histology 21 22 to determine in a prospective manner the frequency of egfr mutations in lung adenocarcinomas from never smokers we sequenced exons 228 of egfr in 15 tumor specimens that fit these clinical criteria these samples were selected from our prospectively collected tissue bank of surgically resected nsclcs derived from patients with stage i ii and iiia disease at the time of surgery and tissue banking none of the patients from whom the primary tumor was resected had received any treatment for lung cancer including tkis in this \\u201cenriched\\u201d population of tumors from never smokers 7 of 15 47 had mutations in the tk domain of egfr one tumor specimen 230 had a deletion in exon 19 del e746a750 and five specimens 3 24 25 42 and 166 had the l858r amino acid substitution in exon 21 table 2 the seventh tumor specimen 77 contained a previously unreported point mutation in exon 21 at nucleotide 2504 a \\u2192 t resulting in substitution of leucine for histidine at position 835 h835l fig 6f this mutation is predicted to lie under or adjacent to the activation loop of the egfr tk domain in contrast to the l858r mutation which is in the loop itself no mutations were found in adjacent normalappearing tissue from all seven patients  table 2 table 2 somatic mutations in the tk domain of egfr are common in surgically resected nsclcs derived from never smokers but infrequent in former or current smokers we also sequenced exons 228 of egfr from an additional 81 primary nsclcs randomly selected from the same tumor bank all tumors in this cohort were derived from former or current smokers and 24 tumors had squamous cell histology four of 81 5 had mutations in the egfr tk domain all were the previously observed l858r amino acid substitution within exon 21 table 2 corresponding normal tissue available for three of these four tumors specimens 5 65 and 134 had only wildtype sequence interestingly among the four tumors with the l858r mutation three specimens 65 98 and 134 arose in patients with a limited exposure to cigarette smoking all three had smoked \\u22641 pack per day for 9 years 9 \\u201cpack years\\u201d and had quit at least 30 years before surgery specimen 5 was resected from an individual with a 14 packyear history who quit 1 month before surgery taken together these data demonstrate that egfr mutations are commonly found in nsclcs from never smokers as opposed to former or current smokers 7 of 15 vs 4 of 81 p  00001 and that tumors likely to be mutationpositive can be identified by using specific clinical characteristics  biochemical properties of egfr mutants to gain further insight as to why cells containing mutant egfrs are selected during the growth of certain nsclcs and why mutations confer susceptibility to tkis we have begun to characterize the mutant proteins in cultured cells wildtype del l747s752 and l858r egfr were produced by transient transfection with expression vectors in 293t cells which have very low levels of endogenous egfr fig 2 expression of total egfr tegfr was assessed by immunoblotting using an antiegfr monoclonal antibody and actin served as an indicator of relative levels of protein per sample the size of mutant egfrs was virtually indistinguishable from wildtype egfr when assessed by immunoblotting interestingly the amount of tegfr relative to actin was on average 3fold higher for the del l747s752 protein than that for wildtype egfr n  5 the differences in tegfr were not likely caused by varying transfection efficiencies as equal numbers of cells were used for each separate transfection and levels of tegfr for another egfr mutant l858r were comparable to that of wild type fig 2c  fig 2 fig 2 the del l747s752 mutant egfr appears to have reduced kinase activity 293t cells were transiently transfected with vector alone v or vector containing wildtype wt egfr del l747s752 or l858r thirtysix hours later cells were serumstarved for  we next used immunoblotting of extracts from cells expressing the various egfrs to assess various aspects of protein activity and drug sensitivity as a surrogate gauge of kinase activity we measured the levels of \\u201cautophosphorylated\\u201d tyr1092 on egfr y1092 the site of binding by adaptor molecules such as grb2 which leads to activation of the mitogenactivated protein kinaseextracellular signalrelated kinase cascade by using an y1092specific antibody ie phosphoegfr or pegfr in relation to levels of tegfr protein we also assessed the pattern and levels of induced tyrosinephosphorylation of cell proteins by using an antiphosphotyrosine antibody rc20 in the absence of serum and egf extracts from cells transfected with wildtype egfr demonstrated on average a 16fold greater ability to autophosphorylate y1092 than did the del l747s752 mutant even after addition of egf n  5 fig 2 a and c consistent with these results the del l747s752 mutant also induced markedly low levels of tyrosinephosphorylated proteins compared to wildtype egfr fig 2 b and c a longer exposure demonstrated that the relative intensities of such proteins appeared to be qualitatively different from that of wild type as well fig 2b right in contrast to the deletion mutant results with the y1092specific antibody for the l858r mutant were similar to those observed with wildtype egfr figs figs2c2c and and3b3b however the pattern of phosphotyrosine staining of cell proteins was still distinct fig 2c  fig 3 fig 3 compared to wildtype egfr the del l747s752 mutant has similar sensitivity to tkis whereas the l858r mutant is inhibited at \\u224810fold lower concentrations of drug a dosedependent inhibition by gefitinib of del l747s752 mutant egfr as compared  finally we assessed the sensitivity of the del l747s752 and l858r egfr mutants to tkis by measuring the ratio of pegfrtegfr in lysates from transiently transfected cells that were serumstarved and pretreated with gefitinib or erlotinib wildtype egfr and the del l747s752 mutant appeared to have approximately the same sensitivity to both gefitinib fig 3a and erlotinib data not shown by contrast the l858r mutant had an \\u224810fold greater sensitivity to both gefitinib data not shown and erlotinib fig 3b and fig 7 which is published as supporting information on the pnas web site  go to discussion in this study we confirm and extend recent work associating egfr mutations with sensitivity to the tki gefitinib 17 18 furthermore we establish that tumors sensitive to a related kinase inhibitor erlotinib contain similar types of egfr mutations when data from the two published reports and this study are used 25 of 31 81 tumors from individuals experiencing partial responses or marked clinical improvement while taking gefitinib or erlotinib contain mutations in the egfr tk domain by contrast none of 29 specimens from patients refractory to these agents had such mutations p < 1010 these findings demonstrate that mutations in the tk domain of egfr are associated with sensitivity to these two drugs whether gefitinib and erlotinib target exactly the same or overlapping sets of nsclc patients and whether distinct mutations confer greater sensitivity to specific egfrtk inhibitors has not yet been determined because the number of sensitive tumors analyzed is still small  we also demonstrate that 11 of 96 12 primary nsclcs resected from untreated patients contain mutations in egfr all within the tk domain none of these tumors were derived from patients of east asian origin taken together with the published literature on egfr mutations in primary lung cancers from patients in the us 14 of 182 tumors 8 are positive for egfr mutations these data could account for the responses seen in the phase ii trials of gefitinib in which \\u224810 of european or american patients experienced radiographic regressions 12 however other mechanisms of drug sensitivity may also apply remarkably by selecting tumors for certain clinical characteristics predictive of response to tkis ie tumors from never smokers with adenocarcinoma histology we enriched the percentage of patients with such mutations thus 7 of 15 tumors from neversmoking patients with adenocarcinoma histology had egfr mutations whereas only 4 of 81 nsclcs from former and current smokers contained them moreover three of the four patients in the latter cohort had relatively short smoking histories these data show that lung tumors from patients with minimal direct exposure to cigarettes and with adenocarcinoma histology usually with features of bac have a distinct molecular phenotype that distinguishes them from the remainder of nsclcs how patients with mutationpositive nsclc at various stages of disease should be treated is unknown but clearly warrants prompt investigation at the same time critical mutations in other kinases should be sought in nsclcs with wildtype egfr from never former and current smokers  among the nsclcassociated egfr mutations reported to date 49 of 56 88 occur in two \\u201chotspots\\u201d fig 4 a total of 29 of these 49 59 are multinucleotide inframe deletions that eliminate four amino acids lrea in exon 19 the other 20 of 49 \\u201chotspot\\u201d mutations 41 are point mutations in exon 21 that result in a specific amino acid substitution at position 858 l858r the remaining 7 of 56 mutations 12 are nucleotide substitutions found in exons 1821 outside of the common sites of mutation including the previously unreported r776c and h835l mutations described here in nearly all cases only one mutation has been detected per tumor however in this study we did find one tumor sample with two mutations patient e3 r776c and l858r the significance of the r776c mutation and whether these two mutations are on the same chromosome different chromosomes in the same cells or from different subclones with the same tumor specimen are all unknown  fig 4 fig 4 summary of mutations reported here and previously detected in the tk domain of egfr in nsclcs schematic view of egfr and key domains with an expanded view of the tk domain encoded by exons 1824 amino acids 718964 yellow sensitive to gefitinib  nsclcassociated egfr mutations are most frequently heterozygous however paez et al 18 reported one mutation involving exon 19 that appeared to be homozygous and we detected two such cases interpretation of mutational status solely from dna sequencing can be problematic on the one hand contaminating normal cells with wildtype egfr could account for apparent heterozygosity on the other hand amplification of mutant egfr as occurs in lung cancer 23 could account for detection of only mutant sequences mouse models expressing mutant egfr proteins in the lung and analysis of mutantpositive nsclcs by fluorescence in situ hybridization andor arraybased comparative genomic hybridization may help address these issues interestingly in one of these tumors g3 we detected a heterozygous intronic polymorphism downstream of exon 19 data not shown in this case it is probable that a gene conversion event occurred encompassing the area of deletion in exon 19  five of 17 reported patients who experienced partial responses or marked clinical improvement while on therapy with tkis had wildtype sequence in exons 1824 in the series reported by lynch et al 17 one of nine gefitinibsensitive patients did not have mutations within the entire coding region of egfr explanations for these results include i tumor fragments analyzed did not represent the tumors assessed while patients were on drug studies ii mutations were present but were below the detection rate of sequencing assays iii mutations lay outside the exons encoding the tk domain and iv other mechanisms involving wildtype egfr eg amplification may confer drug sensitivity in regards to the third point mutations outside the tk domain are unlikely because no other mutations in egfr were found in 240 lung tumor specimens sequenced to date 119 tumors from paez et al 18 25 tumors from lynch et al 17 and 96 resected nsclcs in this study thus it will be important to determine whether wildtype egfr or other kinases play a role in tumor responses to gefitinib and erlotinib  why mutant egfrs are selected for and how they confer susceptibility to tkis are questions that require further investigation to gain further insight we have begun to study various egfrs wild type an exon 19 deletion del l747s752 and an exon 21 aa substitution l858r in cultured cells with low levels of endogenous egfr by using transient transfection assays and parameters of activity such as the ability to phosphorylate tyrosine 1092 on egfr itself and to phosphorylate tyrosine residues on cell proteins in general compared to wildtype egfr the relative intensities of tyrosinephosphorylated proteins induced in cells after transfection by either egfr mutant was qualitatively and quantitatively different the level of phosphotyrosine was especially diminished for the del l747s752 mutant a result that we also obtained in two other cell lines ie cos1 and hpl1d immortalized human peripheral lung epithelia ref 24 and in 293t cells by using another deletion mutant del e746a750 data not shown furthermore in contrast to published data 17 we found that these two egfr mutants had differential sensitivities to tkis as measured by the effect of drug on phosphorylation of egfr at y1092 in cells transiently transfected with various egfrs whereas the l858r mutant was inhibited at \\u224810fold lower concentrations of tki the del l747s752 mutant appeared to have similar sensitivities as wildtype egfr to drug discrepancies with published reports could be due to by different experimental conditions however our data do not support the notion that egfr mutants have enhanced activity as has been suggested 17 rather our results suggest that the del l747s752 and l858r mutants could have altered substrate specificity compared to wildtype protein interestingly mutant egfrs in which the critical k745 residue is changed to methionine or arginine also have reduced kinase activity but can still activate the mitogenactivated protein kinase cascade and have an \\u201cincomplete program\\u201d of cellular tyrosine phosphorylations such signaling is postulated to occur via heterodimerization with other erbb family members such as erbb2neu 2527  the biochemical properties of the del l747s752 mutant as evaluated above are reminiscent of certain braf mutants found in human cancers 28 although the majority of braf mutants have elevated kinase activity 3 of 22 reported braf mutants were found to have reduced kinase activity toward mek in vitro nevertheless these three mutants were found to signal to erk in cells by activating craf possibly via an allosteric or transphosphorylation mechanism  tumors that are sensitive to either gefitinib or erlotinib eventually progress despite continued treatment with tkis in patients with bcrabldependent chronic myelogenous leukemia mutations within or amplification of bcrabl lead to clinical resistance 29 whether resistance to egfr tkis is caused by similar mechanisms affecting egfr andor to other mechanisms affecting alternative molecules remains to be determined  purpose mutations in epidermal growth factor receptor egfr can be used to predict the tumor response of patients receiving gefitinib for nonsmall cell lung cancer nsclc we investigated the association between mutations in egfr tyrosine kinase domain and tumor response and survival in gefitinibtreated nsclc patients  experimental design egfr mutations in exons 18 to 21 were analyzed by dna sequencing of paraffinembedded tumor tissues from gefitinibtreated nsclc patients the results were correlated with clinical variables  results egfr mutations were found in 611 33 of 54 of cases response rate and disease control rate were 568 and 685 respectively there was no significant difference in mutation rates between adenocarcinoma 29 of 43 and nonadenocarcinoma 4 of 11 p  0085 however all four nonadenocarcinomas with egfr mutations had no response to gefitinib presence of egfr mutations was the only independent predictor for disease control p  0003 and tumor response p  0017 in multivariate analysis positive predictive values were 879 and 708 and negative predictive values were 619 and 692 respectively in comparison with patients whose tumor was negative for egfr mutations patients with egfr mutations had better progressionfree survival median 76 versus 17 months p  0011 and overall survival median 147 versus 47 months p  0046  conclusions mutations in egfr tyrosine kinase correlate with treatment response and survival in gefitinibtreated nsclc patients and can be used as a predictive and prognostic factor thus analysis of egfr tyrosine kinase mutations in lung adenocarcinoma is of clinical significance as it can permit the customization of treatment with egfr tyrosine kinase inhibitors  lung cancer is the leading cause of cancerrelated deaths in many countries including taiwan 14 only a small proportion of lung cancer patients present with localized disease that can be effectively treated with conventional therapy the overall 5year survival rate of nonsmall cell lung cancer nsclc is <15 indicating the highly malignant nature of this disease  protein tyrosine kinases play important roles in the pathogenesis of many malignant tumors 5 among them epidermal growth factor receptor egfr tyrosine kinase the first receptor protein tyrosine kinase described has been implicated in the initiation and progression of nsclc 68 monoclonal antibodies and small molecular weight compounds that inhibit the egfr signaling pathway have been developed and shown to have antitumor effects 9  gefitinib a selective egfr tyrosine kinase inhibitor is an orally active agent for advanced nsclc in two phase ii trials ideal 1 and ideal 2 10 11 gefitinib was shown to have substantial effect when used alone as salvage treatment for patients treated previously with chemotherapy in taiwan where chinese descendants represent a major proportion of the population we have shown previously a tumor response rate of 33 12 this outcome was higher than that observed in caucasian patients but similar to that of japanese patients 10 11 these observations suggest ethnic or geographic differences in the response of nsclc patients to gefitinib treatment recently lynch et al 13 and paez et al 14 reported that mutations in the tyrosine kinase domain of egfr play a critical role in determining tumor response in nsclc patients receiving gefitinib interestingly one of these studies also showed a significant difference in the prevalence of egfr mutations between caucasian 28 and japanese 26 nsclc patients 14 although egfr mutations have been shown to be strongly associated with the response of nsclc patients to gefitinib treatment their role in response duration and patient survival is still unclear especially in a population that has a higher rate of gefitinib response  using pretreatment tumor samples we have conducted a retrospective study to detect mutations in the tyrosine kinase domain of the egfr gene and investigated the relationship between egfr tyrosine kinase mutations and clinical outcome in 54 taiwanese nsclc patients  materials and methods  patient selection and clinical data collection advanced nsclc patients n  146 were treated with gefitinib in a clinical study at taipei veterans general hospital taipei taiwan from march 2002 to february 2004 patient eligibility for study participation was histologically or cytologically confirmed inoperable nsclc irrespective of the presence of measurable lesions or good performance status selected patients had failed prior platinumbased chemotherapy or had a poor performance status at diagnosis and were considered at high risk for conventional chemotherapy patients with a life expectancy of <1 week were excluded patients were treated with a 250 mg fixed daily dose of gefitinib iressa zd1839 astrazeneca pharmaceuticals wilmington de as monotherapy treatment continued until there was intolerable toxicity disease progression or death all patients gave written informed consent the treatment protocol was approved by the taipei veterans general hospital institutional review board  histopathology review and dna preparation histopathology slides of specimens from bronchoscopic computed tomographyguided or sonographyguided biopsy wedge resection or lobectomy were reviewed tumors were classified according to the 1999 who histological typing 15 and staged according to the american joint committee on cancer cancer staging manual 16 genomic dna was prepared from formalinfixed paraffinembedded sections of tumor specimens using xylene deparaffinization proteinase k digestion and phenolchloroform extraction followed by ethanol precipitation an alternative dna extraction method using qiaamp dna mini kit qiagen hilden germany was used for larger size specimens extracted dna was evaluated by uv spectrophotometry and pcr of \\u03b2globin gene  pcr and sequencing of egfr gene from genomic dna eight pairs of oligonucleotide primers were used to amplify exons 18 to 21 of egfr gene by nested pcr according to the previously described procedures 13 pcr products were sequenced using the abi prism bigdye terminator cycle sequencing ready reaction kit and the abi prism 3700 genetic analyzer pe applied biosystems foster city ca sequences of pcr amplicons were compared with the cdna sequence of egfr obtained from genbank accession no nm 0052283 by mutation surveyor 203 softgenetics state college pa sequencer 414 gene codes ann arbor mi and manual review  evaluation for tumor response and patient survival pretreatment staging treatment response and adverse events were evaluated as described previously 12 clinical results of the first 76 patients from this study have been published objective tumor response was assessed by unidimensional method in accordance with response evaluation in solid tumors criteria 17 responses of patients with measurable lesions were classified as complete response partial response stable disease or progressive disease in patients with nonmeasurable lesions they were classified as nonprogressive disease or progressive disease response evaluation was done at the eighth week and every 8 weeks thereafter in addition chest xrays were done on days 8 15 and 29 and then every 4 weeks toxicities were assessed on days 8 15 and 29 and then every 4 weeks using the who toxicity criteria 18 except nail change which was graded by the modified national cancer institute common toxicity criteria version 20  objective response rate was calculated from patients with measurable lesions and disease control rate was calculated from all patients who had no disease progression at the first response evaluation duration of response was calculated from the date of documentation of complete response or partial response to the first instance of disease progression or death progressionfree survival was calculated from the date of initiation of gefitinib to the date of disease progression or death overall survival was defined as the period from the start of gefitinib treatment to the date of death  statistical analyses of clinical data and epidermal growth factor receptor mutations univariate analysis of patient characteristics and tumor response was done by the \\u03c72 test and the fisher's exact test the considered variables were age \\u226465 or >65 years gender performance status eastern cooperative oncology group performance status 01 or \\u22652 smoking exsmokercurrent smoker or nonsmoker histology adenocarcinoma or nonadenocarcinoma number of prior chemotherapy regimens 0 1 or \\u22652 number of metastatic organs 01 or \\u22652 and egfr mutation mutant or nonmutant logistic regression model with a backward stepwise procedure was used for multivariate analysis survival curves were plotted by the kaplanmeier method and compared by logrank test for ordinal variables a logrank test of trend was applied cox regression model with a backward stepwise procedure was done for multivariate survival analysis the association between the response and toxicity profile was analyzed using the maximal grade of toxicity experienced during the first month of treatment early toxicity profile analyses and figures were carried out with spss for windows version 120 spss inc chicago il  results  patient characteristics of the 146 patients enrolled for gefitinib treatment 62 had no available tumor tissue samples 21 had only limited samples that precluded research analysis and 9 provided specimens that yielded inadequate dna for analysis consequently tumor tissue samples that were adequate for dna analysis were available in 54 patients in this study there were similar numbers of male and female patients most patients were nonsmokers 667 and had adenocarcinoma 796 patient characteristics are summarized in table 1 the median followup was 161 months range 51245 months 42 patients had progressive disease and 33 patients died  view inlineview popup table 1 patient characteristics n  54 epidermal growth factor receptor mutations thirtythree of the 54 patients 611 had mutations in exons 18 to 21 of egfr gene as summarized in table 2 12 patients had deletion mutations 19 had substitution mutations 1 had combined deletion and substitution mutations and 1 had double substitution mutations all but one deletion mutation were found in exon 19 and all substitution mutations were found in exons other than 19 the frequency of mutations in exons 18 to 21 was 9 22 9 and 24 respectively deletions in exon 19 n  11 and substitution mutation of l858r n  9 in exon 21 were the two most common types of mutations representing 61 of total cases other types of mutations were found in one patient each and with the exception of l861q and g719s 13 14 were unreported previously we identified one deletion mutation and four substitution mutations but no duplicationinsertion mutations in exon 20  view inlineview popup table 2 clinical features and mutation patterns in 33 nsclc patients with mutated egfr gene association between epidermal growth factor receptor mutation and patient characteristics in the univariate analysis there was no statistically significant association between egfr mutations and age \\u226465 versus >65 years 600 versus 625 p  1000 gender male versus female 517 versus 720 p  0213 smoking status nonsmoker versus exsmokercurrent smoker 694 versus 444 p  0139 and histologic subtype adenocarcinoma versus nonadenocarcinoma 674 versus 364 p  0085 none of the patients fulfilled the criteria of bronchioloalveolar carcinoma bac by the 1999 who histological typing we estimated the proportion of bronchioloalveolar growth pattern in each adenocarcinoma specimen with a 10 increment scale accordingly we did a subgroup analysis and found that neither the presence nor the proportion of bac pattern of adenocarcinoma correlated with egfr mutations data not shown  there was no statistically significant difference in demographic characteristics between patients with egfr deletion mutations and patients with egfr substitution mutations data not shown  association between epidermal growth factor receptor mutation and tumor response a total of 37 patients had measurable lesions and 17 had nonmeasurable lesions of the 37 patients with measurable lesions 21 had partial response 7 had stable disease and 9 had progressive disease the objective response rate was 568 95 confidence interval 400735 in univariate analysis histologic subtype adenocarcinoma versus nonadenocarcinoma 700 versus 0 p  0001 and egfr mutations mutant versus nonmutant 708 versus 308 p  0045 were significantly associated with tumor response in multivariate analysis presence of egfr mutations was the only independent variable in predicting tumor response p  0017 table 3  view inlineview popup table 3 multivariate regression model in tumor response and patient survival in further analysis of the tumor response in patients with different patterns of egfr mutations patients with mutations in exon 18 4 of 4 100 and exon 19 6 of 7 857 seemed to have better tumor response rates in comparison with patients with mutations in exon 20 2 of 4 50 and exon 21 5 of 8 625 table 4  view inlineview popup table 4 objective tumor response in patients with egfr mutations in different exons of the 17 patients with nonmeasurable lesions 9 had nonprogressive disease and 8 had progressive disease therefore 37 of the total 54 patients had their disease controlled at the first response evaluation and the disease control rate was 685 95 confidence interval 557813 gender female versus male 880 versus 517 p  0010 histologic subtype adenocarcinoma versus nonadenocarcinoma 791 versus 273 p  0002 and egfr mutations mutant versus nonmutant 879 versus 381 p < 0001 were all correlated with disease control in multivariate analysis presence of egfr mutations p  0003 was the only independent predictor again table 3  of the 33 patients with tumors positive for mutant egfr 4 patients had disease progression within 2 months of gefitinib treatment all these tumors were squamous cell carcinoma and the patterns of mutation were dela767v769 exon 20 a763v mutation exon 20 l858r mutation exon 21 and a double substitution mutation g719s and l861q in exons 18 and 21  table 4 shows the tumor responses of patients with or without egfr mutations with respect to egfr mutations in exons 18 to 21 the positive predictive values were 88 29 of 33 and 71 17 of 24 and the negative predictive values were 62 13 of 21 and 69 9 of 13 for disease control and tumor response respectively  association between epidermal growth factor receptor mutation and patient survival and response duration using kaplanmeier method and logrank test patients with egfr mutation had a significantly longer progressionfree survival median 76 versus 17 months p  0001 fig 1a additionally female gender p  0035 old age p  0035 and adenocarcinoma p < 0001 all favored a longer progressionfree survival in cox regression model with a backward stepwise procedure histologic subtype p < 0001 egfr status p  0013 age p  0007 and performance status p  0008 were all independent predictors for a longer progressionfree survival in univariate analysis of overall survival patients with good performance status p  0002 adenocarcinoma p  0012 or mutated egfr p  0046 had a longer survival fig 1b in multivariate analysis only performance status p  0001 and histologic subtype p  0002 were independent predictors table 3  fig 1 download figureopen in new tabdownload powerpoint fig 1 kaplanmeier plots of progressionfree survival a overall survival b and duration of response c in patients with mutant versus nonmutant egfr kaplanmeier plots of duration of response in patients with egfr mutation in exons 18 19 and 21 versus exon 20 versus nonmutant d in 21 patients with objective tumor responses the median duration of response was 58 months although there was no statistically significant difference between patients with and without egfr mutations there was a trend favoring a longer duration of response in patients with mutations fig 1c however duration of response of patients with mutations in exon 20 was relatively worse and very similar to that of patients without egfr mutation fig 1d no other variables had a correlation with response duration  discussion  in this study we showed a high egfr mutation rate in our nsclc patient population with mutation patterns very similar to those reported previously 13 14 we documented that egfr mutations played a crucial role in determining gefitinib response in nsclc patients confirming findings from two previous landmark studies of <20 gefitinibtreated patients 13 14 for the first time we have also shown that mutations in egfr could confer longer survival in gefitinibtreated nsclc patients furthermore of those patients achieving partial response patients with mutations in egfr had a trend of longer response durations in comparison with patients lacking egfr mutations  gefitinib has shown antitumor activity in pretreated nsclc patients with response rates of 10 to 20 10 11 from these two phase ii clinical studies 10 11 gefitinib seemed to be more effective in females and patients with adenocarcinoma and a good performance status in a subsequent expanded access program study nonsmokers and bac subtype were also considered as promising predictors of gefitinib response 19 the issue such as japanese or nonjapanese patients was initially considered to be irrelevant by the multivariate regression model 10 however accumulated clinical evidence strongly suggests that ethnicgeographic factors play an important role in the response of patients to gefitinib treatment this is based on the high response rates reported exclusively from countries in the asia pacific region including taiwan 12 20 21 although the similarity of high response rates \\u223c30 among each independent group in this region was quite convincing most of these data came from observational studies with no supporting molecular analyses however two studies recently showed a strong association between egfr mutations and gefitinib response 13 14 additionally the prevalence of egfr mutations in nsclc was strikingly higher in japanese patients than in caucasian patients and the figures were close to the previously reported gefitinib response rates in these two patient populations 10 this was the first genetic evidence supporting the theory of ethnicgeographic differences in the response of nsclc patients to treatment with gefitinib  all patients in our study were taiwanese and the majority 83 had been heavily pretreated with chemotherapy the mutation rate 611 in our study was not only higher than that observed in caucasian nsclc patients \\u223c2 but also higher than that usually noted in japanese and taiwanese \\u223c40 treatmentnaive nsclc patients 13 14 22 23 and was most likely due to the skewed population in comparison to the lung cancer patient population demographics of taiwan there was a high proportion of female 46 versus 30 patients and patients with no previous history of smoking 67 versus 40 in this study\\u2014both of these factors could be associated with a higher egfr mutation rate 14 19 24 the most likely reason for the skewed demographics was the \\u201cselection effect\\u201d of prior chemotherapy it was well documented that females and nonsmokers have better prognosis translating to improved survival when treated by systemic chemotherapy 25 26 this would thus result in a tendency to include more females and nonsmokers in studies evaluating the secondline and postsecondline treatment  a retrospective study has shown better response rates in patients with bac 19 and some groups have implied that bac might have a different biology 27 in this study none of the patients fulfilled the criteria of bac by the 1999 who histological typing nevertheless we calculated the bac proportion in each adenocarcinoma specimen and did a subgroup analysis in these patients we did not find any statistical difference of bac proportion in the frequency of egfr mutations objective tumor response rate or disease control rate accordingly at least in our taiwanese patient population we did not observe evidence supporting a difference in gefitinib response among tumors with varied bac components data not shown  in our 33 patients with egfr mutations 11 had deletions in exon 19 9 had a substitution mutation at l858r in exon 21 10 had variable types of singlenucleotide substitution mutations in exon 18 20 or 21 1 had deletion in exon 20 1 had double substitution mutations in exons 18 and 21 and 1 had combined deletion in exon 19 and substitution mutation in exon 20 the patterns of egfr mutations were quite complex and this finding was similar to a previous report from this area 23  to the best of our knowledge in a total of 212 published cases with egfr mutation 13 14 2224 only 3 cases were nonadenocarcinoma 2 were adenosquamous cell carcinomas and 1 was large cell carcinoma none of these three cases had received gefitinib treatment therefore the clinical significance was unknown in this study we found four nonadenocarcinomas with egfr mutations and all were squamous cell carcinomas the mutation patterns were a763v l858r n826s and dell747p753inss in one each patients with the former two mutations had progressive disease after gefitinib treatment the latter two had stable disease with a relatively shorter progressionfree survival 67 and 24 months respectively in 29 patients with adenocarcinoma bearing egfr mutation 17 had partial response 9 had nonprogressive disease 1 had stable disease and only 2 had progressive disease in contrast none of our seven patients with nonadenocarcinoma had gefitinib response irrespective of their tumor egfr mutation status this finding of the lack of predictive value of egfr mutation in lung squamous cell carcinoma needs to be confirmed  among 212 published cases with egfr mutation 33 patients who had egfr mutations in exons 18 19 and 21 had been treated with gefitinib or erlotinib and only 1 case showed no tumor response the mutation in that case resulted in a stop codon and therefore truncated egfr protein 23 thus explaining the failure of gefitinib treatment in our study 4 of 33 patients with mutated egfr had progressive disease after gefitinib treatment the patterns of egfr mutation included dela767v769 double substitution mutation g719s and l861q a763v and l858r in one each both dela767v769 and a763v were in exon 20 and these two patterns are unreported previously substitution mutations at l858r and l861q have both been associated with gefitinib response in previous reports 13 23 substitution mutation of g719s has been reported but not in gefitinibtreated patient 14 22 although lynch et al 13 had reported one gefitinib responder with g719c mutation the association between g719s and gefitinib response is unknown further study is required to clarify the association between the sites and patterns of mutation and the treatment response especially in exon 20  in the present study we used formalinfixed paraffinembedded tumor tissue for mutational analysis of egfr gene the fragmentation of dna may lower the yield of the pcr reactions but does not compromise the accuracy of sequencing in a recent study from taiwan 23 the authors analyzed 16 paraffinembedded tumor tissues and 50 of the specimens had egfr mutations they also did mutation analyses on freshly frozen specimens of 69 adenocarcinoma cases and found a similar mutation rate 551 we consider dna extracted from formalinfixed paraffinembedded tumor tissues to be as adequate and suitable for mutational analysis as that extracted from freshly frozen tissues  in this study there were two findings that were somewhat contradictory to previous reports first a high mutation rate 40 was found in squamous cell carcinomas although the histologic typing of these tumors had been blindly reviewed and confirmed by two other pathologists this result should be carefully interpreted due to the limited number of cases second there was no difference in egfr mutation rates between patients who were current smokers or exsmokers and those who had no previous history of smoking however this finding concurred with a recent report of egfr mutational analysis in 101 unselected nongefitinibtreated nsclc in taiwan 23 ongoing investigations will determine if factors other than a history of smoking for example exposure to viruses or fumes from cooking oils 28 29 play a role in egfr mutations and are involved in the pathogenesis of lung adenocarcinomas  gene amplification plays an important role in the pathogenesis in several types of cancer and frequently precedes and may favor the occurrence of mutations 30 however the association between egfr gene amplification and gefitinib response is unknown egfr gene amplification was observed in 34 of glioblastoma however no objective tumor response was noted in a phase ii trial of gefitinib 31 similarly increased egfr gene copy number was evident in 60 of nsclc but the response rate to gefitinib was only 10 to 20 in the corresponding group of patients 32 egfr gene amplification may create a predisposition to egfr mutations but as a single factor it seems to be unrelated to the gefitinib tumor response  although the frequency of egfr mutations in our selected group of patients was relatively higher at 611 we have confirmed that egfr mutations correlated with a better response to gefitinib treatment as illustrated by response rate 568 and patient survival  cell type mutation status of egfr and performance status but not the number of prior chemotherapy regimen was associated with gefitinib response of ncslc patients 12 when compared with currently available frontline chemotherapy gefitinib showed much higher treatment efficacy in adenocarcinomas with egfr mutations 85 and comparable efficacy in adenocarcinomas with no egfr mutations \\u223c40 egfr mutations had no predictive value in patients with lung squamous cell carcinoma  based on the above findings we suggest that sequencing of egfr tyrosine kinase domain in lung adenocarcinoma patients at least those of east asian origin would be of clinical significance and permit the customization of treatment using egfr tyrosine kinase inhibitors as frontline treatment in neoadjuvant adjuvant or metastatic settings receptor tyrosine kinase genes were sequenced in nonsmall cell lung cancer nsclc and matched normal tissue somatic mutations of the epidermal growth factor receptor gene egfr were found in 15of 58 unselected tumors from japan and 1 of 61 from the united states treatment with the egfr kinase inhibitor gefitinib iressa causes tumor regression in some patients with nsclc more frequently in japan egfr mutations were found in additional lung cancer samples from us patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib but not in gefitinibinsensitive tumors or cell lines these results suggest that egfr mutations may predict sensitivity to gefitinib protein kinase activation by somatic mutation or chromosomal alteration is a common mechanism of tumorigenesis 1 inhibition of activated protein kinases through the use of targeted small molecule drugs or antibodybased strategies has emerged as an effective approach to cancer therapy 24 recently systematic analysis of kinase genes has identified mutations of the protein serinethreonine kinase gene braf in melanoma and other human cancers 5 and of multiple tyrosine kinase genes and the phosphatidylinositol 3kinase p110\\u03b1 catalytic subunit gene pik3ca in human colorectal carcinoma 6 7  lung carcinoma is the leading cause of cancer deaths in the united states and worldwide for both men and women 8 chemotherapy for nonsmall cell lung carcinoma nsclc which accounts for approximately 85 of lung cancer cases remains marginally effective 9  recently the epidermal growth factor receptor egfr tyrosine kinase inhibitor gefitinib iressa was approved in japan and the united states for the treatment of nsclc the original rationale for its use was the observation that egfr is more abundantly expressed in lung carcinoma tissue than in adjacent normal lung 10 however egfr expression as detected by immunohistochemistry is not an effective predictor of response to gefitinib 11  clinical trials have revealed significant variability in the response to gefitinib with higher responses seen in japanese patients than in a predominantly europeanderived population 275 versus 104 in a multiinstitutional phase ii trial 12 in the united states partial clinical responses to gefitinib have been observed most frequently in women in nonsmokers and in patients with adenocarcinomas 1315  to determine whether mutation of receptor tyrosine kinases plays a causal role in nsclc we searched for somatic genetic alterations in a set of 119 primary nsclc tumors consisting of 58 samples from nagoya city university hospital in japan and 61 from the brigham and women's hospital in boston massachusetts the tumors included 70 lung adenocarcinomas and 49 other nsclc tumors from 74 male and 45 female patients none of whom had documented treatment with gefitinib  as an initial screen we amplified and sequenced the exons encoding the activation loops of 47 of the 58 human receptor tyrosine kinase genes 16 table s1 from genomic dna from a subset of 58 nsclc samples that included 41 lung adenocarcinomas three of the tumors all lung adenocarcinomas showed heterozygous missense mutations in egfr not present in the dna from normal lung tissue from the same patients table s2 s0361 s0388 s0389 no mutations were detected in amplicons from other receptor tyrosine kinase genes all three tumors had the same egfr mutation predicted to change leucine858 to arginine fig 1a ctg\\u2192 cgg l858r    download highres image  open in new tab  download powerpoint fig 1 sequence alignment of selected regions within the egfr and braf kinase domains depiction of each type of egfr mutation in human nsclc egfr gbx00588 mutations in nsclc tumors are highlighted in yellow braf gbm95712 mutations in multiple tumor types 5 are highlighted in blue asterisks denote residues conserved between egfr and braf a l858r mutations in activation loop b g719s mutant in ploop c deletion mutants in egfr exon 19  we next examined exons 2 through 25 of egfr in the complete collection of 119 nsclc tumors exon sequencing of genomic dna revealed missense and deletion mutations of egfr in a total of 16 tumors all within exons 18 through 21 of the kinase domain all sequence alterations in this group were heterozygous in the tumor dna in each case paired normal lung tissue from the same patient showed wildtype sequence confirming that the mutations are somatic in origin the distribution of nucleotide and protein sequence alterations and the patient characteristics associated with these abnormalities are summarized in table s2  substitution mutations g719s and l858r were detected in two and three tumors respectively these mutations are located in the gxgxxg motif of the nucleotide triphosphate binding domain or ploop and adjacent to the highly conserved dfg motif in the activation loop 17 respectively the mutated residues are nearly invariant in all protein kinases and the analogous residues g463 and l596 in the braf protein serinethreonine kinase are somatically mutated in colorectal ovarian and lung carcinomas 5 18 fig 1 a and b  we also detected multiple deletion mutations clustered in the region spanning codons 746 to 759 within the kinase domain of egfr ten tumors carried one of two overlapping 15nucleotide deletions eliminating egfr codons 746 to 750 starting at nucleotide 2235 or 2236 del1 fig 1c and table s2 egfr dna from another tumor displayed a heterozygous 24nucleotide gap leading to the deletion of codons 752 to 759 del2 fig 1c representative chromatograms are shown in fig s1  the positions of the substitution mutations and the del1 deletion in the threedimensional structure of the active form of the egfr kinase domain 19 are shown in fig 2 note that the sequence alterations cluster around the active site of the kinase and that the substitution mutations lie in the activation loop and glycinerich ploop structural elements known to be important for autoregulation in many protein kinases 17    download highres image  open in new tab  download powerpoint fig 2 positions of missense mutations g719s and l858r and the del1 deletion in the threedimensional structure of the egfr kinase domain the activation loop is shown in yellow the ploop is in blue and the clobe and nlobe are as indicated the residues targeted by mutation or deletion are highlighted in red the del1 mutation targets the residues elrea in codons 746 to 750  the egfr mutations show a striking correlation with patient characteristics mutations were more frequent in adenocarcinomas 1570 or 21 than in other nsclcs 149 or 2 more frequent in women 945 or 20 than in men 774 or 9 and more frequent in the patients from japan 1558 or 26 and 1441 adenocarcinomas or 32 than in those from the united states 161 or 2 and 129 adenocarcinomas or 3 the highest fraction of egfr mutations was observed in japanese women with adenocarcinoma 814 or 57 notably the patient characteristics that correlate with the presence of egfr mutations are those that correlate with clinical response to gefitinib treatment  to investigate whether egfr mutations might be a determinant of gefitinib sensitivity pretreatment nsclc samples were obtained from 5 patients who responded and 4 patients who progressed during treatment with gefitinib out of more than 125 patients treated at the danafarber cancer institute either on an expanded access program or after regulatory approval of gefitinib 13 four of the patients had partial radiographic responses \\u226550 tumor regression in a computed tomography scan after 2 months of treatment whereas the fifth patient experienced dramatic symptomatic improvement in less than 2 months all of the patients were from the united states and were caucasian  while sequencing of the kinase domain exons 18 through 24 revealed no mutations in tumors from the four patients who progressed on gefitinib all five tumors from gefitinibresponsive patients harbored egfr kinase domain mutations the chisquare test revealed the difference in egfr mutation frequency between gefitinib responders 55 and nonresponders 04 to be statistically significant with p  00027 whereas the difference between the gefitinib responders and unselected us nsclc patients 55 versus 161 was also significant with p < 1012 20 the egfr l858r mutation previously observed in the unselected tumors was identified in one gefitinibsensitive lung adenocarcinoma fig 1a and table s3 ir3t three gefitinibsensitive tumors contained heterozygous inframe deletions fig 1c and table s3 del3 in two cases and del4 in one and one contained a homozygous inframe deletion fig 1c and table s3 del5 each of these deletions was found within codons 746 to 753 of egfr where deletions were also found in unselected tumors each of these three deletions is also associated with an amino acid substitution table s3 in all four samples where matched normal tissue was available these mutations were confirmed as somatic  to determine whether mutations in egfr confer gefitinib sensitivity in vitro the mutation status and response to gefitinib were determined in four lung adenocarcinoma and bronchioloalveolar carcinoma cell lines the h3255 cell line was originally derived from a malignant pleural effusion from a caucasian female nonsmoker with lung adenocarcinoma 21 this cell line was 50 times as sensitive to gefitinib as the other lines with an ic50 of 40 nm for cell survival in a 72hour assay fig 3a    download highres image  open in new tab  download powerpoint fig 3 a lung adenocarcinoma cell line with egfr receptor mutation is sensitive to growth and signaling inhibition by gefitinib a cells were treated with gefitinib at the indicated concentrations and viable cells were measured after 72 hours of treatment percentage of cell growth is shown relative to untreated controls h3255 cells have the egfr l858r mutation whereas the three remaining cell lines have wildtype egfr wt b inhibition of egfr phosphorylation and of downstream phosphorylation of akt and erk12 the cell lines were treated with gefitinib for 24 hours cell extracts were immunoblotted to detect the indicated protein species akt vakt murine thymoma viral oncogene homolog erk extracellular signalresponsive kinase  treatment with 100 nm gefitinib completely inhibited egfr autophosphorylation in h3255 fig 3b such treatment also inhibited the phosphorylation of known downstream targets of egfr such as the extracellular signalregulated kinase 12 erk12 and the vakt murine thymoma viral oncogene homolog akt kinase fig 3b a correlation that has been noted by others 22 in contrast the other three cell lines showed comparable levels of inhibition of target protein phosphorylation only when gefitinib was present at concentrations roughly 100 times as high fig 3b  the sequence analysis of egfr cdna in these four cell lines showed the l858r mutations in h3255 table s3 whereas the other three cell lines did not contain egfr mutations we also confirmed the presence of the l858r mutation in the primary tumor from which h3255 was derived table s3 irg although no matched normal tissue was available the results suggest that l858r mutant egfr is particularly sensitive to inhibition by gefitinib compared with the wildtype enzyme and that this likely accounts for the extraordinary drug sensitivity of the h3255 cell line  the identification of egfr mutations in a subset of human lung carcinomas and the association between egfr mutation and gefitinib sensitivity extend the emerging paradigm whereby genetic alterations in specific kinases and not simply kinase expression render tumors sensitive to selective inhibitors as is the case for imatinib treatment of ckit mutant gastrointestinal stromal tumors 23 thus although randomized trials of cytotoxic therapy with or without gefitinib revealed no survival benefit for the gefitinibtreated nsclc patients 24 25 our current data suggest that gefitinib may be particularly effective for treating lung cancers with somatic egfr mutations and that prospective clinical trials of egfr inhibition in patients with egfr mutations might reveal increased patient survival identification of egfr mutations in other malignancies perhaps including glioblastomas in which egfr alterations are already known 26 may identify other patients who could similarly benefit from treatment with egfr inhibitors  important questions remain to be answered including whether these alterations result in activated and transforming alleles of egfr whether receptors harboring such mutations will show differential sensitivity to any of the multiple egfr small molecule inhibitors and whether egfr receptors harboring such mutations are inhibited by antibodies directed against the extracellular domain furthermore it will be of interest to determine whether resistance to egfr inhibition emerges through secondary mutation as is the case in imatinibtreated chronic myelogenous leukemia 27 these results should stimulate further in vitro studies regarding these questions  finally the striking differences in the frequency of egfr mutation and response to gefitinib between japanese and us patients raise general questions regarding variations in the molecular pathogenesis of cancer in different ethnic cultural and geographic groups and argue for the benefit of population diversity in cancer clinical trials\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"class_adjusted\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 2,\n        \"min\": 0,\n        \"max\": 8,\n        \"num_unique_values\": 9,\n        \"samples\": [\n          7,\n          1,\n          5\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}","type":"dataframe","variable_name":"data_train_pre"},"text/html":["\n","  <div id=\"df-98ef0fc0-2e41-4fa5-bdbb-efaf02df9d9f\" class=\"colab-df-container\">\n","    <div>\n","<style scoped>\n","    .dataframe tbody tr th:only-of-type {\n","        vertical-align: middle;\n","    }\n","\n","    .dataframe tbody tr th {\n","        vertical-align: top;\n","    }\n","\n","    .dataframe thead th {\n","        text-align: right;\n","    }\n","</style>\n","<table border=\"1\" class=\"dataframe\">\n","  <thead>\n","    <tr style=\"text-align: right;\">\n","      <th></th>\n","      <th>combined_text</th>\n","      <th>class_adjusted</th>\n","    </tr>\n","  </thead>\n","  <tbody>\n","    <tr>\n","      <th>0</th>\n","      <td>FAM58A Truncating_Mutations cyclindependent ki...</td>\n","      <td>0</td>\n","    </tr>\n","    <tr>\n","      <th>1</th>\n","      <td>CBL W802* abstract background  nonsmall cell l...</td>\n","      <td>1</td>\n","    </tr>\n","  </tbody>\n","</table>\n","</div>\n","    <div class=\"colab-df-buttons\">\n","\n","  <div class=\"colab-df-container\">\n","    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-98ef0fc0-2e41-4fa5-bdbb-efaf02df9d9f')\"\n","            title=\"Convert this dataframe to an interactive table.\"\n","            style=\"display:none;\">\n","\n","  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n","    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n","  </svg>\n","    </button>\n","\n","  <style>\n","    .colab-df-container {\n","      display:flex;\n","      gap: 12px;\n","    }\n","\n","    .colab-df-convert {\n","      background-color: #E8F0FE;\n","      border: none;\n","      border-radius: 50%;\n","      cursor: pointer;\n","      display: none;\n","      fill: #1967D2;\n","      height: 32px;\n","      padding: 0 0 0 0;\n","      width: 32px;\n","    }\n","\n","    .colab-df-convert:hover {\n","      background-color: #E2EBFA;\n","      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n","      fill: #174EA6;\n","    }\n","\n","    .colab-df-buttons div {\n","      margin-bottom: 4px;\n","    }\n","\n","    [theme=dark] .colab-df-convert {\n","      background-color: #3B4455;\n","      fill: #D2E3FC;\n","    }\n","\n","    [theme=dark] .colab-df-convert:hover {\n","      background-color: #434B5C;\n","      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n","      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n","      fill: #FFFFFF;\n","    }\n","  </style>\n","\n","    <script>\n","      const buttonEl =\n","        document.querySelector('#df-98ef0fc0-2e41-4fa5-bdbb-efaf02df9d9f button.colab-df-convert');\n","      buttonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","\n","      async function convertToInteractive(key) {\n","        const element = document.querySelector('#df-98ef0fc0-2e41-4fa5-bdbb-efaf02df9d9f');\n","        const dataTable =\n","          await google.colab.kernel.invokeFunction('convertToInteractive',\n","                                                    [key], {});\n","        if (!dataTable) return;\n","\n","        const docLinkHtml = 'Like what you see? Visit the ' +\n","          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n","          + ' to learn more about interactive tables.';\n","        element.innerHTML = '';\n","        dataTable['output_type'] = 'display_data';\n","        await google.colab.output.renderOutput(dataTable, element);\n","        const docLink = document.createElement('div');\n","        docLink.innerHTML = docLinkHtml;\n","        element.appendChild(docLink);\n","      }\n","    </script>\n","  </div>\n","\n","\n","<div id=\"df-e4850d56-6553-4c33-9f53-3a65f105f70c\">\n","  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-e4850d56-6553-4c33-9f53-3a65f105f70c')\"\n","            title=\"Suggest charts\"\n","            style=\"display:none;\">\n","\n","<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n","     width=\"24px\">\n","    <g>\n","        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n","    </g>\n","</svg>\n","  </button>\n","\n","<style>\n","  .colab-df-quickchart {\n","      --bg-color: #E8F0FE;\n","      --fill-color: #1967D2;\n","      --hover-bg-color: #E2EBFA;\n","      --hover-fill-color: #174EA6;\n","      --disabled-fill-color: #AAA;\n","      --disabled-bg-color: #DDD;\n","  }\n","\n","  [theme=dark] .colab-df-quickchart {\n","      --bg-color: #3B4455;\n","      --fill-color: #D2E3FC;\n","      --hover-bg-color: #434B5C;\n","      --hover-fill-color: #FFFFFF;\n","      --disabled-bg-color: #3B4455;\n","      --disabled-fill-color: #666;\n","  }\n","\n","  .colab-df-quickchart {\n","    background-color: var(--bg-color);\n","    border: none;\n","    border-radius: 50%;\n","    cursor: pointer;\n","    display: none;\n","    fill: var(--fill-color);\n","    height: 32px;\n","    padding: 0;\n","    width: 32px;\n","  }\n","\n","  .colab-df-quickchart:hover {\n","    background-color: var(--hover-bg-color);\n","    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n","    fill: var(--button-hover-fill-color);\n","  }\n","\n","  .colab-df-quickchart-complete:disabled,\n","  .colab-df-quickchart-complete:disabled:hover {\n","    background-color: var(--disabled-bg-color);\n","    fill: var(--disabled-fill-color);\n","    box-shadow: none;\n","  }\n","\n","  .colab-df-spinner {\n","    border: 2px solid var(--fill-color);\n","    border-color: transparent;\n","    border-bottom-color: var(--fill-color);\n","    animation:\n","      spin 1s steps(1) infinite;\n","  }\n","\n","  @keyframes spin {\n","    0% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","      border-left-color: var(--fill-color);\n","    }\n","    20% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    30% {\n","      border-color: transparent;\n","      border-left-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","      border-right-color: var(--fill-color);\n","    }\n","    40% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-top-color: var(--fill-color);\n","    }\n","    60% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","    }\n","    80% {\n","      border-color: transparent;\n","      border-right-color: var(--fill-color);\n","      border-bottom-color: var(--fill-color);\n","    }\n","    90% {\n","      border-color: transparent;\n","      border-bottom-color: var(--fill-color);\n","    }\n","  }\n","</style>\n","\n","  <script>\n","    async function quickchart(key) {\n","      const quickchartButtonEl =\n","        document.querySelector('#' + key + ' button');\n","      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n","      quickchartButtonEl.classList.add('colab-df-spinner');\n","      try {\n","        const charts = await google.colab.kernel.invokeFunction(\n","            'suggestCharts', [key], {});\n","      } catch (error) {\n","        console.error('Error during call to suggestCharts:', error);\n","      }\n","      quickchartButtonEl.classList.remove('colab-df-spinner');\n","      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n","    }\n","    (() => {\n","      let quickchartButtonEl =\n","        document.querySelector('#df-e4850d56-6553-4c33-9f53-3a65f105f70c button');\n","      quickchartButtonEl.style.display =\n","        google.colab.kernel.accessAllowed ? 'block' : 'none';\n","    })();\n","  </script>\n","</div>\n","\n","    </div>\n","  </div>\n"],"text/plain":["                                       combined_text  class_adjusted\n","0  FAM58A Truncating_Mutations cyclindependent ki...               0\n","1  CBL W802* abstract background  nonsmall cell l...               1"]},"execution_count":8,"metadata":{},"output_type":"execute_result"}],"source":["data_train_pre.head(2)"]},{"cell_type":"code","execution_count":9,"metadata":{"executionInfo":{"elapsed":324,"status":"ok","timestamp":1729622657338,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"-zNl42ECeDB5"},"outputs":[],"source":["text = data_train_pre['combined_text'].tolist()"]},{"cell_type":"code","execution_count":10,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":10,"status":"ok","timestamp":1729622658433,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"aobYZRj-eDB9","outputId":"5aeb8e50-7681-4941-b396-a7d04ecce1a9"},"outputs":[{"data":{"text/plain":["9"]},"execution_count":10,"metadata":{},"output_type":"execute_result"}],"source":["num_class = len(data_train_pre['class_adjusted'].unique())\n","num_class"]},{"cell_type":"code","execution_count":11,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":10,"status":"ok","timestamp":1729622659635,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"Gnfp5IileDB-","outputId":"429b2ed3-1d35-4ad6-ba15-d1c8de65823c"},"outputs":[{"data":{"text/plain":["array([[1., 0., 0., ..., 0., 0., 0.],\n","       [0., 1., 0., ..., 0., 0., 0.],\n","       [0., 1., 0., ..., 0., 0., 0.],\n","       ...,\n","       [1., 0., 0., ..., 0., 0., 0.],\n","       [0., 0., 0., ..., 0., 0., 0.],\n","       [0., 0., 0., ..., 0., 0., 0.]])"]},"execution_count":11,"metadata":{},"output_type":"execute_result"}],"source":["# Create one-hot encode for classification 9 classes\n","label = to_categorical(data_train_pre['class_adjusted'], num_classes=num_class)\n","label"]},{"cell_type":"markdown","metadata":{"id":"xS7o50vQeDB_"},"source":["##### Split"]},{"cell_type":"code","execution_count":12,"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":542},"executionInfo":{"elapsed":1549,"status":"ok","timestamp":1729622664587,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"d9uB-OqyeDB_","outputId":"1e9d01cc-26da-47d8-bf4f-4d5965fa5095"},"outputs":[{"data":{"text/html":["<html>\n","<head><meta charset=\"utf-8\" /></head>\n","<body>\n","    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n","        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.35.2.min.js\"></script>                <div id=\"99644a6c-1b91-4489-86b4-9dd06224f7a0\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"99644a6c-1b91-4489-86b4-9dd06224f7a0\")) {                    Plotly.newPlot(                        \"99644a6c-1b91-4489-86b4-9dd06224f7a0\",                        [{\"alignmentgroup\":\"True\",\"bingroup\":\"x\",\"histnorm\":\"probability density\",\"hovertemplate\":\"variable=0\\u003cbr\\u003evalue=%{x}\\u003cbr\\u003eprobability density=%{y}\\u003cextra\\u003e\\u003c\\u002fextra\\u003e\",\"legendgroup\":\"0\",\"marker\":{\"color\":\"#636efa\",\"pattern\":{\"shape\":\"\"}},\"name\":\"0\",\"offsetgroup\":\"0\",\"orientation\":\"v\",\"showlegend\":true,\"texttemplate\":\"%{value:.2%}\",\"x\":[0,1,1,2,3,3,4,0,3,3,3,3,3,3,4,3,0,3,4,3,5,3,3,3,3,3,3,3,6,3,3,6,3,1,6,3,3,0,0,3,3,0,3,0,3,0,3,0,3,3,0,0,3,0,0,0,3,0,0,0,0,3,3,3,0,0,3,6,6,6,0,1,0,3,0,1,1,6,6,6,1,1,6,6,1,6,6,6,0,3,0,3,3,3,0,3,3,1,3,3,0,0,3,0,3,0,4,0,0,4,0,4,3,3,3,0,0,3,0,0,3,7,8,8,8,8,8,8,8,8,8,8,8,8,8,8,8,3,6,1,4,6,6,6,6,1,6,6,6,6,6,1,6,1,4,1,6,1,6,1,1,1,1,3,6,6,6,6,1,6,6,1,1,6,6,6,6,1,1,1,6,1,1,1,6,1,1,6,6,6,6,5,1,6,6,6,2,6,6,3,6,1,6,6,6,1,6,6,1,1,6,6,6,1,6,1,1,3,6,1,7,6,1,6,1,6,6,6,6,6,6,6,6,6,1,6,6,3,6,6,6,6,6,6,6,6,6,1,1,1,3,6,6,1,6,6,6,1,6,1,0,6,6,6,6,4,1,6,1,1,6,1,6,6,6,1,6,6,1,1,1,6,1,1,6,6,0,0,6,6,0,0,3,5,3,2,0,3,3,6,6,1,1,2,1,2,7,7,7,0,0,4,4,0,4,1,1,1,1,6,1,6,1,1,1,1,1,6,5,1,1,6,6,1,1,1,6,1,1,1,6,0,0,0,3,3,3,0,3,0,0,3,1,3,3,3,6,3,6,0,3,1,1,1,1,1,1,1,1,3,3,0,3,3,0,0,0,3,3,0,3,3,0,5,3,3,0,0,3,1,3,0,3,3,3,0,3,0,3,0,0,5,0,3,3,0,3,0,0,0,5,0,3,0,0,0,3,0,0,3,3,0,0,0,3,0,3,0,3,3,3,0,3,3,0,0,0,0,3,3,0,0,3,3,0,4,0,0,3,0,3,0,3,3,3,0,3,0,0,3,0,0,3,3,0,0,3,0,0,0,0,0,0,0,0,0,0,0,0,3,3,3,2,3,3,3,3,2,0,0,1,3,0,0,3,0,3,0,3,0,0,3,3,5,0,1,0,4,0,0,3,0,8,0,3,0,0,3,3,0,0,3,3,0,5,0,0,3,3,3,0,0,0,0,3,0,4,0,4,3,0,5,0,0,0,0,5,5,0,5,0,0,0,0,0,0,3,0,3,0,0,0,0,3,0,3,3,0,0,3,3,3,3,3,2,3,3,5,0,3,0,0,4,3,0,0,3,3,5,3,4,5,5,0,0,3,3,3,0,0,0,3,3,0,0,3,3,3,0,6,6,6,1,3,6,3,2,3,4,3,0,3,3,0,3,0,0,1,3,0,4,3,0,3,6,0,0,5,3,0,5,0,3,3,3,3,3,3,3,3,0,3,3,3,3,3,3,0,3,3,3,3,3,0,3,3,3,3,3,3,5,3,3,3,3,3,3,0,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,0,3,3,3,3,3,3,0,0,5,4,1,1,1,6,4,5,6,5,6,5,5,5,5,6,6,6,1,5,5,6,6,6,1,3,6,4,4,6,1,5,3,1,0,6,6,6,6,6,6,6,6,1,6,1,6,7,6,3,1,6,5,3,6,6,1,4,6,6,1,4,5,1,3,6,6,1,6,6,5,6,6,4,6,6,6,5,6,6,6,6,6,6,2,2,6,6,5,6,1,2,5,5,5,6,6,6,6,6,0,0,0,0,1,0,0,0,0,0,0,0,0,4,0,0,0,0,0,0,3,5,3,0,0,3,3,0,3,0,3,3,0,3,6,1,1,1,1,1,1,1,1,1,6,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,6,6,6,1,0,0,0,6,1,4,6,4,6,2,6,6,6,6,6,2,6,6,2,6,4,6,6,2,2,6,2,6,1,6,2,6,6,6,6,6,6,2,2,6,6,1,6,6,6,0,6,6,6,4,6,6,6,6,6,6,6,6,6,6,6,6,3,6,0,1,1,1,1,1,1,1,6,1,5,0,1,1,5,1,1,1,6,1,6,3,0,0,3,3,3,3,1,3,1,2,6,6,1,6,6,6,6,6,6,6,6,2,6,6,6,1,6,3,0,1,3,0,3,6,6,3,3,4,3,3,0,2,2,2,3,0,3,0,2,3,3,2,3,3,2,3,0,0,3,0,3,0,0,0,3,3,3,3,0,3,3,0,3,3,0,3,3,3,3,3,3,0,0,3,0,0,3,3,0,6,3,0,3,0,3,0,0,0,3,0,0,0,0,1,6,6,6,1,6,6,1,1,1,6,6,1,6,0,0,0,1,0,0,6,0,6,0,8,8,8,8,8,8,6,6,6,1,5,5,0,5,4,0,0,0,3,3,3,0,3,3,3,3,3,3,0,3,3,0,3,3,3,0,6,5,1,6,6,5,6,6,6,6,6,6,1,6,6,6,1,6,6,1,6,1,6,6,6,1,1,6,6,6,6,6,6,8,8,8,4,3,0,0,3,0,0,0,0,0,6,4,1,6,6,6,6,5,6,6,6,1,5,6,1,6,4,5,6,6,1,4,1,6,6,1,6,6,6,4,6,6,6,4,1,6,6,6,6,4,1,1,6,1,1,4,1,6,6,6,1,1,6,4,6,6,1,6,1,1,1,6,6,1,1,6,0,0,1,5,5,5,5,4,6,6,6,1,6,1,1,3,3,3,0,0,0,3,3,0,0,3,3,0,3,3,3,3,3,0,0,3,0,3,3,6,5,1,6,0,4,6,4,6,1,1,6,6,1,1,1,1,6,6,1,1,6,1,1,1,6,1,1,1,1,0,0,3,5,0,5,3,5,0,4,5,5,3,4,0,3,0,3,4,4,5,5,3,0,3,4,0,3,5,5,4,5,0,5,3,5,5,5,0,1,1,6,2,6,7,2,6,1,6,6,6,6,7,6,6,6,4,4,6,4,6,6,6,6,6,6,2,1,6,8,6,6,6,8,1,6,6,6,6,6,4,6,6,0,6,6,6,1,6,6,6,6,1,4,6,0,1,6,6,6,6,1,1,6,4,1,6,4,0,6,4,4,4,6,6,4,4,6,6,4,6,6,6,6,1,1,6,4,1,6,3,3,3,3,3,3,0,3,3,3,3,0,3,3,3,0,0,3,3,0,0,0,6,6,6,1,4,1,4,0,5,6,6,6,0,6,6,5,6,6,6,6,6,5,1,6,0,6,6,5,6,1,6,0,4,6,6,6,1,6,6,0,6,4,1,6,6,6,1,4,6,6,6,0,0,6,6,6,4,6,6,1,6,2,2,6,6,6,6,6,6,6,6,6,1,6,6,6,6,6,6,1,6,6,6,6,4,1,4,6,6,6,6,6,1,6,6,6,2,6,6,6,2,2,2,2,6,6,6,1,6,2,6,4,2,2,6,6,6,6,6,1,6,4,3,3,3,6,1,1,6,1,3,0,0,0,0,3,0,3,0,3,5,4,0,3,0,3,0,3,0,3,0,3,3,0,0,3,3,0,3,3,3,3,0,0,3,3,0,4,3,3,3,3,0,3,0,3,3,3,3,3,0,3,1,3,3,6,6,6,6,6,6,6,2,2,6,6,2,6,6,6,6,6,6,1,6,1,2,6,1,1,6,2,2,6,6,6,6,6,6,2,6,6,6,6,6,6,6,6,6,6,6,6,6,6,0,4,4,3,3,0,0,0,0,3,2,3,4,3,4,4,0,3,6,6,0,0,3,3,0,3,3,3,3,3,3,0,0,0,0,0,3,0,3,0,6,3,3,0,3,4,3,3,3,0,4,5,3,3,3,3,4,0,0,3,3,3,6,8,8,3,8,8,7,4,3,4,4,4,4,3,3,4,8,8,7,8,7,5,1,0,0,4,0,0,0,0,3,4,6,0,5,5,6,0,6,6,5,6,6,6,6,4,1,4,4,6,5,6,6,1,6,6,6,6,6,1,3,0,3,1,3,0,6,3,3,1,6,1,3,1,0,7,0,6,0,0,0,0,5,0,5,0,0,0,0,3,0,3,1,1,0,3,3,3,3,0,0,0,8,0,1,6,6,2,6,6,1,6,2,6,4,2,6,6,6,6,6,1,6,2,2,6,2,6,6,6,6,1,6,6,6,6,2,1,1,2,1,6,6,2,6,6,6,2,2,6,0,6,0,3,3,3,3,3,0,0,0,0,0,4,1,6,1,1,1,1,1,1,6,1,6,6,6,6,5,6,1,6,1,5,6,6,6,1,1,6,1,6,6,6,6,6,3,6,3,3,3,1,0,0,0,0,3,3,4,2,0,0,6,6,6,1,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,1,5,6,6,6,1,6,4,6,6,6,6,1,3,6,6,6,4,5,6,6,6,1,6,6,5,6,1,5,4,6,1,6,1,6,6,6,4,6,1,6,6,6,5,1,1,5,6,6,6,6,5,6,6,0,6,6,6,6,1,6,6,1,1,6,5,1,6,6,6,6,1,3,0,6,3,3,2,0,5,3,4,4,0,0,0,5,5,5,5,5,0,4,6,6,6,0,0,0,1,0,5,0,0,0,3,4,0,3,3,0,3,3,0,0,1,1,3,3,3,3,0,0,6,1,0,1,3,0,6,1,6,1,6,6,6,6,6,6,1,1,6,6,3,0,3,3,3,3,3,3,3,3,0,3,0,3,0,4,3,3,3,3,3,3,3,3,3,2,3,3,3,3,3,3,3,0,3,3,6,3,3,3,2,3,3,3,3,3,3,3,3,3,4,3,0,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,2,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,2,3,3,3,3,3,3,3,2,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,4,3,3,3,3,3,3,3,3,3,3,3,3,3,0,7,7,0,7,7,0,0,0,0,0,8,0,6,6,6,6,6,1,1,1,5,3,3,3,0,3,6,6,6,6,6,6,6,6,6,6,6,0,6,6,6,6,6,6,1,6,5,5,1,1,5,5,6,6,5,6,5,6,6,5,5,6,5,5,5,6,6,5,6,6,5,6,6,5,6,6,1,6,6,3,3,3,3,3,3,3,3,0,0,3,3,3,4,6,3,6,6,6,1,6,1,1,5,6,6,0,6,3,6,6,1,7,6,6,5,6,6,6,6,1,3,3,3,3,3,3,3,3,3,0,0,3,3,3,3,3,3,5,0,0,0,0,3,5,3,3,0,0,0,4,0,5,3,5,5,2,5,0,0,5,5,5,0,5,0,0,4,0,4,0,5,5,4,4,3,5,5,5,3,5,2,0,3,5,2,3,4,5,3,0,4,0,4,4,4,2,0,4,4,5,0,0,5,3,0,3,4,2,0,0,5,5,0,4,4,2,3,0,3,4,4,4,4,3,4,2,4,0,4,4,5,3,0,2,5,2,0,3,4,0,5,5,4,5,3,4,5,5,0,0,0,0,5,4,4,3,3,4,4,4,3,4,4,3,4,4,5,4,0,3,3,5,4,4,3,4,5,2,0,4,0,4,3,0,4,3,5,4,3,3,5,4,4,0,4,0,3,4,5,3,3,4,3,0,0,0,2,0,0,0,3,4,0,2,5,3,5,2,0,5,5,3,0,5,0,0,0,5,4,4,4,4,5,4,4,5,4,4,4,4,0,4,4,3,4,0,4,3,3,3,5,4,4,3,4,4,4,3,2,5,0,5,4,3,4,0,4,4,3,4,5,0,5,4,3,4,0,4,0,0,3,4,4,3,5,0,4,4,4,4,0,5,4,4,4,3,0,4,5,4,5,4,2,0,4,4,5,6,5,6,3,6,6,6,1,6,4,5,1,6,1,6,6,4,6,3,4,1,4,6,1,1,1,6,6,1,6,1,1,6,1,6,1,1,6,6,6,6,1,6,1,6,1,2,6,6,1,1,4,6,1,6,4,6,1,1,5,4,1,6,6,1,6,1,6,6,1,1,6,1,6,6,6,1,6,6,1,6,6,1,6,6,6,6,1,6,6,6,4,5,0,5,5,5,5,5,0,5,5,5,4,5,5,0,4,5,5,5,5,5,5,5,5,4,0,0,0,5,4,0,0,5,5,5,5,5,5,5,5,5,5,5,5,5,4,5,5,5,4,4,4,5,4,5,5,5,5,5,5,4,0,0,5,5,0,3,5,4,5,0,5,3,5,5,5,5,4,5,4,5,0,4,5,4,5,0,0,5,5,5,5,4,5,4,5,5,3,4,5,5,5,5,4,5,5,0,5,0,5,5,5,5,5,0,5,5,5,5,5,4,5,5,5,5,4,0,5,0,0,0,0,0,0,4,0,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,1,1,6,6,0,0,1,6,3,6,1,1,6,6,6,1,5,5,6,6,1,0,0,6,6,1,1,6,6,0,0,1,1,6,6,1,6,1,6,2,6,6,1,6,6,1,1,6,6,6,6,1,6,6,6,6,6,1,6,1,1,1,1,6,6,6,1,1,6,1,6,1,6,1,6,6,6,1,1,6,6,6,6,1,1,1,1,1,6,5,1,6,6,6,1,1,6,6,1,6,1,1,6,6,1,6,6,1,6,6,1,1,1,1,6,6,6,1,3,6,1,6,6,1,6,6,1,1,1,1,1,6,8,1,1,0,6,1,6,6,1,6,6,1,1,6,6,6,1,0,6,4,1,6,6,1,1,1,3,6,6,6,1,6,1,1,1,0,1,6,6,6,6,6,6,6,1,0,0,1,7,0,0,0,2,7,0,1,6,6,6,1,6,6,6,6,6,6,5,4,5,6,5,6,6,6,2,6,6,6,6,6,6,6,6,6,6,6,6,6,6,6,1,5,6,6,1,6,6,1,6,0,0,1,6,6,6,1,6,6,6,6,1,6,6,6,6,2,0,0,6,6,6,1,3,0,5,5,1,6,1,6,6,6,6,6,1,6,6,0,0,0,0,0,0,0,0,5,5,4,4,0,0,0,0,1,6,3,6,1,1,6,6,6,6,6,6,1,6,6,6,6,0,1,4,6,6,1,6,1,5,6,5,6,6,6,6,5,1,5,1,1,6,0,0,3,2,3,6,6,6,4,6,6,6,6,6,1,4,3,1,1,6,3,1,3,1,3,4,6,6,1,1,4,1,6,1,6,1,6,4,1,6,6,0,6,3,6,3,3,3,3,3,3,3,3,3,0,0,6,0,6,3,3,3,5,3,3,0,0,3,3],\"xaxis\":\"x\",\"yaxis\":\"y\",\"type\":\"histogram\"}],                        {\"template\":{\"data\":{\"histogram2dcontour\":[{\"type\":\"histogram2dcontour\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"choropleth\":[{\"type\":\"choropleth\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"histogram2d\":[{\"type\":\"histogram2d\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"heatmap\":[{\"type\":\"heatmap\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"heatmapgl\":[{\"type\":\"heatmapgl\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"contourcarpet\":[{\"type\":\"contourcarpet\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"contour\":[{\"type\":\"contour\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"surface\":[{\"type\":\"surface\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"mesh3d\":[{\"type\":\"mesh3d\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"scatter\":[{\"fillpattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2},\"type\":\"scatter\"}],\"parcoords\":[{\"type\":\"parcoords\",\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterpolargl\":[{\"type\":\"scatterpolargl\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"scattergeo\":[{\"type\":\"scattergeo\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterpolar\":[{\"type\":\"scatterpolar\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"scattergl\":[{\"type\":\"scattergl\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatter3d\":[{\"type\":\"scatter3d\",\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scattermapbox\":[{\"type\":\"scattermapbox\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterternary\":[{\"type\":\"scatterternary\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scattercarpet\":[{\"type\":\"scattercarpet\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}]},\"layout\":{\"autotypenumbers\":\"strict\",\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"hovermode\":\"closest\",\"hoverlabel\":{\"align\":\"left\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"bgcolor\":\"#E5ECF6\",\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"ternary\":{\"bgcolor\":\"#E5ECF6\",\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]]},\"xaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"automargin\":true,\"zerolinewidth\":2},\"yaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"automargin\":true,\"zerolinewidth\":2},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"geo\":{\"bgcolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"subunitcolor\":\"white\",\"showland\":true,\"showlakes\":true,\"lakecolor\":\"white\"},\"title\":{\"x\":0.05},\"mapbox\":{\"style\":\"light\"}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"value\"}},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"\"},\"showticklabels\":false},\"legend\":{\"title\":{\"text\":\"variable\"},\"tracegroupgap\":0},\"title\":{\"text\":\"Original Density Class\"},\"barmode\":\"relative\",\"showlegend\":false},                        {\"responsive\": true}                    ).then(function(){\n","                            \n","var gd = document.getElementById('99644a6c-1b91-4489-86b4-9dd06224f7a0');\n","var x = new MutationObserver(function (mutations, observer) {{\n","        var display = window.getComputedStyle(gd).display;\n","        if (!display || display === 'none') {{\n","            console.log([gd, 'removed!']);\n","            Plotly.purge(gd);\n","            observer.disconnect();\n","        }}\n","}});\n","\n","// Listen for the removal of the full notebook cells\n","var notebookContainer = gd.closest('#notebook-container');\n","if (notebookContainer) {{\n","    x.observe(notebookContainer, {childList: true});\n","}}\n","\n","// Listen for the clearing of the current output cell\n","var outputEl = gd.closest('.output');\n","if (outputEl) {{\n","    x.observe(outputEl, {childList: true});\n","}}\n","\n","                        })                };                            </script>        </div>\n","</body>\n","</html>"]},"metadata":{},"output_type":"display_data"}],"source":["# Convert One_hot encode\n","class_reverted = np.argmax(label, axis=1)\n","\n","fig = px.histogram(class_reverted, histnorm='probability density' ,title='Original Density Class', text_auto=\".2%\")\n","fig.update_layout(showlegend=False, yaxis_title='')\n","fig.update_yaxes(showticklabels=False)\n","fig.show()"]},{"cell_type":"code","execution_count":13,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":1068,"status":"ok","timestamp":1729622670672,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"4gF3OZM8eDCC","outputId":"5da82455-ea6d-4963-dd09-d5356e895e6e"},"outputs":[{"data":{"text/plain":["['/content/drive/MyDrive/Portfolio/treat_cancer/./data_files/y_test_DISTIL.joblib']"]},"execution_count":13,"metadata":{},"output_type":"execute_result"}],"source":["# Make Split train, test\n","X_train, X_test, y_train, y_test = train_test_split(\n","    text,\n","    label,\n","    test_size=0.2,\n","    random_state=34,\n","    stratify=label\n",")\n","\n","# save\n","joblib.dump(X_train, path + './data_files/X_train_DISTIL.joblib')\n","joblib.dump(X_test, path + './data_files/X_test_DISTIL.joblib')\n","joblib.dump(y_train, path + './data_files/y_train_DISTIL.joblib')\n","joblib.dump(y_test, path + './data_files/y_test_DISTIL.joblib')"]},{"cell_type":"code","execution_count":14,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":337,"status":"ok","timestamp":1729622680158,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"vHb7K5TOeDCC","outputId":"2d800a21-0f03-418e-e2d5-3538dc8238a7"},"outputs":[{"name":"stdout","output_type":"stream","text":["X_train shape: 2652\n","X_test shape: 664\n","y_train shape: 2652\n","y_test shape: 664\n"]}],"source":["print(f\"X_train shape: {len(X_train)}\\n\" \\\n","      f\"X_test shape: {len(X_test)}\\n\" \\\n","      f\"y_train shape: {len(y_train)}\\n\" \\\n","      f\"y_test shape: {len(y_test)}\")"]},{"cell_type":"code","execution_count":15,"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":542},"executionInfo":{"elapsed":339,"status":"ok","timestamp":1729622685898,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"uXDbRfbieDCD","outputId":"96c0026d-d797-49bb-e328-0016053d5901"},"outputs":[{"data":{"text/html":["<html>\n","<head><meta charset=\"utf-8\" /></head>\n","<body>\n","    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n","        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.35.2.min.js\"></script>                <div id=\"53b117c7-9a51-4a0d-b1f3-abbf44bcadc1\" class=\"plotly-graph-div\" style=\"height:525px; width:100%;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"53b117c7-9a51-4a0d-b1f3-abbf44bcadc1\")) {                    Plotly.newPlot(                        \"53b117c7-9a51-4a0d-b1f3-abbf44bcadc1\",                        [{\"alignmentgroup\":\"True\",\"bingroup\":\"x\",\"histnorm\":\"probability density\",\"hovertemplate\":\"variable=0\\u003cbr\\u003evalue=%{x}\\u003cbr\\u003eprobability density=%{y}\\u003cextra\\u003e\\u003c\\u002fextra\\u003e\",\"legendgroup\":\"0\",\"marker\":{\"color\":\"#636efa\",\"pattern\":{\"shape\":\"\"}},\"name\":\"0\",\"offsetgroup\":\"0\",\"orientation\":\"v\",\"showlegend\":true,\"texttemplate\":\"%{value:.2%}\",\"x\":[3,6,0,3,1,0,0,1,3,3,0,6,4,6,5,4,0,0,0,3,3,3,3,3,1,2,6,6,3,4,4,3,5,3,6,3,6,6,3,0,1,3,6,0,3,6,6,5,1,3,6,1,1,6,0,3,6,3,5,0,0,1,6,0,6,3,6,6,6,6,6,6,1,3,0,1,6,6,6,6,6,0,1,1,2,6,1,6,6,3,6,6,5,6,0,6,3,6,1,0,0,6,0,3,6,6,5,3,3,6,0,6,6,6,0,3,7,0,4,0,2,1,1,3,3,0,0,6,3,3,5,5,3,6,0,0,0,1,3,3,3,3,6,3,6,6,0,6,4,4,6,6,0,3,0,6,3,6,3,4,3,6,6,1,1,1,1,6,6,6,3,1,6,0,5,3,4,5,6,6,3,6,1,5,3,2,6,3,3,3,5,0,6,1,6,6,6,0,6,1,0,1,1,1,6,0,2,0,6,1,0,6,6,6,5,0,3,0,3,3,7,3,0,0,0,5,6,1,6,5,6,1,6,3,6,8,0,0,0,1,1,5,5,3,3,2,1,1,6,3,0,3,2,3,8,4,4,8,1,3,3,3,1,6,3,0,3,0,3,6,3,6,4,1,1,0,6,5,3,0,5,0,0,0,6,6,3,0,6,6,5,3,0,6,0,6,3,6,3,6,1,6,3,6,1,3,6,6,0,0,8,3,3,3,6,1,3,4,5,6,1,6,3,0,6,6,3,6,3,6,5,1,6,3,6,4,3,6,6,0,0,6,5,6,1,6,1,0,0,6,3,0,1,1,6,0,5,0,6,0,4,1,6,0,3,1,1,1,5,4,5,5,3,0,6,6,0,3,6,0,6,3,6,6,6,3,8,0,6,0,8,0,6,4,3,4,5,1,5,6,3,3,0,1,3,0,3,6,1,6,6,0,3,5,0,6,0,3,0,3,3,3,6,3,2,0,3,6,0,6,3,6,0,1,5,3,3,3,1,6,1,1,4,1,3,5,2,3,6,6,6,6,1,5,0,0,3,5,5,6,4,6,6,3,5,6,1,3,6,6,3,6,6,6,3,3,6,0,4,3,6,3,4,1,6,6,6,1,8,6,6,3,4,6,5,4,0,1,6,6,6,3,3,6,6,3,1,1,0,7,1,6,6,6,3,4,6,6,6,6,1,1,4,6,6,6,0,4,1,1,0,0,1,1,1,1,6,3,0,6,3,6,1,6,6,6,1,1,0,4,3,6,4,1,2,6,6,1,5,4,2,6,3,1,0,0,1,3,0,3,6,0,6,1,6,3,3,1,1,0,4,6,1,6,3,5,3,1,1,3,6,6,2,4,4,3,5,3,0,0,6,5,6,0,2,6,6,1,3,0,4,3,1,3,1,3,6,0,0,6,1,5,2,0,0,0,6,4,6,6,6,0,3,0,0,3,0,0,3,1,1,1,4,6,1,3,4,7,3,1,3,5,0,6,0,6,0,3,5,6,5,3,0,4,1,0,0,4,0,3,1,6,4,3,3,6,4,3,5,6,4,1,1,0,3,6,6,3,6,3,0,4,1,3,6,5,1,2,5,6,2,5,6,3,3,1,1,0,3,4,3,4,3,6,0,3,6,3,5,3,3,4,6,4,0,6,6,0,3,0,6,5,6,6,3,6,5,6,3,6,5,6,0,3,6,1,0,1,1,4,6,3,3,1,4,3,0,6,3,0,1,5,3,3,3,6,3,6,4,3,0,1,2,6,3,6,5,4,1,3,4,4,4,3,5,0,6,1,3,1,3,7,3,6,3,3,3,5,0,0,3,6,0,3,3,5,6,0,6,5,4,8,6,1,6,0,3,0,0,1,5,3,5,3,0,1,3,3,6,0,0,1,6,6,3,5,6,3,1,1,6,5,3,6,3,5,1,6,6,4,3,0,6,1,0,6,2,6,6,3,6,6,6,3,6,6,0,3,1,0,0,4,6,3,6,1,0,6,0,6,5,6,5,0,4,4,3,6,6,0,3,0,6,6,6,3,6,6,0,6,6,0,4,6,0,2,4,3,3,6,4,3,1,5,4,6,0,5,1,3,0,6,1,1,6,8,6,6,3,8,0,3,1,0,7,0,0,1,6,3,5,0,1,3,5,4,3,6,3,5,3,6,6,6,0,3,6,2,3,1,4,1,4,0,6,1,4,6,3,6,3,2,5,6,0,5,4,4,2,0,6,0,0,6,0,3,3,6,0,0,0,4,3,5,4,2,6,6,5,6,1,0,5,0,5,0,3,5,3,3,6,6,2,6,1,6,6,3,1,6,6,6,3,6,3,1,6,3,6,3,6,6,1,6,5,5,0,3,1,6,6,3,6,5,5,0,0,1,6,1,6,6,1,6,4,6,1,3,1,4,3,0,3,6,4,5,3,1,0,4,6,3,1,6,6,1,3,8,1,4,4,0,5,0,6,0,1,5,3,0,1,1,6,1,3,6,0,0,4,7,0,0,6,6,3,1,2,4,4,1,1,6,5,0,4,1,6,1,3,3,3,6,0,4,3,2,3,3,6,0,0,0,0,6,6,6,0,3,6,2,0,6,6,7,8,6,6,1,5,6,8,4,3,3,3,1,3,3,3,1,1,6,5,2,6,1,0,1,0,5,5,3,0,3,4,5,0,3,5,6,3,3,1,1,2,1,3,1,6,0,6,3,6,6,4,0,6,6,6,6,3,6,1,3,0,0,0,3,6,6,3,0,6,6,0,6,5,5,3,6,3,6,0,3,1,1,5,6,5,8,1,3,3,6,6,6,3,0,3,1,3,6,4,0,3,6,7,3,1,3,6,6,1,0,3,0,3,1,0,1,6,6,6,6,4,0,3,5,4,6,1,3,0,0,6,3,6,5,3,6,6,6,6,6,6,6,6,3,6,3,4,3,6,6,3,7,1,6,3,1,2,1,3,6,0,4,6,3,5,6,6,1,5,4,6,3,6,0,1,1,6,4,1,6,6,3,4,3,0,0,6,6,6,6,3,0,6,6,5,6,6,6,0,2,0,1,4,3,1,0,0,1,2,2,0,6,6,0,6,6,0,3,6,6,4,6,0,2,5,3,8,6,3,6,3,1,6,3,6,6,1,6,5,1,0,3,5,3,5,4,1,0,6,0,6,3,6,3,4,8,3,4,0,3,1,6,6,5,5,6,5,4,6,6,8,4,1,5,0,2,1,6,5,6,0,3,6,3,1,5,1,3,5,3,0,6,0,3,6,3,5,6,3,6,1,2,1,0,4,1,6,5,4,0,6,3,4,5,8,1,1,0,0,3,0,6,2,3,4,3,3,1,6,4,6,3,4,5,6,3,6,6,5,1,6,0,0,6,2,0,4,6,1,1,3,6,6,3,1,6,6,6,5,1,6,4,4,6,1,8,4,6,0,1,1,5,1,0,5,0,3,4,1,1,3,0,1,4,6,4,3,1,5,0,0,0,0,4,0,5,4,3,0,3,1,3,5,2,3,1,8,3,6,0,0,0,0,6,3,0,0,6,3,1,1,6,0,6,0,6,3,6,6,6,0,3,0,6,1,6,6,6,6,6,3,3,0,6,6,6,6,1,6,0,4,6,6,3,4,2,6,0,4,3,0,0,5,4,6,0,6,4,5,1,6,0,3,0,0,3,6,3,2,3,3,4,3,1,6,4,3,3,6,6,0,6,3,8,0,0,1,0,5,4,6,5,5,3,3,0,0,6,6,3,3,0,2,6,5,0,1,2,4,5,3,6,2,3,3,0,5,3,1,5,1,2,0,6,0,1,3,7,3,0,3,5,5,0,6,0,6,1,6,4,0,3,3,1,3,3,3,0,2,6,1,0,3,1,3,5,1,6,1,6,6,5,1,4,3,5,3,3,3,5,0,0,0,3,1,5,3,0,0,6,1,0,0,0,6,0,3,3,3,5,6,0,0,0,3,6,6,3,3,6,3,4,0,3,2,6,6,0,0,6,6,4,5,6,0,6,3,3,6,6,0,1,0,3,5,7,4,5,3,6,0,1,6,6,4,5,1,5,4,1,6,4,8,6,5,0,0,6,5,2,0,3,1,2,5,3,1,6,6,3,6,5,5,1,2,6,6,6,0,3,0,3,6,3,6,6,4,0,5,3,0,6,1,0,3,0,6,1,1,0,1,3,6,0,6,1,0,6,3,6,6,4,3,4,3,3,5,6,3,1,0,0,3,3,3,4,6,1,6,6,6,3,0,4,6,1,6,6,0,3,3,6,4,2,3,1,0,5,6,0,0,0,1,4,6,1,6,1,3,6,0,0,0,5,3,4,6,6,3,3,0,3,5,6,4,5,6,1,4,5,1,0,6,1,5,0,6,3,0,3,1,3,6,3,3,3,5,6,5,1,6,6,0,6,6,0,6,1,0,3,5,6,3,6,1,5,1,0,0,4,6,2,0,1,6,6,0,6,6,1,0,5,1,3,3,1,1,3,6,1,7,1,6,6,1,6,2,5,1,0,1,5,6,3,6,4,0,0,0,6,6,3,5,6,5,2,1,6,2,0,0,0,3,3,0,3,5,6,3,1,1,4,6,6,5,4,6,0,4,1,3,4,6,5,4,6,1,1,0,6,6,6,8,4,3,6,4,6,0,6,6,0,6,6,3,3,6,3,0,6,3,6,4,6,0,3,3,3,4,3,6,6,6,6,0,3,1,1,6,3,1,5,3,6,0,4,4,5,5,6,1,0,1,6,6,1,3,4,5,6,0,0,1,3,0,0,1,0,0,6,3,3,6,1,3,6,6,6,0,3,6,4,2,3,3,0,6,6,6,0,8,4,6,6,6,3,6,2,6,0,4,3,0,4,6,0,4,0,3,4,0,3,6,0,3,1,3,3,4,3,6,6,3,1,6,0,1,4,4,4,0,0,1,0,5,6,3,0,6,5,0,3,3,3,6,0,3,3,6,3,3,6,1,6,3,5,6,6,2,3,3,3,6,0,5,0,6,1,4,5,1,3,6,5,1,1,2,2,6,1,6,0,6,1,0,6,3,8,6,1,1,6,3,6,0,7,0,1,3,0,1,1,6,4,6,6,1,3,6,3,3,3,6,5,1,6,2,6,6,5,5,3,5,0,1,6,0,4,6,6,0,0,3,3,3,6,0,8,6,3,5,2,1,3,6,0,0,3,3,1,1,6,6,4,0,5,6,6,6,6,6,4,6,3,1,6,1,4,1,6,6,3,3,0,6,3,4,6,8,3,2,6,5,3,3,3,6,7,5,1,2,3,5,3,6,3,3,6,0,6,8,3,3,6,3,0,3,5,1,3,6,6,6,4,6,3,1,6,3,5,4,3,0,1,0,3,3,1,0,3,6,6,6,0,1,6,6,6,6,2,6,1,5,3,1,3,4,6,3,0,6,0,6,1,6,6,0,3,3,3,6,5,4,6,1,6,0,0,6,0,1,1,1,8,5,1,5,4,6,6,5,6,5,6,0,1,4,6,3,2,1,3,0,3,6,6,1,1,0,6,6,3,0,4,4,3,1,1,3,6,6,0,3,3,3,6,0,0,1,2,1,6,0,6,6,0,1,6,1,0,5,6,1,5,6,3,3,5,6,0,3,2,6,8,6,0,5,4,5,6,6,6,5,6,5,3,5,5,6,0,6,0,0,4,6,5,6,5,3,6,6,6,6,6,6,0,6,6,0,4,1,6,1,0,3,3,6,1,0,5,6,6,6,1,0,1,3,0,6,1,5,6,1,5,6,3,0,3,0,3,4,6,1,3,6,6,1,6,1,3,6,0,1,3,1,4,1,0,6,4,0,4,6,6,3,4,2,6,0,6,0,5,6,4,0,3,0,6,5,0,6,3,6,6,5,3,1,0,3,3,0,0,6,1,6,3,6,6,0,3,4,3,6,4,3,3,5,2,3,3,6,5,6,1,4,1,0,0,5,6,5,3,2,6,6,0,3,4,6,6,0,1,1,6,3,6,1,0],\"xaxis\":\"x\",\"yaxis\":\"y\",\"type\":\"histogram\"}],                        {\"template\":{\"data\":{\"histogram2dcontour\":[{\"type\":\"histogram2dcontour\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"choropleth\":[{\"type\":\"choropleth\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"histogram2d\":[{\"type\":\"histogram2d\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"heatmap\":[{\"type\":\"heatmap\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"heatmapgl\":[{\"type\":\"heatmapgl\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"contourcarpet\":[{\"type\":\"contourcarpet\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"contour\":[{\"type\":\"contour\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"surface\":[{\"type\":\"surface\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"},\"colorscale\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]]}],\"mesh3d\":[{\"type\":\"mesh3d\",\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}],\"scatter\":[{\"fillpattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2},\"type\":\"scatter\"}],\"parcoords\":[{\"type\":\"parcoords\",\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterpolargl\":[{\"type\":\"scatterpolargl\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"bar\":[{\"error_x\":{\"color\":\"#2a3f5f\"},\"error_y\":{\"color\":\"#2a3f5f\"},\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"bar\"}],\"scattergeo\":[{\"type\":\"scattergeo\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterpolar\":[{\"type\":\"scatterpolar\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"histogram\":[{\"marker\":{\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"histogram\"}],\"scattergl\":[{\"type\":\"scattergl\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatter3d\":[{\"type\":\"scatter3d\",\"line\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scattermapbox\":[{\"type\":\"scattermapbox\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scatterternary\":[{\"type\":\"scatterternary\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"scattercarpet\":[{\"type\":\"scattercarpet\",\"marker\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}}}],\"carpet\":[{\"aaxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"baxis\":{\"endlinecolor\":\"#2a3f5f\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"minorgridcolor\":\"white\",\"startlinecolor\":\"#2a3f5f\"},\"type\":\"carpet\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#EBF0F8\"},\"line\":{\"color\":\"white\"}},\"header\":{\"fill\":{\"color\":\"#C8D4E3\"},\"line\":{\"color\":\"white\"}},\"type\":\"table\"}],\"barpolar\":[{\"marker\":{\"line\":{\"color\":\"#E5ECF6\",\"width\":0.5},\"pattern\":{\"fillmode\":\"overlay\",\"size\":10,\"solidity\":0.2}},\"type\":\"barpolar\"}],\"pie\":[{\"automargin\":true,\"type\":\"pie\"}]},\"layout\":{\"autotypenumbers\":\"strict\",\"colorway\":[\"#636efa\",\"#EF553B\",\"#00cc96\",\"#ab63fa\",\"#FFA15A\",\"#19d3f3\",\"#FF6692\",\"#B6E880\",\"#FF97FF\",\"#FECB52\"],\"font\":{\"color\":\"#2a3f5f\"},\"hovermode\":\"closest\",\"hoverlabel\":{\"align\":\"left\"},\"paper_bgcolor\":\"white\",\"plot_bgcolor\":\"#E5ECF6\",\"polar\":{\"bgcolor\":\"#E5ECF6\",\"angularaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"radialaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"ternary\":{\"bgcolor\":\"#E5ECF6\",\"aaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"baxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"},\"caxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\"}},\"coloraxis\":{\"colorbar\":{\"outlinewidth\":0,\"ticks\":\"\"}},\"colorscale\":{\"sequential\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"sequentialminus\":[[0.0,\"#0d0887\"],[0.1111111111111111,\"#46039f\"],[0.2222222222222222,\"#7201a8\"],[0.3333333333333333,\"#9c179e\"],[0.4444444444444444,\"#bd3786\"],[0.5555555555555556,\"#d8576b\"],[0.6666666666666666,\"#ed7953\"],[0.7777777777777778,\"#fb9f3a\"],[0.8888888888888888,\"#fdca26\"],[1.0,\"#f0f921\"]],\"diverging\":[[0,\"#8e0152\"],[0.1,\"#c51b7d\"],[0.2,\"#de77ae\"],[0.3,\"#f1b6da\"],[0.4,\"#fde0ef\"],[0.5,\"#f7f7f7\"],[0.6,\"#e6f5d0\"],[0.7,\"#b8e186\"],[0.8,\"#7fbc41\"],[0.9,\"#4d9221\"],[1,\"#276419\"]]},\"xaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"automargin\":true,\"zerolinewidth\":2},\"yaxis\":{\"gridcolor\":\"white\",\"linecolor\":\"white\",\"ticks\":\"\",\"title\":{\"standoff\":15},\"zerolinecolor\":\"white\",\"automargin\":true,\"zerolinewidth\":2},\"scene\":{\"xaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2},\"yaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2},\"zaxis\":{\"backgroundcolor\":\"#E5ECF6\",\"gridcolor\":\"white\",\"linecolor\":\"white\",\"showbackground\":true,\"ticks\":\"\",\"zerolinecolor\":\"white\",\"gridwidth\":2}},\"shapedefaults\":{\"line\":{\"color\":\"#2a3f5f\"}},\"annotationdefaults\":{\"arrowcolor\":\"#2a3f5f\",\"arrowhead\":0,\"arrowwidth\":1},\"geo\":{\"bgcolor\":\"white\",\"landcolor\":\"#E5ECF6\",\"subunitcolor\":\"white\",\"showland\":true,\"showlakes\":true,\"lakecolor\":\"white\"},\"title\":{\"x\":0.05},\"mapbox\":{\"style\":\"light\"}}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"value\"}},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0],\"title\":{\"text\":\"\"},\"showticklabels\":false},\"legend\":{\"title\":{\"text\":\"variable\"},\"tracegroupgap\":0},\"title\":{\"text\":\"y_train Density Class\"},\"barmode\":\"relative\",\"showlegend\":false},                        {\"responsive\": true}                    ).then(function(){\n","                            \n","var gd = document.getElementById('53b117c7-9a51-4a0d-b1f3-abbf44bcadc1');\n","var x = new MutationObserver(function (mutations, observer) {{\n","        var display = window.getComputedStyle(gd).display;\n","        if (!display || display === 'none') {{\n","            console.log([gd, 'removed!']);\n","            Plotly.purge(gd);\n","            observer.disconnect();\n","        }}\n","}});\n","\n","// Listen for the removal of the full notebook cells\n","var notebookContainer = gd.closest('#notebook-container');\n","if (notebookContainer) {{\n","    x.observe(notebookContainer, {childList: true});\n","}}\n","\n","// Listen for the clearing of the current output cell\n","var outputEl = gd.closest('.output');\n","if (outputEl) {{\n","    x.observe(outputEl, {childList: true});\n","}}\n","\n","                        })                };                            </script>        </div>\n","</body>\n","</html>"]},"metadata":{},"output_type":"display_data"}],"source":["# Flatten array multidimensions after split\n","class_reverted = np.argmax(y_train, axis=1)\n","\n","fig = px.histogram(class_reverted, histnorm='probability density' ,title='y_train Density Class', text_auto=\".2%\")\n","fig.update_layout(showlegend=False, yaxis_title='')\n","fig.update_yaxes(showticklabels=False)\n","fig.show()"]},{"cell_type":"code","execution_count":16,"metadata":{"executionInfo":{"elapsed":377,"status":"ok","timestamp":1729622693431,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"u7SkNnyfeDCE"},"outputs":[],"source":["# Create validation data\n","X_train, X_val, y_train, y_val = train_test_split(X_train, y_train, test_size=0.1, random_state=27, stratify=y_train)"]},{"cell_type":"code","execution_count":17,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":312,"status":"ok","timestamp":1729622697000,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"IP48It7teDCE","outputId":"0feb0781-e57c-4900-eb44-903c1837df6e"},"outputs":[{"name":"stdout","output_type":"stream","text":["X_train shape: 2386\n","X_val shape: 266\n","y_train shape: 2386\n","y_val shape: 266\n"]}],"source":["print(f\"X_train shape: {len(X_train)}\\n\" \\\n","      f\"X_val shape: {len(X_val)}\\n\" \\\n","      f\"y_train shape: {len(y_train)}\\n\" \\\n","      f\"y_val shape: {len(y_val)}\")"]},{"cell_type":"markdown","metadata":{"id":"a2WZ6310eDCF"},"source":["### Create Model"]},{"cell_type":"code","execution_count":18,"metadata":{"executionInfo":{"elapsed":303,"status":"ok","timestamp":1729622702351,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"LUSPHyPEeDCG"},"outputs":[],"source":["if train_mode:\n","\n","  def build_model(hp):\n","      # Hyperparameters\n","      dropout_rate = hp.Float('dropout_rate', min_value=0.1, max_value=0.5, step=0.1)\n","      learning_rate = hp.Choice('learning_rate', values=[1e-2, 1e-3, 1e-4])\n","\n","      # Model\n","      bert_model = DistilBertClassifier.from_preset(\n","          model_pre,\n","          num_classes=num_class,\n","          activation=\"softmax\",\n","          dropout=dropout_rate)\n","\n","      # Compile\n","      bert_model.compile(\n","          optimizer = Adafactor(learning_rate=learning_rate),\n","          loss='categorical_crossentropy',\n","          metrics=['AUC', 'categorical_accuracy', 'precision', 'recall'])\n","\n","      return bert_model\n","\n","  # Config Callbacks\n","  early_stopping = EarlyStopping(monitor='AUC', patience=5, restore_best_weights=True)\n","  csv_logger = CSVLogger(path + 'train_log_distil.csv', append=True)\n","\n","  # Config RandomSearch\n","  tuner = RandomSearch(\n","      build_model,\n","      objective='AUC',\n","      max_trials=10, # distinct combinations hyperparameters\n","      max_retries_per_trial=2,\n","      directory= path,\n","      project_name='DistilBertClass'\n","  )\n","\n","  # Balance classes\n","  class_weights = compute_class_weight('balanced', classes=np.unique(y_train), y=y_train.ravel())\n","  class_weights = {i: class_weights[i] for i in range(len(class_weights))}"]},{"cell_type":"markdown","metadata":{"id":"Arwq93g_eDCI"},"source":["#### Fit"]},{"cell_type":"code","execution_count":19,"metadata":{"executionInfo":{"elapsed":312,"status":"ok","timestamp":1729622705301,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"8pBev_3JeDCJ"},"outputs":[],"source":["if train_mode:\n","  # Batch size\n","  batch_sizes = [64]\n","  epoch = 12\n","\n","  # Fit Model\n","  for batch_size in batch_sizes:\n","      print(f\"Train with batch_size: {batch_size}\")\n","      history = tuner.search(\n","          X_train,\n","          y_train,\n","          validation_data=(X_val, y_val),\n","          epochs=epoch,\n","          batch_size=batch_size,\n","          # class_weight=class_weights, # explode memory in colab\n","          callbacks=[early_stopping, csv_logger])\n","\n","      best_models = tuner.get_best_models(num_models=1)\n","      keras.saving.save_model(best_models[0], path + f'best_model_distilbertclassifier_{batch_size}.keras')\n","\n","  tuner.results_summary()"]},{"cell_type":"markdown","metadata":{"id":"kta9-pOxeDCM"},"source":["# Evaluate Model"]},{"cell_type":"code","execution_count":22,"metadata":{"executionInfo":{"elapsed":18993,"status":"ok","timestamp":1729622822446,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"AR0Eow08eDCM"},"outputs":[],"source":["diltil_model = keras.models.load_model(path + \"./best_models/DistilbertClassifier_32.keras\")"]},{"cell_type":"code","execution_count":24,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":25499,"status":"ok","timestamp":1729622878721,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"z1aaG1QphvaW","outputId":"d296fd43-b2e1-4be4-cede-418538e2e13a"},"outputs":[{"name":"stdout","output_type":"stream","text":["\u001b[1m21/21\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m24s\u001b[0m 749ms/step\n"]}],"source":["# Predict\n","y_pred = diltil_model.predict(X_test)\n","y_pred_class = np.argmax(y_pred, axis=1)\n","y_true = np.argmax(y_test, axis=1)"]},{"cell_type":"code","execution_count":25,"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"elapsed":328,"status":"ok","timestamp":1729622904511,"user":{"displayName":"Roberto Balbinotti","userId":"07229386195566093991"},"user_tz":180},"id":"WVP9nCeBiJm8","outputId":"90124433-e303-4f63-9162-6920727f3a01"},"outputs":[{"data":{"text/plain":["['/content/drive/MyDrive/Portfolio/treat_cancer/./data_files/y_pred_class_distil.joblib']"]},"execution_count":25,"metadata":{},"output_type":"execute_result"}],"source":["joblib.dump(y_pred, path + './data_files/y_pred_disltil.joblib')\n","joblib.dump(y_pred_class, path + './data_files/y_pred_class_distil.joblib')"]}],"metadata":{"accelerator":"GPU","colab":{"gpuType":"T4","provenance":[]},"kernelspec":{"display_name":".venv","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.11.9"}},"nbformat":4,"nbformat_minor":0}
